<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
	"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">

<!-- Mirrored from informationr.net/ir/13-3/paper353.html by HTTrack Website Copier/3.x [XR&CO'2010], Sat, 25 Dec 2010 19:13:58 GMT -->
<head>
<title>The information environment of researchers in molecular medicine</title>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1" />
<link href="../IRstyle2.css" rel="stylesheet" media="screen" title="serif" />
<link rel="alternate stylesheet" type="text/css" media="screen" title="sans" href="../IRstylesans.css" />
<link rev="made" href="mailto:t.d.wilson@shef.ac.uk" />

<!--Enter appropriate data in the content fields-->
<meta name="dc.title" content="The information environment of researchers in molecular medicine" />
<meta name="dc.creator" content="Annikki Roos, Sanna Kumpulainen, Kalervo Järvelin, Turid Hedlund" />
<meta name="dc.subject" content="The information environment of researchers in molecular medicine" />
<meta name="dc.description" content="Introduction. Describes and analyses the information environment of research work in molecular medicine. We presume an interdependence between the information environment, the research process and the related work tasks. Method. This is a qualitative case study using mixed methods. Empirical data were gathered using two surveys and six semi-structured thematic interviews. Analysis. Analysis was carried out on the merged data of the two surveys. Respondents were divided into groups and information resources were categorized.  Results. The information environment consists of broad categories of data, tools, published material and interpersonal communication. The usage of portals and other integrated resources was substantial. The role of PubMed was central in searching scientific facts. Google was used to locate general Web pages, research groups, methods and tools. Interpersonal communication seemed to be effective in providing information about methods and tools among groups which do laboratory work. The problems reported concerned lack of knowledge about useful resources and how to use them properly, some query formulation problems were also reported, e.g. problems with acronyms, personal names and gene name synonymy. Conclusions. The information related tasks occupied much of the researchers' time. If information problems were solved, more time to the research would be released. The database and tool interfaces should be easier to use. " /> 
<meta name="dc.subject.keywords" content="information use, information seeking behaviour, molecular medicine, information environment" />

<meta name="robots" content="all" />
<meta name="dc.publisher" content="Professor T.D. Wilson" />
<meta name="dc.coverage.placename" content="global" />
<meta name="dc.type" content="text" />
<meta name="dc.identifier" scheme="ISSN" content="1368-1613" />
<meta name="dc.identifier" scheme="URI" content="paper353.html" />
<meta name="dc.relation.IsPartOf" content="infres133.html" />
<meta name="dc.format" content="text/html" />
<meta name="dc.language" content="en" />
<meta name="dc.rights" content="http://creativecommons.org/licenses/by-nd-nc/1.0/" />
<meta name="dc.date.available" content="2008-09-15" />

<script language="javascript" type="text/javascript">

		var flag;
		flag = true;
		function doChangeFont()
		{
			if (flag)
			{
			var htmlDoc = document.getElementsByTagName('head').item(0);
			var css = document.createElement('link');
			css.setAttribute('rel', 'stylesheet');
			css.setAttribute('type', 'text/css');
			css.setAttribute('href', '../sans.css');
			htmlDoc.appendChild(css);
			flag = false;
			} 
			else
			{
			var htmlDoc = document.getElementsByTagName('head').item(0);
			var css = document.createElement('link');
			css.setAttribute('rel', 'stylesheet');
			css.setAttribute('type', 'text/css');
			css.setAttribute('href', '../IRstyle2.css');
			htmlDoc.appendChild(css);
			flag = true;
			}	
		}
		
	</script>
	
<style type="text/css">
.button {
	width: 45em;
	padding: 0 0 0 0;
	font-family: Verdana, Lucida, Geneva, Helvetica, Arial, sans-serif;
	font-size: small;
	font-weight: bold;  
	background-color: #ffffff;
	color: #000000;
	display: inline;
	text-align: center;
	}
		

.button ul {
		list-style: none;
		margin: 0;
		padding: 0;
		border: none;
		display: inline;
		}
		
.button li {
		margin: 0;
		font-family: Verdana, Lucida, Geneva, Helvetica, Arial, sans-serif;
	    font-size: small;
	    font-weight: bold;  
		background-color: #fff000<!-- #2175bc; -->
		color: #000000;
		text-decoration: none;
		display: inline;
		}

.button li a:hover {
		background-color: azure;
		color: #ff0000;
		width: auto;
		}
		
fieldset {
    padding: .5em;
    background: white;
    border: 1px dotted #5E96FD;
    margin-left: 15px;
    margin-right: 15px;
    margin-top: .5em;
	}

legend {
    color: white;
    background-color: #5E96FD;
    font-size: medium;
    padding: .1ex .5ex;
    border-right: 1px solid navy;
    border-bottom: 1px solid navy;
    font-weight: bold;
}


</style>

</head>
<body bgcolor="#ffffff">
<table cellspacing="0" cellpadding="0" align="center" border="0">
  <tbody>
  <tr>
    <td align="center" colspan="5" height="30"><img height="45" alt="header" src="../mini_logo2.gif" 
      width="336" /><br /><span style="font-size: medium; font-variant: small-caps; font-weight: bold;">vol. 13  no. 3, September, 2008</span><br /><br />
      <div class="button">
      <ul>
        <li><a href="infres133.html">Contents</a> |  </li>
        <li><a href="../iraindex.html">Author index</a> |  </li>
        <li><a href="../irsindex.html">Subject index</a> |  </li>
        <li><a href="../search.html">Search</a> |  </li>
        <li><a href="../index-2.html">Home</a>  </li>
   </ul></div></td></tr>
  <tr>
    <td>&nbsp;</td></tr></tbody></table>
<hr style="COLOR: #5E96FD" size="1" />


<h1>The information environment of researchers in molecular medicine</h1>
<br />
<div style="margin-left: 10%; margin-right: 10%;">
<h4><a href="mailto:annikki.roos@ktl.fi">Annikki Roos</a><br />
Swedish School of Economics and Business Administration, Helsinki, Finland and National Public Health Institute, Helsinki, Finland<br /> 
<a href="mailto:sanna.kumpulainen@uta.fi">Sanna Kumpulainen</a> and <a href="mailto:kalervo.jarvelin@uta.fi">Kalervo J&auml;rvelin</a><br />
Department of Information Studies, University of Tampere, Finland<br />
<a href="mailto:turid.hedlund@hanken.fi">Turid Hedlund</a><br />
Swedish School of Economics and Business Administration, Helsinki, Finland
</h4>
</div>

<br />

<form action="#">
<fieldset>
<legend>Abstract </legend>
<blockquote>
<strong>Introduction. </strong>Describes and analyses the information environment of research work in molecular medicine. We presume an interdependence between the information environment, the research process and the related work tasks.<br />
<strong>Method. </strong>This is a qualitative case study using mixed methods. Empirical data were gathered using two surveys and six semi-structured thematic interviews.<br />
<strong>Analysis. </strong>Analysis was carried out on the merged data of the two surveys. Respondents were divided into groups and information resources were categorized. <br />
<strong>Results. </strong>The information environment consists of broad categories of data, tools, published material and interpersonal communication. The usage of portals and other integrated resources was substantial. The role of PubMed was central in searching scientific facts. Google was used to locate general Web pages, research groups, methods and tools. Interpersonal communication seemed to be effective in providing information about methods and tools among groups which do laboratory work. The problems reported concerned lack of knowledge about useful resources and how to use them properly, some query formulation problems were also reported, e.g. problems with acronyms, personal names and gene name synonymy.<br />
<strong>Conclusions. </strong>The information related tasks occupied much of the researchers' time. If information problems were solved, more time to the research would be released. The database and tool interfaces should be easier to use.
</blockquote>
</fieldset>
</form>

<br />
<div align="center">
<input type="button" value="change font" class="btn" style="font-variant: small-caps; font-weight: bold; font-family: Verdana; color: white; background-color: #5E96FD;" onclick="doChangeFont()" /></div>
<br />
 
<h2>Introduction</h2>

<p>This work is the first part of a broader set of studies where our intention is to model the information practices related to the research process of molecular medicine. The meta-theoretical frame for our research is Ingwersen and J&auml;rvelin's model of interactive information seeking (<a href="#ing05">
Ingwersen and J&auml;rvelin 2005</a>; <a href="#jar04">J&auml;rvelin and Ingwersen 2004</a>). We share the view of the authors of the meta-theory about the importance of context, task context and information systems context in information retrieval and search research. We also realize that the interaction between numerous factors in the context is very complicated and should be better understood. This knowledge is a prerequisite for the development of systems and tools which benefit users, which could be one of the final, practical aspirations of research in information studies. </p>

<p><em>Molecular medicine</em> is seen as consisting of several different areas/domains like genetics, biochemistry, bioinformatics etc. and studies on human diseases on the molecular level. In this paper, we concentrate on the description and analysis of the information environment of molecular medicine. There are two reasons for choosing this approach instead of devoting this paper, for example, to information behaviour or information practices. Firstly, we have found it necessary for increasing our understanding of the domain. Our findings will serve as working knowledge for the forthcoming research which will focus more on tasks and context related to the information practices in this domain. As Ingwersen and J&auml;rvelin (<a href="#ing05">2005</a>) urge information practices and activities should not be studied in isolation but in context. We have chosen molecular medicine as a study object because this scientific field seems to have an exceptionally rich and varied context. We believe that the effects of context to the developed information practices are not well understood in information science or information retrieval. Secondly, it is well known that molecular medicine as a discipline is developing rapidly and that the growth of information as well as tools for handling and analyzing it are developing at increasing speed. Even though we do not know how different factors in the context of the research process interact, we assume that the environment is closely related to the research work and the information practices developed by the researchers, and some characteristics of it might affect the development of the whole domain. Because of this, the information environment in molecular medicine deserves special attention. </p>

<p>The term <em>information environment</em> has often been used without precise description. For example, Srinivasan and Pyati (<a href="#sri07">2007</a>) used the term to express the context in which information research must be situated. We want to pose the difference between information environment and information use environment.  By <em>information use environment</em>, Taylor (<a href="#tay91">1991</a>) means elements related to users and the uses of information, and the framework within which the information is used. In the present study we define information environment as the entirety of information resources, consisting of various information objects, like data in databases, published documents etc., and the tools and services that are needed to retrieve, manage and analyse them. We have chosen to use this term because it is sufficiently broad and holistic in its nature. By using it, the variety of resources and tools as well as their relations can be taken into consideration. </p>

<p>In research work, the research process can be described as a set of <em>work tasks</em> forming the means to achieve the research goals. Information related activities form work tasks, which are the focus of this study. Information activities refer to the implemented actions used to interact with the information environment (<a href="#jar04">J&auml;rvelin and Ingwersen 2004</a>). The work task components and activities related to the handling of information form the information practices of the research field. Information activities are not necessarily easy to perceive or study because of their informal or implicit nature (<a href="#bar95">Barry 1995</a>). Information activities include the following three types of activities: information seeking, handling of produced data, and handling of research output. </p>

<p>In this study we have presumed that there is interdependence between the information environment, the research process and the related work tasks (see <a href="#ing05">Ingwersen and J&auml;rvelin 2005</a>: 313-356;  compare also to <a href="#bar05">Bartlett and Toms 2005</a>). We need a better understanding of the relationships and interaction to be able to develop new information systems for effective access and use of various information resources.</p>

<p>The aim of the present study is to describe and analyse the information environment in the context of research work in molecular medicine. The study object consisted of two research units in molecular medicine and the empirical data were collected from the documentation of the units and the researchers working in them. We focused on three problem areas. Firstly, the characteristics of the research work and resources used in current work tasks, also looking at the researchers' experience in research, background disciplines and the current research groups. Secondly, categorizing information resources by their type and main purpose of use. Thirdly, describing the difficulties and hurdles that the users reported when using information resources.</p>

<p> The research questions related to these problem areas were:</p> 

<ul>
<li>What are the general information related features of the research process in molecular medicine?</li>
<li>Which are the most commonly used information resources in molecular medicine, categorized according to purpose of use and type, e.g., resources for research literature, research data (gene and protein sequences etc.) and local vs. Internet-based tools for analysis?</li>
<li>What problems do researchers in molecular medicine experience and describe when accessing and using the information resources and tools?</li>
<li>How do the researchers integrate various resources in their research process?</li>
</ul>

<p>This study was performed as a qualitative case study in order to get a deeper understanding about the research process and the information resources used. We chose two applicable, high quality research units from Finland as research objects. Even though we were aware that the size of these units is smaller compared to the largest international centres, we knew that they were performing well and had a high impact in the research community. This was verified by measuring the number of citations that some recent articles of the researchers received. This paper presents findings from an online questionnaire and semi-structured interviews, thus providing both a broad overview and deeper insight for the analysis.</p>

<p>The data collected from the surveys from the two units were merged and the results and the analysis cover the whole data set. The follow-up interviews were used for gathering information about the use, integration and problems with the information resources.</p>

<p>The outline of the paper is as follows: firstly, definitions of central concepts and background information on related research and the research units that are the focus of the study are presented. Secondly, the methods are described and the results reported. The paper ends with discussion and conclusions. </p>

<h2>Background</h2>

<h3>Related research</h3>

<p>Related studies can be classified into the following categories. First, the large category of studies on the general usage of information resources (<a href="#cur97">Curtis <em>et al.</em> 1997</a>; <a href="#deg03">De Groote and Dorsch 2003</a>; <a href="#kno07">Knowlton 2007</a>; <a href="#ten04">Tenopir <em>et al.</em> 2004</a>). These studies, mainly based on survey data, concentrate on user behaviour towards the change process from print to electronic for scientific journals or to the impact of new information technologies. Secondly, studies that analyse the high level tasks of molecular biology or classifying of bioinformatics tasks, where the aim is planning better query systems and other tools (e.g., <a href="#mac05">MacMullen and Denn 2005</a>; <a href="#ste01">Stevens <em>et al.</em> 2001</a>; <a href="#tra04">Tran <em>et al.</em> 2004</a>). Thirdly, user studies on bioinformatics. As Geer notices, there are few studies about bioinformatics user groups and their special information needs (<a href="#gee06">Geer 2006</a>). We were not able to find much literature about the information environment of molecular medicine as a whole. There are two papers that describe information-oriented tasks in molecular biology and bioinformatics and consider the research processes of these disciplines from the point of view of information science (<a href="#bar05">Bartlett and Toms 2005</a>; <a href="#mac05">MacMullen and Denn 2005</a>).</p>

<p>According to the study by Grefsheim <em>et al.</em> (<a href="#gre91">1991</a>) from the late 1980s, there were three major sources of information among the biotechnologists, namely their own experiments, personal communication with other scientists, and print or electronic textual material. Personal communication was mentioned as the most important source for timely information in this as well as many other earlier studies (see also <a href="#bar05">Bartlett and Toms 2005</a>). Already in the late 1980s, 60% of the respondents in Grefsheim's study mentioned special databanks as a very useful source of information. Almost ten years later in a study by Yarfirz and Ketchell (<a href="#yar00">2000</a>), the share had risen to 83%. Brown (<a href="#bro03">2003</a>) analysed the use and acceptance of genome and proteomic databases from the perspective of scientific discourse at the beginning of this decade. In her study, 93% of the respondents were accessing such databases at least once a month. She noticed that the databases played an integral role in the daily activities as well as in scientific communication among molecular biologists.</p>

<p>Bartlett and Toms (<a href="#bar05">2005</a>) took another perspective to the use of information in bioinformatics. They combined information behaviour research and task analysis in bioinformatics by taking one specific task, the functional analysis of a gene, as their focus and they succeeded in modelling its protocol. The choice of the task was based on a recommendation made by one bioinformatics expert.</p>

<p>Geer (<a href="#gee06">2006</a>) had the practical goal of establishing the educational and supportive programmes for researchers in mind when she described different user classes according to their apparent skills and information needs. She pointed out that the effective use of the available bioinformatics resources is obstructed by being unaware of the resources and the lack of skills to use them. She divided the users of bioinformatics resources according to their awareness of resources, skills in searching and the critical analysis of data. Besides noticing the difference between academic degrees she also pointed to people who have some special work tasks, namely researchers working mainly in laboratories and those with scientific computing tasks (computational biologists) (<a href="#gee06">Geer 2006</a>).</p>

<p>Bioinformatics resources are an essential part of the scientific discourse and the work processes of researchers in molecular biology and related fields. As Bartlett and Toms (<a href="#bar05">2005</a>) argue, knowing this is not enough. To be able, for example, to develop intelligent systems to facilitate access to the resources, there is a need to have a more detailed picture of a) the <em>high goals</em> of the domain, work processes and tasks aimed at achieving them, b) the needs and skills of different user groups, and c) the information environment as an entity. In this study, our starting point is the information environment. We will also give some attention to the different user groups, proposing that it will give a good background to the forthcoming research.</p>

<h3>The field of molecular medicine</h3>

<p>Molecular medicine, mainly understood as one sub-discipline of biomedicine, is composed of biomedical and molecular biological research. Molecular biology again is a combination of biochemistry, cell biology, genetics and virology. Molecular medicine is an interdisciplinary, practice-oriented and applied science, which utilizes molecular and genetic techniques in the study of biological processes and mechanisms of diseases. Its final, practical task is to provide new and more efficient approaches to the diagnosis, prevention, and treatment of a wide spectrum of congenital and acquired disorders (<a href="#goo99">Goossens 1999</a>; <a href="#mac05">MacMullen and Denn 2005</a>; <a href="#mie06">Miettinen <em>et al.</em> 2006</a>).  More information on the basic concepts can be found at the <a href="http://www.ebi.ac.uk/2can/home.html" target="_blank">2Can Web resource</a> or in the resources of the <a href="http://www.ncbi.nlm.nih.gov/sites/gquery" target="_blank">National Center for Biotechnology Information</a>.</p>

<p>The discipline of molecular medicine is evolving and growing exponentially. Research is often highly competitive and teams worldwide are working on similar projects (<a href="#wel03">Wellcome Trust 2003</a>). Growth has been explained by many simultaneous factors, like the substantial increase in government support, the continued development of the biotechnology industry, and the increasing adoption of molecular-based medicine (<a href="#bue05">Buetow 2005</a>). Together with this development and the overall technological development, especially Web-technology, the amount of data, databases, documents and different analysis and management tools have increased enormously. </p>

<p>The situation of the <a href="http://www.ncbi.nlm.nih.gov/Genbank/index.html" target="_blank">
GenBank</a>, a comprehensive public database of nucleotide sequences and supporting bibliographic and biological annotation, is a good example on the growth rate of data. This database doubles in size about every eighteen months. At the beginning of 2007, it contained over sixty-five billion nucleotide bases from more than sixty-one million individual sequences (<a href="#bue05">Benton 1996</a>).</p>

<p>It is characteristic of most of the research databases that they are cooperatively updated and globally shared. The types of data in the databases are diverse and change constantly. There also exist needs to integrate different kinds of data, e.g., to move between the biological and chemical processes, organelle, cell, organ, organ system, disease specific, individual, family, community and population (<a href="#bue05">Buetow 2005</a>). As Butler notices, some software already exists that allows data from different sources to be combined seamlessly. For example, a researcher can retrieve a gene sequence from the GenBank database, its homologues using the BLAST alignment service, and the resulting protein structures from the Swiss-Model site in one step (<a href="#but06">Butler 2006</a>).</p>

<p>One actively maintained <a href="http://bioinformatics.ca/links_directory/" target="_blank">directory of bioinformatics links</a>  describes well the growing number of different tools needed for data retrieval, analysis and management. It lists almost 1200 Web resources, servers and other useful tools, databases and resources for bioinformatics and molecular biology research in 2007 (<a href="#fox07">Fox <em>et al.</em> 2007</a>).</p>

<p>The most important bibliographic reference database in biomedicine, <a href="http://www.ncbi.nlm.nih.gov/pubmed/" target="_blank">PubMed</a>, indexed about 5,200 journal titles in 2007 and consisted of seventeen million references. The growth rate of the database is about 12,000 references every week, which means over 600,000 new references annually. The majority of the indexed titles are available online and the references include a link to the full text. The number of online journals in science, technology and medicine in general is estimated to be over 93% of all titles (<a href="#war06">Ware 2006</a>).</p>

<p>It could be concluded that some of the most typical features of the information environment in molecular medicine are: a large, and constantly growing, volume of data and published online material; diversity of data types; and a great number of retrieval, analysis and other tools. The environment is interdisciplinary, globally shared (open) and updated, and there is disintegration between databases, published material and tools. This information environment comprises a fertile ground for research work and for the creation of new knowledge and inventions. However, the great variety and the lack of integration is an increasing challenge to the development. </p>

<p>In line with the previous literature, it might be expected that in this study also the number of named resources will be large and the respondents might experience problems in relation to the variety of the resources and tools.</p>

<h2>Study settings, methods and data collection</h2>

<h3>Participating institutions</h3>

<p>For our study, two Finnish research units were selected from high quality international Finnish research organizations in the field of molecular medicine. The researchers from these two Finnish research units were selected as participants.</p>

<p>Research Unit A consisted of ten research groups with ninety-one researchers. Among them there were fifty-seven PhD students, five undergraduate students and twenty-nine group leaders and other senior researchers.  The unit declares that it produces top-level research into the molecular background of cardiovascular, immunological and neuropsychiatric diseases. Some of the research groups were concentrating on the genetic background of common diseases (<em>complex diseases</em>, for example, coronary heart disease), some mainly on molecular genetics of monogenic diseases, like the Meckel Syndrome. There was also one bioinformatics group and one which specialized mainly in systems biology, one in quantitative genetics and two groups focused mainly on the cell and molecular biology of specific diseases.</p>

<p>The second research unit, Unit B, harboured fourteen research groups with 103 researchers, fifty of whom were doctoral students, thirty-nine senior researchers and fourteen group leaders. The research programmes of the groups cover molecular genetics of human diseases, and molecular mechanisms of immune responses and disorders. The genetic programs focus on cytoskeletal and genetic aberrations in the pathogenesis of cancer, and on the role of mitochondrial mutations in human disorders. The topics of the immunology programmes include the basis of the autoimmune disorders APECED and cliac diseases, as well as HIV-1 pathogenicity factor Nef, and cytokine- and antigen-induced transcriptional regulation. There was also a bioinformatics group concentrating on the structural basis of inherited human immunodeficiency syndromes.</p>

<h3>Data and Methods</h3>

<p>In this study mixed methods were used. Empirical data were gathered using two surveys and six semi-structured thematic interviews. The survey data were collected during winter (Unit A) and summer (Unit B) 2007 and the interviews were conducted during summer and autumn 2007. </p>

<p>The surveys were conducted to find out the components of the evolving information environment in molecular medicine. The interviews aimed at understanding more deeply how the researchers interacted in the information environment, i.e., how they used the services available, and why they used them. Observation and thinking aloud, when the interviewees were asked to demonstrate their information seeking procedure using their most recently accomplished seeking incident, were used. The case study was based on the interviews and is an interpretation of how a research case could look in a molecular medicine research unit.</p>

<h4>Surveys</h4>

<p>Two Web questionnaires were distributed to the researchers at the two units. The purpose of the surveys was to get an extensive view of the information environment of the researchers in their current research projects and current work tasks. We intended to get at least a rough overview of the research process and an idea about the relationship between work tasks and different information objects connected to them.</p>

<p>At the beginning of the questionnaire we collected some background information about the researcher, concerning their main discipline, current research subjects, duration of their research work, and about their research group and supervisor (if applicable).</p>

<p>The categories of current work tasks were based on the common knowledge about the general stages of the research process (e.g., <a href="#bla01">Blaxter 2001</a>) and on some information sources where stages of experimental research were described more precisely (e.g., <a href="#den02">Denn and MacMullen 2002</a>, <a href="#hyv02">Hyv&auml; 2002</a>). One senior scientist in Unit A was also consulted. Work tasks were asked about at a general level to find out what kind of tasks they were occupied with and whether they performed any information or data retrieval within their current tasks. For the type of current work tasks, there were nine alternatives. Researchers were also able to add new tasks if none of the given matched.</p>

<p>The information environment was explored through questions on the information resources and preferred information channels used in current research projects. The researchers were asked what services they used to search for literature, as well as which collections of bioinformatics data they were using. There were many open questions and the respondents were advised to describe the resources they were using whilst performing their work tasks. The researchers were also asked to comment on their future publication plans.</p>

<p>The questionnaires directed to Units A and B contained minor differences. At Unit A, the respondents were asked to choose one of the named channels as the most important source of information. At Unit B, the researchers were asked about the problems and hurdles they were facing while seeking information. However, as the research domains and the components of the information environments match, and the researchers have similar education, we believe that the results can be generalized to pertain to both research units.</p>

<p>At Unit A, the group leaders were informed about the survey in one of their normal weekly meetings and after that an email containing the link to the Web-questionnaire was sent to all researchers at the unit. One general reminder was sent via email to all and after that non-respondents were invited to participate via a personal email. 63 researchers responded at Unit A, which gave a response rate of 69.2%. 41 (65%) were students and 22 (35%) were senior researchers, post-doctoral researchers and group leaders. Altogether 66.1% of the juniors and 75.9% of the seniors responded.</p>

<p>At Unit B, a motivation letter was sent by email. The purpose of the study was explained in the letter and the researchers were asked to fill in the Web-questionnaire. There were problems getting the researchers motivated, so three reminder letters were sent. Researchers of the larger research groups, who had not responded, were sent a personal email. There were total of 53 respondents and the response rate was 51.5% at this unit. There were 25 junior researchers (doctoral students) (47%) and 28 senior researchers (53%). In Unit B, the response rates were 50% (juniors) and 52.8% (seniors).</p>

<p>From both units together, there were a total of 116 responses. The combined response rate was 59.8%. There were 66 (56.9%) junior researchers and 50 (43.1%) senior researchers. The corresponding response rates were 58.9 % of the students and 61% of the senior researchers.</p>

<p>The information collected by the surveys does not allow the elicitation of the motives and the ways the researchers use the services in their information environment. Hence, six researchers were interviewed to collect information, which is presented in the analysis of results and in the narrative case descriptions in the results section.</p>

<h4>Interviews</h4>

<p>The interviews were designed to gather information about how the data and information resources were actually used. The case interviews made it possible to describe and analyse the value of the services from the users' point of view. The data also allowed the capturing of events which occurred during the research work and which might influence a researcher's ability to go forward in the process.</p>

<p>There were six complementary case interviews. Firstly, the interviews were conducted to elucidate the use of the tools for information/literature and data retrieval. Secondly, the aim was to obtain a picture of the process or the task which they were actually doing in their research projects when searching, using or analysing information or data. Thirdly, researchers at different phases of their career were included to be able to acquire a comprehensive view of the research process.</p>

<p>The interviews were semi-structured. Each interview began with a question where the respondent was asked to recall a recent critical incident and to demonstrate it. The interviewees were then asked to describe their typical routines accessing bioinformatics databases and searching relevant literature. Six researchers were interviewed, four from Unit A and two from Unit B. The cases were selected to represent different research areas within molecular medicine, different phases of the research projects and researchers in different stages of their research career. Because of this, two doctoral students and their supervisors were interviewed at Unit A. During the interviews, the researchers were also asked to describe and to demonstrate their most recently accomplished data retrieval process and literature retrieval process. The interviews lasted on average forty-five minutes.</p>

<p>The interviewees belonged to three broad sub-disciplinary classes and they were at different stages in their career: bioinformatics (one PhD student), molecular and cellular biology (one PhD student and one group leader) and genetics (two doctoral students and one group leader). </p>

<h3>Data analysis</h3>

<p>The survey data from the two study units were merged before the analysis. The respondents were divided into groups by their background discipline, research career (junior and senior) and current research subject. The information resources and current work tasks were categorized. The aim was to make the actual information environment visible and articulated in these two research units.</p>

<p>The cases were analysed based on emergent themes and the themes were compared to the research questions. The case descriptions were written to explain why and how the services were used to acquire literature and bioinformatics data, and to find typical features of the interaction with the information environment. Similar features of the cases were explored. The detailed cases reveal the sequence of events of interactions within the information environment. Why and how they are using the systems or tools is explained. </p>

<h2>Results from the surveys</h2>

<h3>Tasks</h3>

<p>Current work tasks were asked about at a general level. As the tasks may be overlapping, and the researchers were able to choose among several tasks, the total task percentage was over 100%. We nevertheless obtain a frequency distribution of tasks. The current work tasks of junior and senior researchers are illustrated in Figure 1.</p>

<div align="center"><img src="p353fig1.png" width="747" height="308" alt="Current tasks of senior and junior researchers" /></div>
<div align="center"><br /><strong>Figure 1: Current tasks of senior and junior researchers (n=116)</strong></div>

<p>According to the responses to the questionnaire, the junior researchers were at the moment of the survey mostly writing articles, retrieving and reading literature and working in the laboratory. 50% of the senior researchers were also reading and an even larger part, almost 70%, were occupied with writing articles. The seniors were also, more than the juniors, doing planning work and research administration or other administrative duties. The senior researchers were retrieving bioinformatics data more often than the juniors. When asked about the information activities  on which this study focuses the results showed that literature retrieval was a more common task than the retrieval of bioinformatics data. These information retrieval tasks and the laboratory work task are shown by discipline in Figure 2. An average of 46.6% of all researchers were retrieving information on literature and 27.6% were retrieving bioinformatics data. There were differences between the disciplines: in medicine and biochemistry the data were produced in laboratories, not retrieved from databases to the same extent as in bioinformatics, genetics or biotechnology. In bioinformatics, there were no laboratory work tasks at all. A considerably large share of their tasks consisted of information retrieval tasks, both of literature and of bioinformatics data.</p>

<div align="center"><img src="p353fig2.png" width="660" height="298" alt="Current tasks: search tasks and laboratory work" /></div>
<div align="center"><br /><strong>Figure 2: Current tasks: search tasks and laboratory work by discipline (n=116)</strong></div>

<h3>First source of information</h3>

<p>The researchers from Unit A were asked to choose which channel they prefer as the first source of information in their current project. In all, 68% of the respondents named the Internet or intranet as their first source of information, while 27% preferred contacting colleagues. The more experienced the researchers were, the higher was the proportion preferring the Internet or intranet. Personal contacts were named as the first source of information by 37,5% of graduate students and 23% of senior researchers. </p>

<h3>Literature retrieval and resources</h3>

<p>When the researchers were asked about the literature search services they use, most of them used PubMed, which was mentioned by 99% of the researchers. Google (19%) was used quite often; Web of Science (15%), Google Scholar (14%), and the Finnish University Libraries' search portal Nelli (10%) were also used. The Nelli portal was available at Unit B and to a small portion of the researchers at Unit A.
When asked about which kind of published material the respondents had used, all of them named scientific articles and 39% mentioned that they also use scholarly books. Databases and Internet pages were named by 13.5% of the respondents. Some special material, like laboratory protocols or computer program manuals, was also mentioned.</p>

<p>The respondents were asked to name the journals which they follow on a regularly basis. The most popular titles were the general scientific journals <em>Nature</em> and <em>Science</em>. The number of followed journal titles named per respondent varied between 0 and 15. Interestingly, 10% of the respondents replied that they do not follow any particular journals, but rather follow topics using the PubMed alerting service.</p>

<h3>General Internet resources</h3>

<p>Google is a very popular tool to search general information on the Internet among researchers in molecular medicine. It was named by 90% of the respondents when asked which tools they use when trying to find essential information concerning their current project on the Internet. Wikipedia was named by 47% (54/116) and Web pages in general, like pages of suppliers, or pages of other research groups were named by 10% (12/116). Only 5% (6/116) of respondents named collaborative news sites or forums as their sources of information.</p>

<h3>Data retrieval and resources</h3>

<p>The data were retrieved predominantly from different Web-based databases, which includegene and protein sequence, structure and function information and genome databases. Up to 91% of the respondents declared that they had used some database during their current project. Data were retrieved from databases and processed with various analysis tools either locally in the researchers' own computers, in other local computers or on a local server or with other tools over the Internet. Bioinformatics database portals, like National Center for Biotechnology Information's Entrez and EMBL-EBI, were popular. Many of these portals enable interlaced use of different databases. Below we show the databases, first by the main discipline of the researchers, and, secondly, by broad research subjects of the research groups. Table 1 shows the average popularity of database types in Nucleic Acid Research's Database Categories (see <a href="#gal07">Galperin 2007</a>). The participants are divided by discipline. The Nucleic Acid Research categories are used at a broad level. The numbers in Table 1 corroborate the findings above that the least database search tasks are performed in biochemistry and medicine, and the most of all in bioinformatics; here the smallest number of databases is mentioned in biochemistry and medicine, and a large number of databases were mentioned among the bioinformatics researchers.</p>

<table width="95%" border="1" cellspacing="0" cellpadding="3" align="center" style="border-right: #99f5fb solid; border-top: #99f5fb solid; font-size: smaller; border-left: #99f5fb solid; border-bottom: #99f5fb solid; font-style: normal; font-family: verdana, geneva, arial, helvetica, sans-serif; background-color: #fdffdd">
<caption align="bottom"><br /><strong>Table 1: How many times on average a database in a category is mentioned by a researcher in a certain discipline.</strong></caption>
<tr>
<th rowspan="2">NAR category</th><th colspan="8">Discipline</th></tr>
<tr>	<th>Bio- chemistry (N=15)</th>	<th>Molecular and cellular biology (N=28)</th>	<th>Genetics (N=40)</th>	<th>Bio- technology (N=7)</th>	<th>Bio- informatics (N=9)</th><th>	Medicine (N=11)</th>	<th>Other (N=6)</th>	<th>All (N=116)</th></tr>
<tr><td>Portals</td>	<td align="center">0.80</td>	<td align="center">1.07</td>	<td align="center">1.05</td>	<td align="center">1.00</td>	<td align="center">1.00</td>	<td align="center">0.73</td>	<td align="center">0.17</td>	<td align="center">0.95</td></tr>
<tr><td>Nucleotide sequence databases	</td><td align="center">0.00</td><td align="center">0.25</td><td align="center">0.08</td><td align="center">0.43</td><td align="center">0.67</td><td align="center">0.09</td><td align="center">0.17</td><td align="center">0.18</td></tr>
<tr><td>RNA sequence databases</td><td align="center">0.00</td><td align="center">0.00</td><td align="center">0.08</td><td align="center">0.00</td><td align="center">0.00</td><td align="center">0.00</td><td align="center">0.00</td><td align="center">0.03</td></tr>
<tr><td>Protein sequence databases</td><td align="center">0.13</td><td align="center">0.07</td><td align="center">0.03</td><td align="center">0.14</td><td align="center">1.00</td><td align="center">0.00</td><td align="center">0.17</td><td align="center">0.14</td></tr>
<tr><td>Genomics databases (non-vertebrate)</td><td align="center">0.00</td><td align="center">0.14</td><td align="center">0.00</td><td align="center">0.14</td><td align="center">0.00</td><td align="center">0.18</td><td align="center">0.00</td><td align="center">0.06</td></tr>
<tr><td>Metabolic and signaling pathways</td><td align="center">0.07</td><td align="center">0.11</td><td align="center">0.00</td><td align="center">0.00</td><td align="center">0.00</td><td align="center">0.09</td><td align="center">0.00</td><td align="center">0.04</td></tr>
<tr><td>Human and other vertebrate genomes</td><td align="center">0.53</td><td align="center">0.50</td><td align="center">1.35</td><td align="center">0.29</td><td align="center">1.00</td><td align="center">0.64</td><td align="center">0.00</td><td align="center">0.81</td></tr>
<tr><td>Human genes and diseases</td><td align="center">0.07</td><td align="center">0.11</td><td align="center">0.58</td><td align="center">0.00</td><td align="center">0.11</td><td align="center">0.09</td><td align="center">0.00</td><td align="center">0.25</td></tr>
<tr><td>Microarray data and other gene expression databases</td><td align="center">0.00</td><td align="center">0.00</td><td align="center">0.00</td><td align="center">0.00</td><td align="center">0.00</td><td align="center">0.00</td><td align="center">0.00</td><td align="center">0.03</td></tr>
<tr><td>Proteomics resources</td><td align="center">0.13</td><td align="center">0.00</td><td align="center">0.03</td><td align="center">0.14</td><td align="center">0.00</td><td align="center">0.00</td><td align="center">0.00</td><td align="center">0.03</td></tr>
<tr><td>Other</td><td align="center">0.07</td><td align="center">0.11</td><td align="center">0.03</td><td align="center">0.00</td><td align="center">0.22</td><td align="center">0.00</td><td align="center">0.17</td><td align="center">0.07</td></tr>
<tr><td>Sum</td><td align="center">1.80</td><td align="center">2.50</td><td align="center">3.20</td><td align="center">2.14</td><td align="center">4.11</td><td align="center">1.82</td><td align="center">0.67</td><td align="center">2.59</td></tr>
<tr><td colspan="9">Note: The value 1 indicates that on average every researcher in each discipline group mentioned one database of that type. Categories are based on NAR categories (<a href="#gal07">
Galperin 2007</a>), the category <em>Portals</em> is additional.
</td></tr>
</table>
 								
<p>The most frequently mentioned databases fell in to the additional category <em>Portals</em>. These include,for example, the <a href="http://www.ncbi.nlm.nih.gov/sites/gquery" target="_blank">Entrez database portal</a> and the <a href="http://www.embl.de/" target="_blank">EMBL portal</a>. These services include several integrated or separate databases that fall into different Nucleic Acid Research categories.</p>

<p>In Table 2, the respondents are divided differently according to the broad research subjects of their research groups. Some of the headings are similar to Table 1, but the headings stand for the generalized research subjects and thus the distribution is different.</p>

<table width="95%" border="1" cellspacing="0" cellpadding="3" align="center" style="border-right: #99f5fb solid; border-top: #99f5fb solid; font-size: smaller; border-left: #99f5fb solid; border-bottom: #99f5fb solid; font-style: normal; font-family: verdana, geneva, arial, helvetica, sans-serif; background-color: #fdffdd">
<caption align="bottom"><br /><strong>Table 2: How many times on average a database in a category is mentioned per researcher in each research group type.</strong></caption>

<tr><th rowspan="2">NAR category</th><th colspan="5">Research subject of research groups</th></tr>
<tr><th>Molecular and cellular biology (N=36)</th><th>Genetics (N=66)</th><th>Bio- informatics (N=8)</th><th>Systems biology (N=6)</th><th>All (N=116)</th></tr>

<tr><td>Portals</td><td align="center">0.92</td><td align="center">0.88</td><td align="center">1.25</td><td align="center">1.50</td><td align="center">0.95</td></tr>
<tr><td>Nucleotide sequence databases</td><td align="center">0.22</td><td align="center">0.14</td><td align="center">0.50</td><td align="center">0.00</td><td align="center">0.18</td></tr>
<tr><td>RNA sequence databases</td><td align="center">0.00</td><td align="center">0.05</td><td align="center">0.00</td><td align="center">0.00</td><td align="center">0.03</td></tr>
<tr><td>Protein sequence databases</td><td align="center">0.11</td><td align="center">0.05</td><td align="center">1.13</td><td align="center">0.00</td><td align="center">0.14</td></tr>
<tr><td>Genomics databases (non-vertebrate)</td><td align="center">0.06</td> <td align="center">0.06</td><td align="center">0.00</td><td align="center">0.17</td><td align="center">0.06</td></tr>
<tr><td>Metabolic and signaling pathways</td><td align="center">0.08</td><td align="center">0.03</td><td align="center">0.00</td><td align="center">0.00</td><td align="center">0.04</td></tr>
<tr><td>Human and other vertebrate genomes</td><td align="center">0.31</td><td align="center">1.08</td><td align="center">1.00</td><td align="center">0.67</td><td align="center">0.81</td></tr>
<tr><td>Human genes and diseases</td><td align="center">0.06</td><td align="center">0.39</td><td align="center">0.13</td><td align="center">0.00</td><td align="center">0.25</td></tr>
<tr><td>Microarray data and other gene expression databases</td><td align="center">0.00</td><td align="center">0.03</td><td align="center">0.00</td><td align="center">0.33</td><td align="center">0.03</td></tr>
<tr><td>Proteomics resources</td><td align="center">0.11</td><td align="center">0.00</td><td align="center">0.00</td><td align="center">0.00</td><td align="center">0.03</td></tr>
<tr><td>Other</td> <td align="center">0.03</td><td align="center">0.06</td><td align="center">0.25</td><td align="center">0.17</td><td align="center">0.07</td></tr>
<tr><td>Sum</td><td align="center">1.89</td><td align="center">2.76</td><td align="center">4.25</td><td align="center">2.83</td><td align="center">2.59</td></tr>
<tr><td colspan="6"><strong>Note</strong>: The value 1 indicates that on average every researcher in each group mentioned one database of that type. Categories are based on NAR categories (<a href="#gal07">
Galperin 2007</a>), the category <em>Portals</em> is additional.)</td></tr>
</table>

<p>The research groups with research subjects related to molecular and cellular biology seem to use a less varied assortment of databases than on average. The bioinformatics research groups used more varied databases than the other groups, and specifically protein databases. The genetics research groups used more databases from the category Human genes and diseases than the other groups. When both tables are considered, the most commonly mentioned databases fell into the Portals category, and into the Human and other vertebrate genomes category. </p>

<h3>Tools</h3>

<p>Many research databases integrate or include tools for analysis. The category of tools is very broad and could be grouped in several ways (for example: data analysis tools, laboratory data analysis tools, literature analysis tools, programming tools, statistical tools, and taxonomies and ontologies). However, we will only mention here the most important ones regardless of their type and leave out the grouping possibility. </p>

<p>In total, ninety-seven different tools and software were named, most of which were open or free software, and twenty-five of them were named more than once. <a href="http://primer3.sourceforge.net/" target="_blank">Primer3</a> (mentioned thirty-eight times) and <a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" target="_blank">BLAST</a> (mentioned nineteen times) were the most popular tools. Primer3 is a tool for designing polymerases chain reactions primers which are used in amplifying specific regions in a DNA strand. BLAST is a tool for sequence alignment, searching similarities between sequences. Among the twenty-five most often named tools, there were at least three that were made at the research units in question and two were statistical tools (R and SPSS).</p>

<h3>Problems experienced</h3>

<p>The researchers at Unit B were asked about problems and hurdles they experienced while seeking information (see the subsection on surveys under Data and methods). The majority of the researchers (71%) answered that they had experienced some difficulties.</p> 

<p>The perceived problems while seeking information or data in general were classified in five categories: (a) information overflow, (b) lack of information about suitable sources, (c) difficulties in use (lack of skills/difficult interfaces/poor documentation), (d) technical problems (slow connections etc.), and (e) limited access/no access to desired sources. The categories were formed according to the answers: if the amount of information was emphasized, the category (a) was chosen and if difficulties, for example in search key selection or query phrase formulation, were mentioned, category (c) was chosen. </p>

<p>Most of the problems that were reported fell into category (c), difficulties in use. These kinds of problems were experienced by 43% of the respondents who reported problems. Some of the interfaces were too difficult to use and some of the data structures were not suitable for the required use. The subjects' searching skills were not satisfactory and the results of literature searches were inexact. Limited access or no access to desired sources, (e), was problematic to 22% due to lack of institutional subscriptions to some scientific sources. Lack of information on suitable sources, (b), was reported by 14% of the respondents. Information overflow, (a), was the issue in 8% of the reported problems.</p>

<h2>Results from the interviews</h2>

<p>The case interviews were recorded for the analysis. The cases were analysed based on the emergent themes and on the themes reflected in the interview and the research questions. The case descriptions were written after the interviews to explain why and how the services to acquire literature and data were used. The aim was to find typical features describing the interaction of the researchers (as users) with the information environment. The case descriptions were therefore analysed for similar features. The narrative case description below reveals the sequence of events of interactions within the information environment. Why and how the interviewed researchers are using the systems or tools is also explained.</p>

<h3>The work tasks</h3>

<p>The basic research problem in molecular and cellular biology and genetics is to find genes which cause a particular disease, and to understand their functional characteristics. Laboratory work is an essential part of the research work. Bioinformatics research is, on the other hand, pure "in silico" work, including database maintenance and development as well as designing Web services and tools to analyse the data.
Two of the PhD students were, at the time of the study, working in the laboratory, the third one was searching information and the fourth writing an article. The seniors were mainly concentrating on research administration, planning and other administrative duties. It became evident that all of them were doing several tasks simultaneously.</p>

<h3>Literature</h3>

<p>Literature is used at different stages of the research work and it is accessed in several ways. In the beginning of a project it is typical to search by focal terms or even by the name of a known specialist through PubMed. </p>

<p>When the core literature is available, current awareness about the topic is arranged by setting up an automated update service in PubMed or using PubCrawler for this purpose. Seniors follow citations to their own, most important articles in Web of Science, a service provided by Thomson Scientific. Table of contents or alerting services for the most important journal titles were also used for current awareness. 
When the project proceeds, new interests and questions appear and thus a new process for information searching and retrieval starts. Researchers get tips from colleagues or supervisors and might start a chaining process from a new reference. Literature seeking may also start from a homepage of a research group or in some cases from a homepage of a journal. Using links from bioinformatics databases to references of literature services was typical in these cases. </p>

<h3>Databases</h3>

<p>According to the interviews, the number of relevant databases used was rather limited. Protein Data Bank (PDB) was used by the bioinformatics researcher because it was recognized as an authority in that field. The database was used to find structural information about proteins. Even though the needed data were downloaded to the local computer, PDB's Web service was used continuously to check out some details. Usually, the code of the structure was used as a search key to be able to directly get to the structure information. Links to Medline and literature are provided in the database and this feature was frequently used. </p>

<p>Genetics researchers used NCBI's databases frequently. From the Entrez portal, the databases Gene, OMIM, SNP and Protein were named as the most popular. When the gene is not known, SNP array data was used to find homozygote areas from DNA strands. Interesting homozygote areas were found through an analysis tool. These areas were then searched from general gene databases (e.g., Gene) to find out which genes are situated in these areas. There were several genome browsers, and dissimilar opinions existed about the reasons to choose one. Some of the interviewees said that it is mainly the force of habit; some others said that every browser has special functionality which means that they are chosen according to the specific purpose of use. The research group's preferences could affect the selection; if others in the group used some particular service, it was more likely other researchers would choose the same service. Usability (ease of use) and simplicity were also named as the reasons to choose between almost similar services. Genes were searched by the code or abbreviation and links were used to access published articles and other databases for deeper information. Google was used to locate different databases on the Internet.</p>

<h3>Tools, methods and protocols</h3>

<p>It appeared that colleagues are an important source of information when tools, methods and laboratory protocols are chosen. This was especially apparent in the research areas where laboratory work is done. In the bioinformatics case, Google and Web-pages functioned as information sources.</p>

<h3>Problems experienced</h3>

<p>Problems concerning information retrieval emerged from the case interviews. These corroborate the findings from the questionnaire. The researchers were using the databases to find evidence from previous research to support their laboratory findings and to interpret new information from the available data. It is hard work to plough through the vast amount of information to find out if the subject has already been investigated and what is known about it (category a). It seemed, however, that there might be differences in this respect between the research areas. The more complex the disease, the higher the amount of published material. There was also a lack of knowledge about the useful sources (category b), or how to use some resources (category c). The documentation of some available databases was found poor or lacking (category c). It was impossible to reuse data, if there existed no information about the format of the data. The limited access to sources, especially to published material was present in the case interviews (category e). The cases showed some query formulation problems (category a). The researchers were having problems with acronyms and short names or with popular names (e.g. Chinese personal names). Gene name synonymy was also a problem: e.g., literature had to be searched with several gene names appearing in PubMed.</p>

<h3>The case story</h3>

<p>The case story is an interpretation of the data collected for this study. The case focuses on the information resources used and the problems encountered in the retrieval and usage of information resources, it is not a complete report on the research process. The complete process is much more complicated and not within the focus of this study.</p>

<p>We present a case of a fictional doctoral student in the midstream of her dissertation project. Her research subject is the genetic background of a common, complex disease. There exists an animal model of the disease and a part of the project is to test it. She is working as a member of a research group which has collaborators outside the primary unit both nationally and internationally. This group is led by a distinguished senior, who has been working in the field for about ten to fifteen years.</p>

<blockquote><p>A long process of hypothesis creation preceded the start of this new project. The supervisor of the interviewee had, among other things, had discussions about this new, interesting subject with some colleagues and made searches from PubMed using some focal terms as search keys. As a result the supervisor got a list of core articles about the subject which she gave to the interviewee. She started reading at the same time as some routine laboratory work was done.</p>

<p>The selection of the research method and laboratory protocol has, according to the doctoral student, been one of the most essential decisions during the research process. At their unit, many have the habit of making very practical decisions: the method will be selected from those that are already well known or utilized in some other group. The choice is made after discussions with colleagues. She knows that there are some researchers who use, for example, Google or some known research fellow's home pages for this or other general search purposes. </p>

<p>When she worked with the laboratory data she used an analysis tool which was created at the unit. She used one database, dbSNP from the NCBI's Entrez portal to find the positions of genetic markers and the surrounding sequence. Haploview, open access software available on the Internet, was used to view the dependencies between genetic markers. </p>

<p>After screening some interesting candidate genes, broader information about them was searched for from PubMed. If the genes were totally unknown to her, she started from OMIM (NCBI), which consists of the core literature of all human genes. She can also search by the acronym of the gene from Entrez Gene (NCBI), which contains links to many tools and related databases where she can proceed deeper. Sometimes she also uses UCSC Genome Browser, which she knows to be quite popular in many groups because of its simplicity. At the time of the interview she was using Ensembl because she needed clear pictures for her next article and, in her opinion, Ensembl is best in this respect.
Her working habit is, like many others at the unit, to use many resources simultaneously and repeatedly even during the writing process. </p>

<p>She keeps up to date during the whole process by using the current awareness service of PubMed where she has a profile which consists of the name of the disease and the codes or acronyms of the possible candidate genes. Every week she gets an email from PubMed, via PubCrawler, which contains the most recent references. PubCrawler is a free update alerting service for PubMed and GenBank. The email contains links to PubMed, and from PubMed it is easy to continue to the full text, presuming that the unit has a license to the journal in question. Recently, she has also started to order table of contents from some journals in which they are planning to publish their research results.</p>

<p>Problems are mostly related to the amount of, and access to, information and lack of knowledge about how to use different tools and databases. She named filtering essential information from the unessential as one of the main problems.</p></blockquote>

<h2>Discussion</h2>

<p>The information environment in molecular medicine appeared to be as varied as was expected. It can be categorized broadly into data, tools, literature and interpersonal communication. Every category has at least some sub-groups. In this study, laboratory work was important to all but the groups in bioinformatics. There are some extensive portals, like NCBI's Entrez which integrate different databases and enable simultaneous searching from different resources. In addition, there are more resources that are connected via links to these databases.</p> 

<p>The usage of portals and other integrated resources was substantial. The role of PubMed is central in searching scientific literature and facts, whereas Google is a search engine used to locate general Web pages, research groups, methods and tools. Interpersonal communication seemed to be an effective way to provide information about methods and tools among those groups which do laboratory work. One reason for the importance of interpersonal communication among these groups might be insufficient documentation of laboratory-related work processes.</p>

<p>It is noteworthy that even though scientific articles are a very important source of information to all of the respondents, the importance of journals as the basic object of interest is not equally self-evident. We are not able to explain in this study the causes of this result. A deeper process of change might be happening, this might be just a feature in certain sub-disciplines or it might depend on the career stage of researchers.</p>  

<p>Most of the resources in molecular medicine are globally shared and updated, and changing constantly. Compared with other disciplines, database and tool development has been based more often on the common effort and open access/source, in contrast to commercial actors in other disciplines. Researchers are information users and, at the same time, also creators of data and information, and developers of tools. Even though the researchers use a considerable number of different databases, there is not enough time to learn the full potential of these services. </p>

<p>Emblematic features of the information and data seeking was the interlaced use of the resources available. Data and information are sought seamlessly through different databases and there appeared not to be a lucid border between information services. Some of the resources, e.g., Entrez databases, are well integrated and therefore the data found in bioinformatics databases was used to find literature within the same interface without the researcher ever constructing an explicit query and vice versa. It seems that in this respect database creators have succeeded well in understanding how the research process proceeds. Even though we were not able in this study to find out how research work actually proceeds in molecular medicine and at which point a certain type of information is typically needed, we assume that the workflow may involve iterative steps of querying, analysis and optimization (compare with <a href="#cas05">
Castro <em>et al.</em> 2005</a>).</p>

<p>Research in information science has seldom concentrated on other than more or less traditional textual information objects (see e.g., <a href="#ing05">Ingwersen and J&auml;rvelin 2005</a> or compare with <a href="#cha07">Chaim 2007</a>).</p>

<p>Our study suggests that the integration between documents and data is becoming more and more important in molecular medicine research work, and it might be profitable to information science in general to focus more on this aspect and to examine the whole information environment comprehensively. </p>

<h3>Limitations</h3>

<p>There are some limitations in the present study. As in most Web surveys, there was some bias in terms of who responded; some of the research groups are under-represented. The research groups interested in the development of novel information systems were more likely to answer the questionnaire. Response was voluntary, so presumably the most active researchers did respond. The survey was, however, complemented by the case interviews.</p>

<p>The interviews and the surveys convey ex-post-facto accounts of performance and, therefore, may suffer from omissions, gaps and rationalizations that are not entirely faithful to the actual process. The interviews were valuable in augmenting the survey data. More generally, a study on task-based information access in molecular medicine should start with the information requirements triggered by the actual work situations.</p>

<h3>Future research</h3>

<p>Information access and use in molecular medicine needs more investigation. But as Barry (<a href="#bar95">1995</a>) noticed, at least some parts of the information practices and activities might be implicit and not realized by the respondents. It is possible that there are problems that were not expressed in this stage of the present study. If the problems are not expressed by the researchers, they should be observed in their work when accessing information.</p>

<p>Cognitive work analysis has been found to be an effective way to identify potentially desirable system features and useful bioinformatics tools, and to provide better understanding of some of the unmet needs of the researchers (<a href="#bar05">Bartlett and Toms 2005</a>). The starting point of the research into information practices in molecular medicine should be trying to understand the research processes of these domains, and the work tasks performed to achieve research goals, in more detail. It is presumable that the progress and other changes in the information environment affect research processes and this relationship also needs to be understood (cf. <a href="#ing05">Ingwersen and J&auml;rvelin 2005</a>: 78). The extensive collaboration may also have implications for the information environment. One approach to studying the possibilities for enhancing access to information in this field is to observe the researchers while they are conducting their daily work tasks. If the information activities and the work process can be described, it contributes to the design of novel systems.</p>

<h2>Conclusions</h2>

<p>This paper addressed the information environment in molecular medicine. The information environment consists of several types of databases, information retrieval systems and analysis tools. These are used seamlessly so that data located (or constructed) in one are used to access the other. The most used resources are firstly, for data retrieval, different portals and genome databases. Bioinformatics researchers use more databases than others, both when the distribution is dissected by discipline or research subjects. For acquiring literature, the bibliographic information database PubMed is the most used. The service provides links to the full text documents. In molecular medicine researchers use several data analysis tools , either for the data produced by themselves in laboratories or for the data retrieved from the open databases over the Internet. A typical feature of the domain is the vast number of open source databases, which offer research data for reuse.</p>

<p>The researchers in molecular medicine described lots of problems they experience while they access and use the information resources and tools. Some of the information systems were too difficult to use and some of the data structures were not suitable for any use. The researchers did not have time to learn how to use the services properly. Even when the open source databases and journals are quite typical to the domain, the limited access to the information sources is a big problem. Up-to-date research literature is extremely important to the researchers.</p>

<p>Information related tasks took much of the researchers' time. If the information problems could be relieved, it would release more time for other research work. The database and tool interfaces should be easier to use. There is consequently a need for more research in order to be able to build systems that enhance access to information.</p>

<h2>Acknowledgements</h2>

<p>The authors gratefully acknowledge partial funding from the Academy of Finland grants (#120996, #1124131, #204978 and #205993) and the helpful comments of the anonymous referees.</p>

<form action="#">
<fieldset>
<legend style="color: white; background-color: #5E96FD; font-size: medium; padding: .1ex .5ex; border-right: 1px solid navy; border-bottom: 1px solid navy; font-weight: bold;">References</legend>
<ul>
<li><a name="bar95" id="bar95"></a>Barry, C.A. (1995). Critical issues in evaluating the impact of IT on information activity in academic research: developing a qualitative research solution. <em>Library &amp; Information Science Research</em>, <strong>17</strong>(2), 107-134.</li>

<li><a name="bar05" id="bar05"></a>Bartlett, J.C. &amp; Toms, E.G. (2005). Developing a protocol for bioinformatics analysis: An integrated information behavior and task analysis approach. <em>Journal of the American Society for Information Science and Technology</em>, <strong>56</strong>(5), 469-482.</li>

<li><a name="ben96" id="ben96"></a>Benton, D. (1996). Bioinformatics&mdash;principles and potential of a new multidisciplinary tool. <em>Trends in Biotechnology</em>, <strong>14</strong>(8), 261-272.</li>

<li><a name="bla01" id="bla01"></a>Blaxter, L., Hughes, C. &amp; Tight, M. (2001). <em>How to research</em>. Buckingham, UK: Open University Press.</li>

<li><a name="bro03" id="bro03"></a>Brown, C. (2003). The changing face of scientific discourse: analysis of genomic and proteomic database usage and acceptance. <em>Journal of the American Society for Information Science</em>, <strong>54</strong>(10), 926-938.</li>

<li><a name="bue05" id="bue05"></a>Buetow, K.H. (2005). Cyberinfrastructure: empowering a "third way" in biomedical research. Science, 308(5723), 821-824.</li>

<li><a name="but06" id="but06"></a>Butler, D. (2006). Mashups mix data into global service. <em>Nature</em>, <strong>439</strong>(7072), 6-7.</li>

<li><a name="cha07" id="cha07"></a>Chaim, Z. (2007). Knowledge map of information science. <em>Journal of the American Society for Information Science and Technology</em>, <strong>58</strong>(4), 526-535.</li>

<li><a name="cas05" id="cas05"></a>Castro, A.G., Thoraval, S., Garcia, L.J. &amp; Ragan, M.A. (2005). <a href="http://www.webcitation.org/5b4ni0EIW">Workflows in bioinformatics: meta-analysis and prototype implementation of a workflow generator</a>. <em>BMC Bioinformatics</em>, <strong>6</strong>. Retrieved 23 September, 2008 from http://www.biomedcentral.com/1471-2105/6/87  (Archived by WebCite&reg; at http://www.webcitation.org/5b4ni0EIW)</li>

<li><a name="cur97" id="cur97"></a>Curtis, K.L., Weller, A.C. &amp; Hurd, J.M. (1997). <a href="http://www.webcitation.org/5b4nrC4FJ">
Information-seeking behavior of health sciences faculty: the impact of new information technologies.
</a> <em>Bulletin of Medical Library Association</em>, <strong>85</strong>(4), 402-410. Retrieved 23 September, 2008 from http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=226298&amp;pageindex=1 (Archived by WebCite&reg; at http://www.webcitation.org/5b4nrC4FJ)</li>

<li><a name="deg03" id="deg03"></a>De Groote, S.L. &amp; Dorsch, J.L. (2003). <a href="http://www.webcitation.org/5b4nz9gTb">
Measuring use patterns of online journals and databases.</a> <em>Journal of the Medical Library Association</em>, <strong>91</strong>(2), 231-240. Retrieved 23 September, 2008 from http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=12883574 (Archived by WebCite&reg; at http://www.webcitation.org/5b4nz9gTb)</li>

<li><a name="den02" id="den02"></a>Denn, S.O. &amp; MacMullen, W.J. (2002). The ambiguous bioinformatics domain: a conceptual map of information science applications for molecular biology. <em>Proceedings of the American Society for Information Science and Technology</em>, <strong>39</strong>(1), 556-558. </li>

<li><a name="fox07" id="fox07"></a>Fox, J.A., McMillan, S. &amp; Ouellette, B.F. (2007). <a href="http://www.webcitation.org/5b4oLk88Y">
Conducting research on the Web: 2007 update for the bioinformatics links directory</a>. <em>Nucleic Acids Research</em>, <strong>35</strong>, W3-5. Retrieved 23 September, 2008 from http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=17586821 (Archived by WebCite&reg; at http://www.webcitation.org/5b4oLk88Y)</li>

<li><a name="gal07" id="gal07"></a>Galperin, M. Y. (2007). <a href="http://www.webcitation.org/5b4oUhjQV">The molecular biology database collection: 2007 update.</a> <em>Nucleic Acids Research</em>, <strong>35</strong>, D3. Retrieved 23 September, 2008 from http://nar.oxfordjournals.org/cgi/content/full/35/suppl_1/D3  (Archived by WebCite&reg; at http://www.webcitation.org/5b4oUhjQV)</li>

<li><a name="gee06" id="gee06"></a>Geer, R. C. (2006). <a href="http://www.webcitation.org/5b4ogoqbo">Broad issues to consider for library involvement in bioinformatics</a>. <em>Journal of the Medical Library Association</em>, <strong>94</strong>(3), 286-298. Retrieved 23 September, 2008  from http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1525323  (Archived by WebCite&reg; at http://www.webcitation.org/5b4ogoqbo)</li>

<li><a name="goo99" id="goo99"></a>Goossens, M. (1999). Principles of molecular medicine. <em>New England Journal of Medicine</em>, <strong>340</strong>(20), 1601-1602.</li>

<li><a name="gre91" id="gre91"></a>Grefsheim, S., Franklin, J. &amp; Cunningham, D. (1991). <a href="http://www.webcitation.org/5b4oq70wT">Biotechnology awareness study, part 1: where scientists get their information</a>. <em>Bulletin of the Medical Library Association</em>, <strong>79</strong>(1), 36-44. Retrieved 23 September, 2008 from http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=225482  (Archived by WebCite&reg; at http://www.webcitation.org/5b4oq70wT)</li>

<li><a name="hyv04" id="hyv04"></a>Hyv&auml; tutkimustapa Kansanterveyslaitoksessa (2004). Kansanterveyslaitoksen julkaisuja B 2. Helsinki: Kansanterveylaitos. Retrieved February 2008 from http://www.ktl.fi/portal/suomi/julkaisut/julkaisusarjat/kansanterveyslaitoksen_julkaisuja_b/hyva_tutkimustapa_ktl_ssa </li>

<li><a name="ing05" id="ing05"></a>Ingwersen, P. &amp; J&auml;rvelin, K. (2005). <em>The turn: integration of information seeking and retrieval in context</em>. Dordrecht: Springer.</li>

<li><a name="jar04" id="jar04"></a>J&auml;rvelin, K. &amp; Ingwersen, P. (2004). <a href="http://www.webcitation.org/5b4p6LfaH">
Information seeking research needs extension towards tasks and technology</a>. <em>Information Research</em>, <strong>10</strong>(1), paper 212. Retrieved 23 September, 2008 from http://informationr.net/ir/10-1/paper212.html . (Archived by WebCite&reg; at http://www.webcitation.org/5b4p6LfaH)</li>

<li><a name="kno07" id="kno07"></a>Knowlton, S.A. (2007). <a href="http://www.webcitation.org/5b4pFVDxv">Continuing use of print-only information by researchers.</a> <em>Journal of the Medical Library Association</em>, <strong>95</strong>(1), 83-88. Retrieved 23 September, 2008 from http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=17252072 (Archived by WebCite&reg; at http://www.webcitation.org/5b4pFVDxv)</li>

<li><a name="mac05" id="mac05"></a>MacMullen, W. J. &amp; Denn, S. O. (2005). Information problems in molecular biology and bioinformatics. <em>Journal of the American Society for Information Science and Technology</em>, <strong>56</strong>(5), 447-456. </li>

<li><a name="mar06" id="mar06"></a>Marc Ware Consulting. (2006). <em><a href="http://www.webcitation.org/5b4pduVQ5">Scientific publishing in transition: an overview of current developments.</a></em> Bristol: Marc Ware Consulting Ltd. Retrieved 23 September, 2008 from http://www.stm-assoc.org/storage/Scientific_Publishing_in_Transition_White_Paper.pdf (Archived by WebCite&reg; at http://www.webcitation.org/5b4pduVQ5)</li>

<li><a name="mie06" id="mie06"></a>Miettinen, R., Tuunainen, J., Knuuttila, T. &amp; Mattila, E. (2006). Tieteest&auml; tuotteeksi? Yliopistotutkimus muutosten ristipaineessa. Helsinki: Yliopistopaino</li>

<li><a name="sri07" id="sri07"></a>Srinivasan, R. &amp; Pyati, A. (2007). Diasporic information environments: reframing immigrant-focused information research. <em>Journal of the American Society for Information Science and Technology</em>, <strong>58</strong>(12), 1734-1744.</li>

<li><a name="ste01" id="ste01"></a>Stevens, R., Goble, C., Baker, P. &amp; Brass, A. (2001). A classification of tasks in bioinformatics. <em>Bioinformatics</em>, <strong>17</strong>(2), 180-188.</li>

<li><a name="tay91" id="tay91"></a>Taylor, R.S. (1991). Information use environments. <em>Progress in communication sciences</em>, <strong>10</strong>, 217-255.</li>

<li><a name="ten04" id="ten04"></a>Tenopir, C., King, D. W. &amp; Bush, A. (2004). <a href="http://www.webcitation.org/5b4pxNsz7">
Medical faculty's use of print and electronic journals: changes over time and in comparison with scientists
</a>. <em>Journal of the Medical Library Association</em>, <strong>92</strong>(2), 233-241. Retrieved October 2007 from http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=15098053 (Archived by WebCite&reg; at http://www.webcitation.org/5b4pxNsz7)</li>

<li><a name="tra04" id="tra04"></a>Tran, D., Dubay, C., Gorman, P. &amp; Hersh, W. (2004). <a href="http://www.webcitation.org/5b4q8sjHg">Applying task analysis to describe and facilitate bioinformatics tasks.</a> In Marius Fieschi, Enrico Coiera, Yu-Chan Jack Li (Eds.). <em>Medinfo 2004; Proceedings of the 11th World Congress on Medical Informatics</em>, (pp. 818-822). Retrieved October 2007 from http://medir.ohsu.edu/~hersh/medinfo-04-bioinformatics.pdf . (Archived by WebCite&reg; at http://www.webcitation.org/5b4q8sjHg)</li>

<li><a name="wel03" id="wel03"></a>Wellcome Trust. (2003). <em><a href="http://www.webcitation.org/5b4qdt0bl">
Economic analysis of scientific research publishing</a></em>. Retrieved 23 September, 2008 from http://www.wellcome.ac.uk/stellent/groups/corporatesite/@policy_communications/documents/web_document/wtd003182.pdf  (Archived by WebCite&reg; at http://www.webcitation.org/5b4qdt0bl)</li>

<li><a name="yar00" id="yar00"></a>Yarfitz, S. &amp; Ketchell, D. S. (2000). <a href="http://www.webcitation.org/5b4qm4kHr">
A library-based bioinformatics services program.</a> <em>Bulletin of the Medical Library Association</em>, <strong>88</strong>(1), 36-48. Retrieved October 2007 from http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pmcentrez&amp;artid=35196 . (Archived by WebCite&reg; at http://www.webcitation.org/5b4qm4kHr)</li>
</ul>
</fieldset>
</form>


<form action="#">
<fieldset>
<legend style="color: white; background-color: #5E96FD; font-size: medium; padding: .1ex .5ex; border-right: 1px solid navy; border-bottom: 1px solid navy; font-weight: bold;">How to cite this paper</legend>
<div><br />
Roos, A., Kumpulainen, S., J&auml;rvelin, K and Hedlund, T. (2008).  &quot;The information environment of researchers in molecular medicine&quot; <em>Information Research</em>, <strong>13</strong>(3) paper 353. [Available at http://InformationR.net/ir/13-3/paper353.html]</div>
</fieldset>
</form>


<table cellspacing="10" align="center">
	<tr>
		<td colspan="3" align="center" style="background-color: #5E96FD; color: white; font-family: verdana; font-size: small; font-weight: bold;">Find other papers on this subject</td></tr>
	<tr>
		<td align="center" valign="top">

<form method="get" action="http://scholar.google.com/scholar" target="_blank">
			<table bgcolor="#ffffff">
				<tr>
					<td nowrap="nowrap" valign="top" align="center" height="32"><input type="hidden" name="q" size="31" maxlength="255" value="&quot;molecular medicine&quot; (&quot;information seeking&quot; OR &quot;information searching&quot; OR &quot;information use&quot;)"></input> <br />
<input type="submit" name="sa" value="Scholar Search"  style="font-size: small; font-family: Verdana; font-weight: bold;"></input>
<input type="hidden" name="num" value="100"></input>
					</td>
				</tr>
			</table>

</form>
		</td>
		<td align="center" valign="top">
<!-- Search Google -->

<form method="get" action="http://www.google.com/custom" target="_top">
			<table bgcolor="#ffffff">
				<tr>
					<td nowrap="nowrap" valign="top" align="center" height="32"><input type="hidden" name="q" size="31" maxlength="255" value="&quot;molecular medicine&quot; (&quot;information seeking&quot; OR &quot;information searching&quot; OR &quot;information use&quot;)"></input><br />
<input type="submit" name="sa" value="Google Search" style="font-family: Verdana; font-weight: bold; font-size: small;"></input>
<input type="hidden" name="client" value="pub-5081678983212084"></input>
<input type="hidden" name="forid" value="1"></input>

<input type="hidden" name="ie" value="ISO-8859-1"></input>
<input type="hidden" name="oe" value="ISO-8859-1"></input>
<input type="hidden" name="cof" value="GALT:#0066CC;GL:1;DIV:#999999;VLC:336633;AH:center;BGC:FFFFFF;LBGC:FF9900;ALC:0066CC;LC:0066CC;T:000000;GFNT:666666;GIMP:666666;FORID:1;"></input>
<input type="hidden" name="hl" value="en"></input>
					</td>
				</tr>
			</table>
</form>
</td>
<td align="center" valign="top">
<form method="get" action="http://www.live.com/" target="_blank">
			<table bgcolor="#ffffff">
				<tr>

					<td nowrap="nowrap" valign="top" align="center" height="32"><input type="hidden" name="q" size="31" maxlength="255" value="&quot;molecular medicine&quot; (&quot;information seeking&quot; OR &quot;information searching&quot; OR &quot;information use&quot;)"></input> <br />
<input type="submit" name="sa" value="Windows Live"  style="font-size: small; font-family: Verdana; font-weight: bold;"></input>
<input type="hidden" name="num" value="100"></input>
					</td>
				</tr>
			</table>
</form>

		</td>
	</tr>
</table>

<!-- <div align="center">Articles citing this paper, <a href="http://scholar.google.com/scholar?hl=en&lr=&cites=15087050128968045568" target="_blank">according to Google Scholar</a></div>
<br /> -->
<div align="center">
<img src="http://static.delicious.com/img/delicious.small.gif" alt="icon" /> <a href="http://del.icio.us/post" onclick="window.open('http://delicious.com/post?v=4&amp;noui&amp;jump=close&amp;url='+encodeURIComponent(location.href)+'&amp;title='+encodeURIComponent(document.title), 'delicious', 'toolbar=no,width=700,height=400'); return false;">Bookmark This Page</a>
</div>
<hr size="1" style="color: #5E96FD;" />

<table align="center" cellpadding="10">
<tr><td align="center" valign="top"><div><a href="http://www.digits.com/" target="_blank">
<img src="http://counter.digits.com/?counter={45a042b1-c9d0-03e4-e9d0-cc0fc9fb8bfc}&amp;template=simple"
alt="Hit Counter by Digits" border="0" />
 <br />Web Counter</a>
</div></td>

<td align="center" valign="top"><div>
&copy; the authors, 2008. <br />Last updated: 18 October, 2007
</div></td>

         <td align="center" valign="middle"><img src="../valid-xhtml10.gif" alt="Valid XHTML 1.0!" height="16" width="44" /><!--ONESTAT SCRIPTCODE START-->
<!--
// Modification of this code is not allowed and will permanently disable your account!
// Account ID : 281971
// Website URL: http://InformationR.net/ir/
// Copyright (C) 2002-2006 OneStat.com All Rights Reserved
-->
<div id="OneStatTag"><table border='0' cellpadding='0' cellspacing='0'><tr><td align='center'>
<script type="text/javascript">
<!--
function OneStat_Pageview()
{
    var d=document;
    var sid="281971";
    var CONTENTSECTION="";
    var osp_URL=d.URL;
    var osp_Title=d.title;
    var t=new Date();
    var p="http"+(d.URL.indexOf('paper353.html')==0?'s':'')+"://stat.onestat.com/stat.aspx?tagver=2&sid="+sid;
    p+="&url="+escape(osp_URL);
    p+="&ti="+escape(osp_Title);
    p+="&section="+escape(CONTENTSECTION);
    p+="&rf="+escape(parent==self?document.referrer:top.document.referrer);
    p+="&tz="+escape(t.getTimezoneOffset());
    p+="&ch="+escape(t.getHours());
    p+="&js=1";
    p+="&ul="+escape(navigator.appName=="Netscape"?navigator.language:navigator.userLanguage);
    if(typeof(screen)=="object"){
       p+="&sr="+screen.width+"x"+screen.height;p+="&cd="+screen.colorDepth;
       p+="&jo="+(navigator.javaEnabled()?"Yes":"No");
    }
    d.write('<a href="http://www.onestatfree.com/aspx/login.aspx?sid='+sid+'" target=_blank><img id="ONESTAT_TAG" border="0" src="'+p+'" alt="This site tracked by OneStatFree.com. Get your own free site tracker."></'+'a>');
}

OneStat_Pageview();
//-->
</script>
<noscript>
<a href="http://www.onestatfree.com/"><img border="0" src="http://stat.onestat.com/stat.aspx?tagver=2&amp;sid=281971&amp;js=No&amp;" alt="online Web site analytics" /></a>
</noscript>
</td></tr><tr><td align='center'><div style="COLOR:black;display:none;FONT-FAMILY:'Verdana';"><a href="http://www.onestat.com/" style="text-decoration:none;">online Web site analytics</a><br /></div></td></tr></table></div>
<!--ONESTAT SCRIPTCODE END-->
</td></tr>
</table>
<hr size="3" style="color: #5E96FD;" />

<table align="center"><tr><td><div class="button">


<ul>
	<li><a href="infres133.html">Contents</a> | </li>
	<li><a href="../iraindex.html">Author index</a> | </li>
	<li><a href="../irsindex.html">Subject index</a> | </li>
	<li><a href="../search.html">Search</a> | </li>
	<li><a href="../index-2.html">Home</a></li>
</ul> 

</div></td></tr></table>

<hr size="3" style="color: #5E96FD;" />
<script src="http://www.google-analytics.com/urchin.js" type="text/javascript">
</script>
<script type="text/javascript">
_uacct = "UA-672528-1";
urchinTracker();
</script>
</body>

<!-- Mirrored from informationr.net/ir/13-3/paper353.html by HTTrack Website Copier/3.x [XR&CO'2010], Sat, 25 Dec 2010 19:13:58 GMT -->
</html>
