<!doctype html public "-//W3C//DTD HTML 4.0 Transitional//EN">
<html>

<!-- Mirrored from informationr.net/ir/6-2/ws1.html by HTTrack Website Copier/3.x [XR&CO'2010], Sat, 25 Dec 2010 19:11:17 GMT -->
<head>
<link rel="stylesheet" href="../IRstyle.css" type="text/css">
<title>Facilitating access to and use of bioinformatics resources</title>
<link rev=made href="mailto:t.d.wilson@shef.ac.uk">
<meta name="keywords" content="Human Genome Project, bioinformatics, information resources, databases, biological information, software, biomedical research, scientists, use, information behaviour, behavior, information-seeking behaviour">
<meta name="description" content="As the Human Genome Project nears completion, a vast and ever-expanding body of bioinformatics resources is being developed. This includes databases of biological information, and software tools that manipulate and analyze the data. Bioinformatics resources have significantly affected the conduct of biomedical research. However, very little research has been conducted into how scientists use these resources. This paper presents proposed dissertation research into one approach to facilitating access to bioinformatics resources.">
<meta name="rating" content="Mature">
<meta name="VW96.objecttype" content="Document">
<meta name="ROBOTS" content="ALL">
<meta name="DC.Title" content="Facilitating access to and use of bioinformatics resources">
<meta name="DC.Creator" content="Joan C. Bartlett">
<meta name="DC.Subject" content="Human Genome Project, bioinformatics, information resources, databases, biological information, software, biomedical research, scientists, use, information behaviour, behavior, information-seeking behaviour">
<meta name="DC.Description" content="As the Human Genome Project nears completion, a vast and ever-expanding body of bioinformatics resources is being developed. This includes databases of biological information, and software tools that manipulate and analyze the data. Bioinformatics resources have significantly affected the conduct of biomedical research. However, very little research has been conducted into how scientists use these resources. This paper presents proposed dissertation research into one approach to facilitating access to bioinformatics resources.">
<meta name="DC.Publisher" content="T.D. Wilson, Information Research">
<meta name="DC.Coverage.PlaceName" content="Global">
<script language="JavaScript" type="text/JavaScript">
<!--
function MM_preloadImages() { //v3.0
  var d=document; if(d.images){ if(!d.MM_p) d.MM_p=new Array();
    var i,j=d.MM_p.length,a=MM_preloadImages.arguments; for(i=0; i<a.length; i++)
    if (a[i].indexOf("#")!=0){ d.MM_p[j]=new Image; d.MM_p[j++].src=a[i];}}
}

function MM_findObj(n, d) { //v4.01
  var p,i,x;  if(!d) d=document; if((p=n.indexOf("?"))>0&&parent.frames.length) {
    d=parent.frames[n.substring(p+1)].document; n=n.substring(0,p);}
  if(!(x=d[n])&&d.all) x=d.all[n]; for (i=0;!x&&i<d.forms.length;i++) x=d.forms[i][n];
  for(i=0;!x&&d.layers&&i<d.layers.length;i++) x=MM_findObj(n,d.layers[i].document);
  if(!x && d.getElementById) x=d.getElementById(n); return x;
}

function MM_nbGroup(event, grpName) { //v6.0
  var i,img,nbArr,args=MM_nbGroup.arguments;
  if (event == "init" && args.length > 2) {
    if ((img = MM_findObj(args[2])) != null && !img.MM_init) {
      img.MM_init = true; img.MM_up = args[3]; img.MM_dn = img.src;
      if ((nbArr = document[grpName]) == null) nbArr = document[grpName] = new Array();
      nbArr[nbArr.length] = img;
      for (i=4; i < args.length-1; i+=2) if ((img = MM_findObj(args[i])) != null) {
        if (!img.MM_up) img.MM_up = img.src;
        img.src = img.MM_dn = args[i+1];
        nbArr[nbArr.length] = img;
    } }
  } else if (event == "over") {
    document.MM_nbOver = nbArr = new Array();
    for (i=1; i < args.length-1; i+=3) if ((img = MM_findObj(args[i])) != null) {
      if (!img.MM_up) img.MM_up = img.src;
      img.src = (img.MM_dn && args[i+2]) ? args[i+2] : ((args[i+1])? args[i+1] : img.MM_up);
      nbArr[nbArr.length] = img;
    }
  } else if (event == "out" ) {
    for (i=0; i < document.MM_nbOver.length; i++) {
      img = document.MM_nbOver[i]; img.src = (img.MM_dn) ? img.MM_dn : img.MM_up; }
  } else if (event == "down") {
    nbArr = document[grpName];
    if (nbArr)
      for (i=0; i < nbArr.length; i++) { img=nbArr[i]; img.src = img.MM_up; img.MM_dn = 0; }
    document[grpName] = nbArr = new Array();
    for (i=2; i < args.length-1; i+=2) if ((img = MM_findObj(args[i])) != null) {
      if (!img.MM_up) img.MM_up = img.src;
      img.src = img.MM_dn = (args[i+1])? args[i+1] : img.MM_up;
      nbArr[nbArr.length] = img;
  } }
}
//-->
</script>

<body bgcolor="#ffffff" onLoad="MM_preloadImages('../figs/iauthori1.gif','../figs/isubji1.gif','../figs/isearch1.gif','../figs/ihome1.gif','../figs/contents1.gif')">
<table align="center" border="0" cellpadding="0" cellspacing="0">
<tr><td height="30" align="center" colspan="5"><h4>Information Research, Vol. 6 No. 2, January 2001</h4></td></tr>
  <tr> 
    <td><a href="infres62.html" target="_top" onClick="MM_nbGroup('down','group1','contents','',1)" onMouseOver="MM_nbGroup('over','contents','../figs/contents1.gif','',1)" onMouseOut="MM_nbGroup('out')"><img name="contents" src="../figs/contents.gif" border="0" alt="" onLoad=""></a></td>
    <td><a href="../iraindex.html" target="_top" onClick="MM_nbGroup('down','group1','authorindex','',1)" onMouseOver="MM_nbGroup('over','authorindex','../figs/iauthori1.gif','',1)" onMouseOut="MM_nbGroup('out')"><img src="../figs/iauthori.gif" alt="" name="authorindex" width="120" height="20" border="0" onload=""></a></td>
    <td><a href="../irsindex.html" target="_top" onClick="MM_nbGroup('down','group1','subjindex','',1)" onMouseOver="MM_nbGroup('over','subjindex','../figs/isubji1.gif','',1)" onMouseOut="MM_nbGroup('out')"><img src="../figs/isubji.gif" alt="" name="subjindex" width="120" height="20" border="0" onload=""></a></td>
    <td><a href="../search.html" target="_top" onClick="MM_nbGroup('down','group1','search','',1)" onMouseOver="MM_nbGroup('over','search','../figs/isearch1.gif','',1)" onMouseOut="MM_nbGroup('out')"><img src="../figs/isearch.gif" alt="" name="search" width="120" height="20" border="0" onload=""></a></td>
    <td><a href="../index.html" target="_top" onClick="MM_nbGroup('down','group1','home','',1)" onMouseOver="MM_nbGroup('over','home','../figs/ihome1.gif','',1)" onMouseOut="MM_nbGroup('out')"><img name="home" src="../figs/ihome.gif" border="0" alt="" onLoad=""></a></td>
  </tr>
</table>
<hr color=#ff00ff size=3>
<h1>Facilitating access to and use of bioinformatics resources</h1>


<h4 align="center">
<a href="mailto:bartlett@fis.utoronto.ca">Joan C. Bartlett</a><br>
Faculty of Information Studies<br>
University of Toronto<br>
Toronto, Canada</h4>
<br><br>


<h2>Introduction</h2>

<p>As the Human Genome Project nears completion, a vast and ever-expanding
body of bioinformatics resources is being developed. This includes databases
of biological information, and software tools that manipulate and analyze
the data. Bioinformatics resources have significantly affected the conduct
of biomedical research. However, very little research has been conducted
into how scientists use these resources. This paper presents proposed dissertation
research<sup><a href="#1">1</a></sup> into one approach to facilitating
access to bioinformatics resources.</p>

<h2>Background</h2>

<p>The data generated through molecular biology research has led to the development
of a wide range of molecular biology databases, non-bibliographic databases
of information such as DNA sequences (e.g., GenBank), protein sequences
(e.g., Swiss-Prot), and genomic mapping information (e.g., Genome Database).
The development, management and use of these databases is one component
of the field of bioinformatics, "the computer-assisted data management
discipline that helps us gather, analyze and represent this information"
(<a href="#ref2">Persidis, 1999</a>: 828).</p>

<p>The Human Genome Project began in 1990, with the goal of sequencing
the entire human genome, localizing the estimated 100 000 genes, and creating
a genetic and physical map of the genome (<a href="#ref2">National Human Genome Research Institute, 2000</a>). The completion of a 'working draft' of the human genome
sequence was announced June 26, 2000, and one-third of the genome is anticipated
to be completely sequenced by 2001 (<a href="#ref2">Wadman, 1999</a>). The current goal is to have the complete sequence by 2002, two years ahead of the schedule
set in 1990 (<a href="#ref1">Baxevanis, 2000</a>: 1).

<p>The latest annual database issue of the journal Nucleic Acids Research
listed over 200 molecular biology databases (<a href="#ref1">Baxevanis, 2000</a>). The information in these databases includes sequences (DNA, RNA, and protein), gene structure, maps, mutations and genomes, among others. The GenBank database alone grows
an average of 2 million base pairs per day (<a href="#ref1">Baxevanis, 2000</a>). The National Library of Medicine ENTREZ system, the main search interface to both bibliographic (e.g., Medline, HealthSTAR) and molecular biology databases (e.g., GenBank), is queried 50 000 times per day (<a href="#ref2">Persidis, 1999</a>).

<p>As the number and size of molecular biology databases grow and their
use increases, concerns have been raised about the lack of expertise in
the area of bioinformatics (<a href="#ref1">Collins <i>et al.</i>, 1998</a>; <a href="#ref2">MacLean &amp; Miles, 1999</a>). Scientists are overwhelmed by the range and volume of bioinformatics resources available to them, and frequently lack the expertise to use them (<a href="#ref2">Yarfitz &amp; Ketchell, 2000</a>). Anecdotal evidence shows that it is not uncommon for scientists to spend a long period of time conducting wet lab research to obtain results that could also be obtained through the use of bioinformatics resources. The use of these resources can lead to savings
of both time and money for the researcher.

<p>To date, there has been little research into the use of these essential
research resources. However, as bioinformatics resources continue to expand
and develop in complexity and number, it is essential to understand the
users, what information they need from the resources, how they search for
information, and how they use that information. The results of research
in these areas could be used for many purposes. They can inform the design
and development of bioinformatics resources in order to make them responsive
to the needs of the users. They can be used as a basis for developing training
and education programs, so that scientists may become better informed and
more proficient in the use of bioinformatics resources. Research could
also lead to the development of tools to guide and assist scientists when
they are using these resources.

<p>Much of the past research into the information behaviour of life scientists
and molecular biologists has focused on the use of traditional, bibliographic
resources (<a href="#ref1">Bayer &amp; Jahoda, 1981</a>; <a href="#ref1">Curtis, Weller, &amp; Hurd</a>, 1993, 1997;  <a href="#ref1">Dillon, 1981</a>; <a href="#ref1">French, 1990</a>; <a href="#ref2">Garvey, Tomita, &amp; Woolf, 1974</a>; <a href="#ref2">Palmer</a>, 1991a, 1991b; <a href="#ref2">Rolinson, Meadows, &amp; Smith, 1995</a>; <a href="#ref2">Skelton, 1973</a>). Little research has addressed the use of non-bibliographic bioinformatics resources. Studies of molecular biologists found that, as early as 1991, molecular sequence databases were seen as having an increasingly important role, with the need to either become proficient with their use, or be left behind (<a href="#ref2">Grefsheim <i>et al.</i>, 1991</a>). At the same time, there was a call for help in managing the expanding array of information available (<a href="#ref2">Grefsheim <i>et al.</i>, 1991</a>). Almost 10 years later, a study found that 70% of molecular biologists surveyed were using molecular sequence databases on a weekly or monthly basis (<a href="#ref2">Yarfitz &amp; Ketchell, 2000</a>). While personal contacts with colleagues was the primary source of information about these resources, there was considerable interest in bioinformatics consultation services, classes, and other services to provide assistance in the use of bioinformatics resources (<a href="#ref2">Yarfitz &amp; Ketchell, 2000</a>).

<h2>Research objectives</h2>

The intent of my research is twofold: a) to assess the patterns of use
of bioinformatics resources (both molecular biology databases and related
analytical software tools) by expert users, and b) to create and test a
tool, based on a predictive model, that will facilitate the access and
use of bioinformatics resources by scientists. Among the challenges facing
scientists is the appropriate choice and use of bioinformatics resources.
This research will analyze how experts use them, to capture patterns of
use. That is, for scientists engaged in a particular type of research,
are there patterns or common features to their use of bioinformatics resources?
From these patterns, I will build a predictive model for the use of bioinformatics
resources for a particular scientific task, and use the model to develop
a tool to assist scientists in their selection and use.

<p>There are two specific research objectives:

<p>1. To identify and assess the patterns of use of bioinformatics resources
by experts, so that a predictive model can be developed.

<p>2. To create and test a computer-based tool, based on the predictive
model, that can assist novices to access and use bioinformatics resources.

<h2>Research design</h2>

<h3>Phase I</h3>

<p><i>Objective: To identify and assess the patterns of use of bioinformatics
resources</i></p>

<p>The objective of the first phase is to identify and assess the patterns
of use of bioinformatics resources by expert users, scientists who are
proficient in their use. The scientists will all be involved in similar
types of research, so that the tasks for which they require information
will be similar.</p>

<p>Twenty scientists will be interviewed using a semi-structured interview
script. This will allow the scientists to recount their experiences in
their own words, discussing what they consider to be important, while still
allowing the researcher to ensure that key points are covered. The focus
of the interview will be to understand the scientific task prompting the
use of bioinformatics resources, and the process by which they were used.
The initial interview question will ask the scientist to describe the steps
they would follow, and the databases and tools they would use to complete
a scientific task. Probe questions will be used to ensure that essential
points are covered. These will include:</p>

<ul>
<li>What sequence of events was followed?</li>
<li>What databases and tools were used?</li>
<li>What triggers this process?</li>
<li>What information do you have at the start of the process?</li>
<li>Does that information have to be prepared in any way?</li>
<li>What information comes out of the process at the end?</li>
<li>What happens after this?</li>
<li>How do you know when you are finished?</li>
</ul>

<p>Additional questions will explore the amount of variation in the process
as described by the scientist, in order to understand if the process could
be generalized to other situations. It is also important to understand
the key decision points in the process, why particular resources were chosen,
and the purpose of each type of analysis.</p>

<p>The data from the interviews will be analyzed to capture patterns in
the use of bioinformatics resources. That is, for a given scientific task,
are the bioinformatics resources accessed and utilized in a consistent
pattern. From these patterns, a predictive model will be developed. This
will be an iterative process, with the model being developed as the data
is analyzed, and tested and refined with each new set of data.</p>

<p>In addition to understanding the processes followed by the scientists,
it will also be necessary to understand the various resources used. Analysis
of the resources will involve a taxonomic classification. It is important
to understand the purpose and functions of the various databases and analytical
tools, in terms of how each one is used, the capabilities of each, and
the similarities and differences. It will be particularly important to
understand which resources have similar or overlapping functions, and which
are different.</p>

<p>The interview data will also be analyzed to understand the reasons behind
the processes described. It will be important to understand not only what
resources were used, but how and why. The qualitative analysis will enhance
the quantitative focus that will lead to the development of the model.</p>

<h3>Phase II</h3>

<p><i>Objective: To develop and test a tool to facilitate access to and
use of bioinformatics resources</i></p>

<p>The second phase will involve the creation and evaluation of a tool
to assist scientists to access and use bioinformatics resources. The tool
will be developed based on the model from Phase I, and could take several
forms. Among these are a search guide, a decision support tool, or training
materials. The findings from the interviews will help determine the most
appropriate form of the tool. The exact form of the tool, or what it will
cover will not be known until the data from the first phase has been analyzed.</p>

<p>Once developed, the tool will be tested with a group of scientists who
are not expert in bioinformatics, to determine whether it can facilitate
the access to and use of bioinformatics resources. The testing will evaluate
both the model on which the tool was based, and the tool itself. However,
the emphasis will be on testing and refining the predictive value of the
model and its use as the basis of a tool, rather than on the usability
of the tool. Test participants will be assigned a standardized scientific
task (such as the generation of a restriction map, or the characterization
of a novel sequence), for which bioinformatics resources could provide
a solution. They will be asked to solve the problem, and will be randomly
assigned to either use the tool, or not. Among the factors to be considered
in the evaluation are efficiency, effectiveness and user satisfaction.</p>

<h2>Outcomes</h2>

<p>This research will determine whether the usage patterns of bioinformatics
experts are sufficiently predictive to assist other (non-bioinformatics
expert) scientists to access and use bioinformatics resources. It is anticipated
that there will be patterns that can be followed to complete a scientific
task.</p>

<p>An simple example of such a task is the determination of how to remove
a specific segment of DNA from a bacterial chromosome. First, the entire
sequence of the chromosome must be identified. This information is available
from publicly accessible sequence databases. Then, the retrieved sequence
must be analyzed by a computer program which searches for and identifies
the location of the specific points on the DNA which will be cut by specialized
enzymes. The output of the analysis is a map of all of the cut sites on
the chromosome. The researcher would then identify those sites which flank
the segment of interest that they wish to remove.</p>

<p>This example, while describing a fairly simple scientific task, nonetheless
demonstrates the type of patterns which may emerge from the usage patterns
of experts. The database and analytical tool used may vary among scientists,
and the choice may be influenced by several factors, including the specific
details of the task (for example, the organism being studied), the resources
available, and personal preference. However, the class of resource used
would be predictable.</p>

<p>It is expected that this dissertation research will lead to the development
of a tool to assist scientists in the application of bioinformatics resources
to one particular task. This could provide a solution to the problem of
scientists being overwhelmed by the range of resources available to them,
and assist them in navigating among them. This research will also investigate
the feasibility of using a predictive model, developed by studying expert
users, to create a decision support tool, a search guide or training materials.
If this is found to be effective, then the approach can be applied to other
scientific tasks for which bioinformatics resources are used. The approach
could also be applied to other disciplines with comparable information
resources.</p>

<br>
<hr color="#85FCA9" size="3">
<h4>Note</h4>
<p><a name="1"></a><sup>1</sup> Dissertation Committee: <br>Elaine Toms (Supervisor), Associate Professor, Faculty of Information Studies, University of Toronto;&nbsp;&nbsp;Joan Cherry, Professor and Associate Dean, Faculty of Information Studies, University of Toronto;&nbsp;&nbsp;Jamie Cuticchia, Head and Senior Bioinformatics Scientist, Bioinformatics Supercomputing Centre, Hospital for Sick Children, Toronto.
<hr color="#85FCA9" size="3">
<br>
<h2>References</h2>

<ul><a name="ref1"></a>
<li>Baxevanis, A. D. (2000). &quot;The Molecular Biology Database Collection: an
online compilation of relevant database resources.&quot; <i>Nucleic Acids Research</i>,
<b>28</b>(1), 1-7.</li>

<li>Bayer, A., &amp; Jahoda, G. (1981). &quot;Effects of online bibliographic
searching on scientists' information style.&quot; <i>On-Line Review</i>, <b>5</b>,
323-33.</li>

<li>Collins, F. S., Patrinos, A., Jordan, E., Chakravarti, A., Gesteland,
R., &amp; Walters, L. (1998). &quot;New goals for the U.S. Human Genome Project:
1998-2003.&quot; <i>Science</i>, <b>282</b>(5389), 682-9.</li>

<li>Curtis, K. L., Weller, A. C., &amp; Hurd, J. M. (1993). &quot;Information-seeking
behaviour: a survey of health sciences faculty use of indexes and databases.&quot;
<i>Bulletin of the Medical Library Association</i>, <b>81</b>(4), 383-392.</li>

<li>Curtis, K. L., Weller, A. C., &amp; Hurd, J. M. (1997). &quot;Information-seeking
behavior of health sciences faculty: the impact of new information technologies.&quot;
<i>Bulletin of the Medical Library Association</i>, <b>85</b>(4), 402-10.</li>

<li>Dillon, M. (1981). &quot;Serving the information needs of scientific research.&quot;
<i>Special Libraries</i>, <b>72</b>, 215-23.</li>

<li>French, B. A. (1990). &quot;User needs and library services in agricultural
sciences.&quot; <i>Library Trends</i>, <b>38</b>, 415-41.</li>

<li><a name="ref2"></a>Garvey, W. D., Tomita, K., &amp; Woolf, P. (1974). &quot;The dynamic scientific
information user.&quot; <i>Information Storage and Retrieval</i>, <b>10</b>,
115-31.</li>

<li>Grefsheim, S., Franklin, J., &amp; Cunningham, D. (1991). &quot;Biotechnology
awareness study, part 1: where scientists get their information.&quot; <i>Bulletin
of the Medical Library Association</i>, <b>79</b>(1), 36-44.</li>

<li>MacLean, M., &amp; Miles, C. (1999). &quot;Swift action needed to close the
skills gap in bioinformatics.&quot; <i>Nature</i>, <b>401</b>, 10.</li>

<li>National Human Genome Research Institute. (2000). &quot;Human Genome Project
Goals: 1998-2003.&quot; Bethesda, MD: Retrieved March 24, 2000 from the World
Wide Web: http://www.nhgri.nih.gov/HGP/.</li>

<li>Palmer, J. (1991a). &quot;Scientists and information: I. Using cluster analysis
to identify information style.&quot; <i>Journal of Documentation</i>, <b>47</b>(2),
105-29.</li>

<li>Palmer, J. (1991b). &quot;Scientists and information: II. Personal factors
in information behaviour.&quot; <i>Journal of Documentation</i>, <b>47</b>(3),
254-75.</li>

<li>Persidis, A. (1999). &quot;Bioinformatics.&quot; <i>Nature Biotechnology</i>,
<b>17</b>, 828-30.</li>

<li>Rolinson, J., Meadows, A. J., &amp; Smith, H. (1995). &quot;Use of information
technology by biological researchers.&quot; <i>Journal of Information Science</i>,
<b>21</b>(2), 133-9.</li>

<li>Skelton, B. (1973). &quot;Scientists and social scientists as information
users: a comparison of results of science user studies with the investigation
into information requirements of the social sciences.&quot; <i>Journal of Librarianship</i>, <b>5</b>(2), 138-56.</li>

<li>Wadman, M. (1999). &quot;Human Genome Project aims to finish 'working draft'
next year.&quot; <i>Nature</i>, <b>398</b>, 177.</li>

<li>Yarfitz, S., &amp; Ketchell, D. S. (2000). &quot;A library-based bioinformatics
services program.&quot; <i>Bulletin of the Medical Library Association</i>,
<b>88</b>(1), 36-48.</li>



<hr color="#85FCA9" size="4">
<p style="text-align : center; color : Red; font-weight : bold;">How to cite this paper:</i></p>

<p style="text-align : left; color : black;">Bartlett, Joan C.  (2001) &nbsp; &quot;Facilitating access to and use of bioinformatics resources&quot;. &nbsp;Information Research, <strong>6</strong>(2)  Available at: http://InformationR.net/ir/6-2/ws1.html</p>

<p style="text-align : center">&copy; the author, 2001.  Updated: 5th January 2001</div>

<hr color="#85FCA9" size=4>
<table border="0" cellpadding="15" cellspacing="0" align="center">
<tr> 
    <td><a href="infres62.html"><h4>Contents</h4></a></td>
   <td align="center" valign="top"><h5 align="center"><img src="http://counter.digits.com/wc/-d/-z/6/-b/FF0033/ws1" align=middle  width=60 height=20 border=0 hspace=4 vspace=2><br><a href="http://www.digits.com/">Web Counter</a></h5></td>
    <td><a href="../index.html"><h4>Home</h4></a></td>
  </tr>
</table>		
<hr color="#85FCA9" size=4>
</body> 

<!-- Mirrored from informationr.net/ir/6-2/ws1.html by HTTrack Website Copier/3.x [XR&CO'2010], Sat, 25 Dec 2010 19:11:17 GMT -->
</html>


