<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
	"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd"> 
<html xmlns="http://www.w3.org/1999/xhtml"> 

<!-- Mirrored from informationr.net/ir/15-1/paper427.html by HTTrack Website Copier/3.x [XR&CO'2010], Sat, 25 Dec 2010 19:33:51 GMT -->
<head> 
<title>A conceptual framework of information requirements for scientists using human biological samples</title> 
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1" /> 
<link href="../IRstyle2.css" rel="stylesheet" media="screen" title="serif" /> 
<link rel="alternate stylesheet" type="text/css" media="screen" title="sans" href="../IRstylesans.css" /> 
<link rev="made" href="mailto:t.d.wilson@shef.ac.uk" /> 
 
<!--Enter appropriate data in the content fields--> 
<meta name="dc.title" content="A conceptual framework of information requirements for scientists using human biological samples" /> 
<meta name="dc.creator" content="Sujin Kim" /> 
<meta name="dc.subject" content="A conceptual framework of information requirements for scientists using human biological samples" /> 
<meta name="dc.description" content="Introduction. This study was undertaken to develop an information requirement framework for scientists who use biological samples and related data in their research. Method. A self-reporting questionnaire completed by 137 respondents was used to collect data regarding demographics, bio-sample management, bio-sample use and requirements, data requirements, and work and research-related roles and activities.  Analysis. Descriptive and TwoStep Cluster analyses were used to analyse the survey data necessary for developing a framework of information requirements.  Results. Two groups of biomedical scientists (clinical group and basic scientist group) were formed by their distinct characteristics. A conceptual framework of information requirements for bio-sample researchers was formed. The study determined the following as core components: work roles, tasks, characteristics of data and bio-sample needs, factors affecting information seeking, and outcomes. Conclusions. This study will enable the system designer to understand bio-sample users by means of their information requirements resulted in the proposed framework. Future empirical studies should assess potential users, types of information required depending on their work-related roles, factors affecting information seeking, and the evaluation of information seeking effectiveness. " /> 
<meta name="dc.subject.keywords" content="bio-samples, information requirements, information needs, biomedicine, bio-repositories" /> 
 
<!--leave the following to be completed by the Editor--> 
<meta name="robots" content="all" /> 


<meta name="dc.publisher" content="Professor T.D. Wilson" /> 
<meta name="dc.coverage.placename" content="global" /> 
<meta name="dc.type" content="text" /> 
<meta name="dc.identifier" scheme="ISSN" content="1368-1613" /> 
<meta name="dc.identifier" scheme="URI" content="paper427.html" /> 
<meta name="dc.relation.IsPartOf" content="infres151.html" /> 
<meta name="dc.format" content="text/html" /> 
<meta name="dc.language" content="en" /> 
<meta name="dc.rights" content="http://creativecommons.org/licenses/by-nd-nc/1.0/" /> 
<meta name="dc.date.available" content="2009-03-15" /> 
 
<script language="javascript" type="text/javascript"> 
 
		var flag;
		flag = true;
		function doChangeFont()
		{
			if (flag)
			{
			var htmlDoc = document.getElementsByTagName('head').item(0);
			var css = document.createElement('link');
			css.setAttribute('rel', 'stylesheet');
			css.setAttribute('type', 'text/css');
			css.setAttribute('href', '../sans.css');
			htmlDoc.appendChild(css);
			flag = false;
			} 
			else
			{
			var htmlDoc = document.getElementsByTagName('head').item(0);
			var css = document.createElement('link');
			css.setAttribute('rel', 'stylesheet');
			css.setAttribute('type', 'text/css');
			css.setAttribute('href', '../IRstyle2.css');
			htmlDoc.appendChild(css);
			flag = true;
			}	
		}
		
	</script> 
	
<style type="text/css"> 
.button {
	width: 45em;
	padding: 0 0 0 0;
	font-family: Verdana, Lucida, Geneva, Helvetica, Arial, sans-serif;
	font-size: small;
	font-weight: bold;  
	background-color: #ffffff;
	color: #000000;
	display: inline;
	text-align: center;
	}
		
 
.button ul {
		list-style: none;
		margin: 0;
		padding: 0;
		border: none;
		display: inline;
		}
		
.button li {
		margin: 0;
		font-family: Verdana, Lucida, Geneva, Helvetica, Arial, sans-serif;
	    font-size: small;
	    font-weight: bold;  
		background-color: #fff000<!-- #2175bc; --> 
		color: #000000;
		text-decoration: none;
		display: inline;
		}
 
.button li a:hover {
		background-color: azure;
		color: #ff0000;
		width: auto;
		}
		
fieldset {
    padding: .5em;
    background: white;
    border: 1px dotted #5E96FD;
    margin-left: 15px;
    margin-right: 15px;
    margin-top: .5em;
	}
 
legend {
    color: white;
    background-color: #5E96FD;
    font-size: medium;
    padding: .1ex .5ex;
    border-right: 1px solid navy;
    border-bottom: 1px solid navy;
    font-weight: bold;
}
 </style> 
 </head> 
<body bgcolor="#ffffff"> 
<table cellspacing="0" cellpadding="0" align="center" border="0">
  <tbody>
  <tr>
    <td align="center" colspan="5" height="30"><img height="45" alt="header" src="../mini_logo2.gif" 
      width="336" /><br /><span style="font-size: medium; font-variant: small-caps; font-weight: bold;">vol. 15  no. 1, March, 2010</span><br /><br />
      <div class="button">
      <ul>
        <li><a href="infres151.html">Contents</a> |  </li>
        <li><a href="../iraindex.html">Author index</a> |  </li>
        <li><a href="../irsindex.html">Subject index</a> |  </li>
        <li><a href="../search.html">Search</a> |  </li>
        <li><a href="../index-2.html">Home</a>  </li>
   </ul></div></td></tr>
  <tr>
    <td>&nbsp;</td></tr></tbody></table>


<hr style="COLOR: #5E96FD" size="1" /> 
 
<h1>A conceptual framework of information requirements for scientists using human biological samples</h1> 
<br /> 
<div style="margin-left:10%;margin-right:10%"> 
<h4><a href="#author" target="_blank">Sujin Kim</a><br /> 
339 Lucille Little Library Building, School of Library and Information Science and Department of Pathology and Laboratory Medicine, University of Kentucky, Lexington, Kentucky 40506-0224, USA</h4> 
</div> 
<br /> 
 
<form action="#"> 
<fieldset> 
<legend><i>Abstract </i></legend> 
<blockquote>
<strong>Introduction.</strong> This study was undertaken to develop an information requirement framework for scientists who use biological samples and related data in their research.<br />  
<strong>Method.</strong> A self-reporting questionnaire completed by 137 respondents was used to collect data regarding demographics, bio-sample management, bio-sample use and requirements, data requirements, and work and research-related roles and activities.  <br />  
<strong>Analysis.</strong> Descriptive and TwoStep Cluster analyses were used to analyse the survey data necessary for developing a framework of information requirements. <br />  
<strong>Results.</strong> Two groups of biomedical scientists (clinical group and basic scientist group) were formed by their distinct characteristics. A conceptual framework of information requirements for bio-sample researchers was formed. The study determined the following as core components: <i>work roles, tasks, characteristics of data and bio-sample needs, factors affecting information seeking,</i> and <i>outcomes</i>.<br /> 
<strong>Conclusions.</strong> This study will enable the system designer to understand bio-sample users by means of their information requirements resulted in the proposed framework. Future empirical studies should assess potential users, types of information required depending on their work-related roles, factors affecting information seeking, and the evaluation of information seeking effectiveness. </blockquote> 
</fieldset> 
</form> 
 
<br /> 
<div align="center"> 
<input type="button" value="change font" class="btn" style="font-variant: small-caps; font-weight: bold; font-family: Verdana; color: white; background-color: #5E96FD;" onclick="doChangeFont()" /></div> 
<br /> 
 
<h2>Introduction</h2> 
 
<p>In recent years, the biomedical research community, specifically those branches related to human biological repository networks, have been a special target of information sharing efforts. However, very little research has been conducted to investigate the specific information needs of biomedical scientists relevant to the sharing network. These biomedical scientists are a group of scientists whose primarily interest is studying biological functions, phenomena and interaction in the context of medical science. These groups of professionals are unique in terms of the complexity of their multidisciplinary, collaborative research and practices. Traditionally, they have been characterized by their academic disciplines, such as biology, chemistry, physiology and medical science. However, the traditional boundaries of the academic disciplines are increasingly expanding to embrace other fields. In this regard, several biomedical fields have emerged to collaborate with other fields within the sciences for the pursuit of effective and advanced biomedical discoveries. </p> 
 
<p>Increasingly, science, technology, and medical libraries and information centers have worked hard to serve the interdisciplinary group of biomedical professionals in the fast growing genomic era. The human bio-repository network has become critical to genomic research. The network is described as a collaborative resource sharing vehicle for human biological samples (such as human tissues, blood and urine) among multidisciplinary biomedical scientists. The major issue encountered by information centers is the lack of research on the information requirements of biomedical scientists. More importantly, information centres serve their clients within the traditional boundaries of information sources such as journal articles, reference books, monographs, etc. Obviously, biomedical scientists seek not only the scientific discoveries published in academic papers, but also scientific (raw) data such as genomic and proteomic sequences accessible through databases, which are important sources for further references. More demands on non-traditional library collections, such as scientific data, hospital records, clinical images, genomic and proteomic sequences, will challenge libraries and information professionals to understand both the multidisciplinary nature of these scientists and their specific information requirements in various formats.</p>

<p>Obviously, it will be beneficial to biological repositories to learn how the resources are acquired, catalogued, retrieved, and circulated through the library's collaborative approach (e.g., cooperative cataloguing, union collection development, and interlibrary loan). In this way, libraries and information scientists are able to expand their roles by learning the complex nature of biomedical scientists and their information requirements. This study aimed to investigate the characteristics of biomedical scientists and their information requirements, including biological materials as well as data requirements. The understanding of the major characteristics found in this study resulted in a proposed conceptual framework of information requirements for researchers using bio-samples. The framework developed by the current study will enable a system designer to understand such researchers in relation to their professional work roles. More importantly, information scientists will be able to expand their understanding of information organization and the use of physical objects, such as tissues and blood, along with non-traditional information types such as clinical, morphological, and genetic information.</p>

<h2>Background</h2>

<p>The following sections review three research areas that support the groundwork of this study: biological sample repositories (bio-repositories) and their foremost issues, information requirements of healthcare professionals, and bio-repositories in Korea.</p> 

<h4>Issues confronting bio-repository networks</h4>

<p>As the demand for biological materials and their associated information increases, there has been a consequent demand for high-quality human bio-samples to support various biomedical researchers worldwide (<a href="#mit02">Mitchell 2000</a>; <a href="#gaj02">Gajiwala 2002</a>; <a href="#tet05">Tettamanti <em><em>et al.</em></em> 2005</a>; <a href="#alv03">Alvarez <em><em>et al.</em></em> 2003</a>; <a href="#goe05">Goebell 2005</a>). The conventional sources for acquiring biological materials provide limited access to those who work in a research environment where an organization's sample collection is only available to in-house researchers. Even if the local collections are available, it is hard to identify the relevant samples for an individual project because of the lack of accompanying sample information. Moreover, the residual samples in the local collection are seldom shared among researchers, many of whom would not have to collect similar samples if a network were available. Potentially, researchers who share similar samples can generate higher validity-based research using a larger sample pool.</p>

<p>To increase accessibility to biological samples, sharing networks have been established. The centralized biological repository does not have to be a physically centralized repository (<a href="#eis03">Eiseman <em><em>et al.</em></em> 2003</a>). A virtual network of biological materials accompanied by associated sample information is a more desirable approach (<a href="#ber03">Berman 2003</a>; <a href="#gil04">Gilder 2004</a>; <a href="#man01">Manley 2001</a>; <a href="#lee06">Lee 2006</a>). The concept of the collaborative sharing network is much like inter-library lending in the library community. It is obvious that all of the required amounts and varieties of samples cannot be supported through a single institution. Geographically diverse locations make the cooperative management of biological samples even harder. In a collaborative research setting, such as multi-centre genetic studies, it is difficult to get approval for the use of biological samples because of varying degrees of local policies on human samples for research. Considering the tremendous amount of research funds and efforts spent for biological samples, it is imperative to construct a centralized repository for a bio-sample resource sharing network (<a href="#com05">Compton 2005</a>; <a href="#fri03">Friede <em>et al.</em> 2003</a>). </p>

<p>Lack of accessibility to quality human samples is not the only issue for bio-repositories. Most attention has been given to the collection of samples, while no standardized surrogate tool, such as catalogue-like accessibility, has been discussed for the repositories. Repositories range from the commercially available to the federally funded, and more biological samples are accessible through federated online searching tools such as the Specimen Resource Locator, the Tissue Expediter, and National Cancer Institute-Supported Specimen Resources (OBBR) (<a href="#nci08">National Cancer Institute... 2008</a>). However, these tools are limited to a small number of collections on specific disease categories funded by agencies affiliated to the National Institutes of Health. No standardized practice of description has focused on the samples available through repositories. In addition, the information accompanying the collected samples is minimal and can only support limited searching options. It is critical to build an applicable standard for sharing information about quality human samples (<a href="#kim07">Kim and Gilbertson 2007</a>; <a href="#kim08">Kim and Rasmussen 2008</a>).</p>

<p>Additionally, there has been a noticeable dearth of studies on bio-sample users and their data requirements. Various types of biomedical researchers working on complex projects frequently need correlations among the experimental result bio-samples with known parameters for the sample in the experiment. Complex data on each specimen (e.g., sample types, processing, amount, storage, quality assurance and quality control) need to be combined with large numbers of data elements (e.g., pathology data, outcome data and therapy data), and extracted from multiple data sources (e.g., electronic medical records, registries and surgical pathology reports) (<a href="#kim07">Kim and Gilbertson 2007</a>; <a href="#kim08">Kim and Rasmussen 2008</a>). Moreover, the study results for bio-samples (e.g., assay results) are correlated positively to the experimental samples so that the experimental results can be shared among researchers with a similar research interest. As this new area develops and databases or applications are constructed, it is imperative to understand the characteristics of researchers in this field and their data requirements for better resource management.</p>

<h4>Information requirements by healthcare professionals</h4>

<p>Drawing upon the contexts of information seeking (behaviour) studies, information requirements by professionals in a specific domain are an important component of an information model. In his seminal review on information behaviour research, Wilson (<a href="#kim07">2000</a>) emphasized that '<em>the performance of particular tasks, and the processes of planning and decision-making</em>' at <em>work role</em> level is important to information needs. Leckie <em>et al.</em> (<a href="#lec96">1996</a>) found that work roles and tasks are considered prime factors in the information requirements of professionals. Related to the work roles and tasks are professionals' complex job roles involving multiple dimensions of performance (for example, they are expected to perform clinically, scholarly, didactically and managerially). Obviously, the analysis of the information requirements of professionals performing multiple and complex roles is essential to effectively support scientific advancements in biomedical research. </p>

<p>As is evident from previous research findings, few studies have investigated how research-oriented professionals in healthcare seek information to satisfy job-related activities. Most relevant findings can be drawn from studies on basic scientists and engineers. Research done on work-related use of information by basic scientists shows that an information requirement arises from a project, task or problem and the received information affects their productivity and the types of activities they undertake. Studies of diverse professional groups have all concluded that professionals are frustrated in their search for relevant and necessary information (<a href="#orr70">Orr 1970</a>). Frustration becomes more evident if the availability and accessibility of required resources such as bio-samples and the accompanying data are relatively restricted. </p>

<p>In the Orr model, an acceptable timeline and the costs associated with the relevant information are major factors affecting successful information delivery. In Leckie's model, the most important variables are familiarity and prior success with the sources, along with the trustworthiness, packaging, timeliness, cost, quality and accessibility of the sources (<a href="#lec96">Leckie 1996</a>). Therefore, a conceptual framework attempting to capture distinct elements of the information requirement patterns of professionals in the bio-sample domain should be studied for better scientific productivity as well as for effective resource management. </p>

<p>While not widely implemented, there are some data standards for bio-repository information systems which are under extensive review. The Cancer Data Standards Repository, the Cooperative Prostate Cancer Tissue Resources common data element and cancer Text Information Extraction System are currently adopted as data standards by some bio-repository information systems (<a href="#pat06">Patel 2006</a>). It is highly recommended that repositories supported by the National Cancer Institute should be interoperable with core data sets in the Cancer Data Standards Repository (<a href="#cov03">Covitz <em>et al.</em> 2003</a>; <a href="#mcs08">McSherry 2008</a>). The College of American Pathologists cancer check lists are also regarded as a pathology findings standard within the cancer Biomedical Informatics Grid semantic framework (<a href="#cap08">College of American Pathologists 2008</a>; <a href="#tob06">Tobias <em>et al.</em> 2006</a>). In addition, the best-practice document of the International Society for Biological and Environmental Repositories specifies the essential data sets to be associated with biological samples (<a href="#isb08">International Society... 2008</a>). These include specimen location, other sample descriptors and  additional information for human specimens such as donor information, diagnosis, diagnostic procedures, type of treatment, surgical procedure information, medical history, family history, smoking history, vitals, clinical laboratory values and availability of other biological specimens from the same donor. Likewise, the data elements recommended by various organizations should be compiled into a simple framework for better understanding data and sample requirements. Considering the primitive use of the existing resource sharing networks, the current study was designed to capture pre-analysis of the requirements used for the early stage of system design.  </p>

<h4>bio-repositories in Korea</h4>

<p>This study is limited to data and sample requirements of Korean biomedical researchers; therefore, the review of the bio-repository network in Korea is briefly described in the following section. Previously, the human bio-repositories in Korea, sponsored by the Korean National Research Resource Centre, have been a major driving force for the systematic collection and storage of biological samples. Currently, more publicly available samples are available through the Centre's network repositories. In the pursuit of a centralized bio-sample network, human repositories such as the Korean Frozen Lung Tissue Bank, Korean Cell Line Bank, Korean Leukemia Cell and Gene Bank, Korean Eye Tissue Bank, Korean Liver Cancer Tissue Bank and a number of branch banks, extend their services to basic and clinical scientists nationwide. For instance, the Korean Frozen Lung Tissue Bank has a virtual tissue network that connects to more than seventeen member branch banks to share human frozen lung tissues and developed comprehensive sample descriptions for advanced searching options. For the broader spectrum of research resource sharing perspectives, the Korean National Research Resource Centre and Singapore Tissue Network, in collaboration with the International Society for Biological and Environmental Repositories, is undertaking discussions to build an Asia-Pacific bio-sample sharing network.</p>
 
<h2>Methods</h2>

<p>The study was conducted to understand the major characteristics of biospecimen researchers and their bio-sample and data requirements so that biological resources can be effectively and efficiently shared among those who require the resources for scientific discoveries. The following section describes research methods and data analysis performed in the study.</p>

<h3>Research questions</h3>

<p>The study addressed the following four research questions:</p>

<ol>
<li>What are general characteristics of bio-sample-based researchers and their requirements in terms of demographics, work or research related issues, bio-sample use and requirements and data requirements? (General characteristics of respondents).</li>
<li>Are there any distinct characteristics of the respondents by their work roles? (Distinct characteristics of the respondents by their work roles).</li>
<li>Are there any distinct groups of the respondents characterized by bio-sample and data requirements? (Natural groups of the participants).</li>
<li>Are the study's findings applicable to the development of a framework of information requirements for biomedical scientists? (A framework development).</li>
</ol>

<p>Several operational definitions are used in this study. 'Information requirements' refers to identified information needs of biomedical scientists that satisfy a work-related goal (modified from <a href="#wil00">Wilson 2000</a>). For studying work-related characteristics, the study used the conventional work roles in bio-repositories, including tissue bankers, medical doctors, and pathology specialists. 'bio-sample-based researchers' are those who require biological samples for their research or work. 'Tissue-bankers' refers to a group of science laboratory workers whose primary responsibilities are to collect, store, annotate and distribute biological samples. 'Pathology specialists' are a more research-oriented group compared to tissue bankers in terms of their research role. The 'medical doctor' group refers to those whose primarily responsibilities are patient care as well as clinical research. The work roles are not exclusive because a pathologist whose primary job is to make a histologic diagnosis for liver cancer may also run a clinical trial research laboratory for discovering effective biomarkers for liver cancer diagnosis. More detailed measures in relation to individual survey questions are discussed below.</p>

<h3>Survey respondents and data collection</h3>

<p>Data for this study were obtained from a sample of 137 individuals who answered a request to take part in an online survey between June 19, 2007 and October 19, 2007. The survey was limited to Korean biomedical researchers who were 18 years or older. Invitations to participate in the survey were sent to the Biological Research Information Centre, a major biomedical research message board (<a href="http://bric.postech.ac.kr/" target="_blank">http://bric.postech.ac.kr/</a>) and the Korea Human bio-repository Network, a major Korean network. The questionnaire (in Korean) was securely linked to a database in which all information was collected. The institutional review board exemption certificate was obtained before survey distribution. The survey consisted of a self-administered questionnaire and included questions about participants' demographics, research or work-related matters, sample use and requirements and data requirements. The English translation of the questionnaire is included in <a href="#appA">
Appendix A</a>.</p>

<h3>Measures</h3>

<p>The following measures were used to analyse the survey data, including research or work-related matters, sample use and requirement and data requirements. A summary of measures corresponding to survey questions is given in Table 1. The survey asked participants about six different researches or work related activities, including: area of specialties (question 6), funding sources (question 19), interesting organ parts (question 20), interesting diseases (question 22), publication venues (question 23), and attending conferences (question 24). All of these questions target distinct research or work-related activities which can allow us to further assess bio-sample and data requirements. Multiple answers were allowed for those who specialized in more than one area of interest. </p>

<table width="80%" border="1" cellspacing="0" cellpadding="3" align="center" style="border-right: #99f5fb solid; border-top: #99f5fb solid; font-size: smaller; border-left: #99f5fb solid; border-bottom: #99f5fb solid; font-style: normal; font-family: verdana, geneva, arial, helvetica, sans-serif; background-color: #fdffdd">

<caption align="bottom"><br /><strong>Table 1: Measures corresponding to survey questions</strong></caption>

<tr><th style="width: 10%;">Measure</th><th style="width: 20%;">Survey questions</th><th style="width: 10%;">Measure</th><th style="width: 20%;">Survey questions</th><th style="width: 10%;">Measure</th><th style="width: 20%;">
Survey questions</th></tr>

<tr><td rowspan="6">Research or work-related activities</td><td valign="top" align="center">Area of specialties<br /> (question 6)</td><td rowspan="6" align="center">Sample use and requirements</td><td valign="top" align="center">Human &amp; animal tissue use<br /> (questions 7 &amp; 8)</td><td rowspan="8" align="center">Data equirements</td><td valign="top" align="center">Patient demographics<br /> (questions 44-53)</td></tr>

 <tr><td valign="top" align="center">Funding sources<br /> (question 19)</td><td valign="top" align="center">Experience with bio-repositories (questions 9-11, &amp; 17)</td><td valign="top" align="center">General health conditions <br />(questions 54-60)</td></tr>

 <tr><td valign="top" align="center">Interesting organ parts<br /> (question 20)</td><td valign="top" align="center">Sample types <br />(question 12)</td>
  <td valign="top" align="center">Laboratory findings<br /> (questions 61-67)</td></tr>
  
 <tr><td valign="top" align="center">Interesting diseases <br />(question 22)</td><td valign="top" align="center">Processing and storage methods<br /> (questions 13 and 14)</td><td valign="top" align="center">Sample-specific elements (questions 68-90)</td></tr>

 <tr><td valign="top" align="center">Publication venues<br />(question 23)</td><td valign="top" align="center">Requesting amount, frequency and planned budget<br /> (questions 15, 16, &amp; 18)</td><td valign="top" align="center">Sample collection <br />(questions 68,70,78-79 &amp; 84)</td></tr>

 <tr><td valign="top" align="center">Attending conferences<br /> (question24)</td><td valign="top" align="center">Request specific criteria<br /> (question 21)</td><td valign="top" align="center">Sample processing and storage <br />(questions 69 &amp; 80-82)</td></tr>
 <tr><td rowspan="2" colspan="4" valign="top">&nbsp;</td><td valign="top" align="center">Sample annotation<br /> (questions 71-77)</td></tr>
 <tr><td valign="top" align="center">Sample distribution<br /> (questions 86-90)</td></tr>
</td>
</table>

<p>Sample use and requirements were assessed using thirteen questions, which asked respondents about their experiences of bio-sample usage and current or expected sample requirements, including human and animal tissue use (question 7 &amp; 8); experience with bio-repositories (questions 9-11, &amp; Q17); sample types (question 12); processing and storage methods (questions 13 and 14); requesting amount, frequency, and planned budget (questions 15, 16 &amp; 18); and request specific criteria (question 21). The response options were partially modified from caTISSUE Core search fields (cancer Biomedical Informatics Grid tissue bank repository tool) and National Biospecimen Network Blueprint survey questions on bio-sample and data requirements (<a href="#kim08">Kim and Rasmussen 2008</a>; <a href="#fri03">Friede <em>et al.</em> 2003</a>; <a href="#mcs08">McSherry and Paul 2008</a>). The caTISSUE Core was used in this study because it is the open source recommended by the National Cancer Institute for wide usage by comprehensive cancer centers in the U.S.</p>

<p>A list of required data elements to assess the participants' needs for specific kinds of information was collected from three sources, including the (US) Health Insurance Portability and Accountability Act (HIPAA) identifiers, Korean Frozen Lung Tissue Bank and caTISSUE Core, which are considered the representatives of protected and usable data elements regulated or recommended by government-affiliated agencies. A measure for data requirements was further divided into HIPAA protected identifiers as opposed to non-HIPAA elements. The HIPAA elements were included in the current study because it is original source for the Korean Bioethics regulation. The HIPAA-regulated data elements exist to protect patients' private and confidential information. Since human biological samples are subject to the HIPAA rule, it is meaningful to assess whether or not the requested data elements are HIPAA indicators or not. The non-HIPAA data elements were then further grouped into demographics (questions 44-53), general health conditions (questions 54-60), lab findings (questions 61-67), and sample-specific elements (questions 68-90). In addition, the study did not intend to exhaustively cover the entire spectrum of data elements required for sample annotation, rather to survey core annotation elements based on the existing sources such as the caTISSUE Core and Korean Frozen Lung Tissue Ban system. The sample-specific questions were also analysed into sub-activities of repositories, such as collection (questions 68, 70, 78-79 &amp;84), processing and storage (questions 69 &amp; 80-82), annotation (questions 71-77), and distribution (questions 86-90).</p>

<h3>Data analysis</h3>

<p>The study used three data analyses including descriptive analysis, cross-tabulation, and the TwoStep Cluster analysis for the research questions. SPSS (version 15) statistical software was used for the statistical analyses. For research question 1, the study used descriptive statistics to characterize the participants based on the survey data collected. Cross-tabulation was also performed to compare different groups of participants including the medical doctor group (N=11, 8%), pathology specialists (N=33, 24.1%), and tissue bankers (N=37, 27%) for their bio-samples and data requirements. The TwoStep Cluster analysis was used to assess whether there were any natural groups of the participants whose sample and data requirements were distinct from one another. The TwoStep Cluster method was chosen because the algorithm was built to handle mixed data types (continuous, binary, categorical, etc.) (SPSS Inc., 2008). Both continuous and categorical variables (or attributes) were used to calculate similarity (or dissimilarity) to identify any natural groups. The first step was to pre-cluster the cases (or records) into many small sub-clusters. The desired number of clusters resulting from the pre-cluster step were determined if specific numbers were not defined.</p>

<h2>Results</h2>

<h4>General characteristics of the respondents (research question 1)</h4>

<p>The study sought to describe the survey respondents by assessing general demographic characteristics, work or research-related characteristics, bio-sample use and requirement-related characteristics, and data requirement characteristics. 137 respondents who replied to the survey between July 19, 2007 and October 19, 2007 were included in the study. Of these, 114 (83.2%) were human sample users and 89 (65%) were animal sample users. Sixty-six respondents used both human and animal samples. Ninety-seven participants (71%) were and 28 (21%) were female. Ten people (8%) did not answer. Eleven participants (8%) had medical doctorates were and 33 (c.24%) identified themselves as pathology laboratory medicine specialists. Thirty-seven participants (27%) answered that they had experience working at a bio-repository. </p>

<p>The sample was not equally distributed among medical doctorates, pathology specialists and tissue bankers, which may have an impact on the study results. The most prevalent age group found in this survey was between 26 and 35, representing relatively junior researchers. 'Years of work experience' also showed that the survey respondents were new entrants to the field of biomedical sciences. Abridged descriptive survey results are shown in Table 2.</p>

<table width="70%" border="1" cellspacing="0" cellpadding="3" align="center" style="border-right: #99f5fb solid; border-top: #99f5fb solid; font-size: smaller; border-left: #99f5fb solid; border-bottom: #99f5fb solid; font-style: normal; font-family: verdana, geneva, arial, helvetica, sans-serif; background-color: #fdffdd">

<caption align="bottom"><br /><strong>Table 2: Descriptive characteristics of survey participants</strong><br />
[Note: Asterisk (*) in the survey question column denotes survey questions that allowed for multiple answers, therefore the frequency and the percentage of each answer category is based on the total number (N=137).]</caption> 

<tr>
<th width="77">Measure</th><th width="143">Result</th><th width="81">Freq.&nbsp;&nbsp;(%)</th><th width="104">Measure</th><th width="159">Result</th><th width="72">Freq.&nbsp;&nbsp;(%)</th> </tr>
 <tr>
<td width="77" rowspan="7">* Specialty (question 6)</td><td width="143" valign="top">Molecular biology</td><td width="81" valign="top" align="center">14&nbsp;&nbsp;(10.2)</td>
<td width="104" valign="top">Human sample (question 7)</td>
<td width="159" valign="top" align="center">Yes</td>
<td width="72" valign="top" align="center">114&nbsp;&nbsp;(83.2)</td>
 </tr>
 <tr><td width="143" valign="top">Genetics</td>
<td width="81" valign="top" align="center">26&nbsp;&nbsp;(19.0)</td>
<td width="104" valign="top">Animal sample use (question 8)</td>
<td width="159" valign="top" align="center">Yes</td>
<td width="72" valign="top" align="center">89&nbsp;&nbsp;(65)</td>
 </tr>
 <tr>
<td width="143" valign="top">Pathology</td>
<td width="81" valign="top" align="center">33&nbsp;&nbsp;(24.1)</td>
<td width="104" valign="top">Repository use (question9)</td>
<td width="159" valign="top" align="center">Yes</td>
<td width="72" valign="top" align="center">44&nbsp;&nbsp;(32.1)</td>
 </tr>
 <tr>
<td width="143" valign="top">Biochemistry</td>
<td width="81" valign="top" align="center">12&nbsp;&nbsp;(8.8)</td>
<td width="104" valign="top">Repository work (question 10)</td>
<td width="159" valign="top" align="center">Yes</td>
<td width="72" valign="top" align="center">37&nbsp;&nbsp;(27)</td>
 </tr>
 <tr>

<td width="143" valign="top">Public health</td>
<td width="81" valign="top" align="center">54&nbsp;&nbsp;(39.4)</td>
<td width="104" rowspan="6">* Types of repository work (question 11)</td>
<td width="159" valign="top">Sample collection</td>
<td width="72" valign="top" align="center">28&nbsp;&nbsp;(20.4)</td>
 </tr>
 <tr>
<td width="143" valign="top">Biostatistics</td>
<td width="81" valign="top" align="center">5&nbsp;&nbsp;(3.6)</td>
<td width="159" valign="top">Sample processing</td>
<td width="72" valign="top" align="center">20&nbsp;&nbsp;(14.6)</td>
 </tr>
 <tr>
<td width="143" valign="top">Health administration</td>
<td width="81" valign="top" align="center">1&nbsp;&nbsp;(0.7)</td>
<td width="159" valign="top">Sample storage</td>
<td width="72" valign="top" align="center">27&nbsp;&nbsp;(19.7)</td>
 </tr>
 <tr>
<td width="77" rowspan="7">* Diseases (question 22)</td>
<td width="143" valign="top">Liver and digestive system</td>
<td width="81" valign="top" align="center">27&nbsp;&nbsp;(19.7)</td>
<td width="159" valign="top">Sample distribution</td>
<td width="72" valign="top" align="center">13&nbsp;&nbsp;(9.5)</td>
 </tr>
 <tr>
<td width="143" valign="top">Musculoskeletal &amp; connective tissue</td>
<td width="81" valign="top" align="center">17&nbsp;&nbsp;(12.4)</td>
<td width="159" valign="top">Information management</td>
<td width="72" valign="top" align="center">17&nbsp;&nbsp;(12.4)</td>
 </tr>
 <tr>
<td width="143" valign="top">Neoplasms (lung, breast, ovary)</td>
<td width="81" valign="top" align="center">26&nbsp;&nbsp;(19)</td>
<td width="159" valign="top">Ethical, legal, administration</td>
<td width="72" valign="top" align="center">12&nbsp;&nbsp;(8.8)</td>
 </tr>
 <tr>
<td width="143" valign="top">Cardiac-related</td>
<td width="81" valign="top" align="center">19&nbsp;&nbsp;(13.9)</td>
<td width="104" rowspan="4">* Sample types (question 12)
</td>
<td width="159" valign="top">Molecular</td>
<td width="72" valign="top" align="center">89&nbsp;&nbsp;(65)</td>
 </tr>
 <tr>
<td width="143" valign="top">Infectious and parasitic</td>
<td width="81" valign="top" align="center">32&nbsp;&nbsp;(23.4)</td>
<td width="159" valign="top">Cell</td>
<td width="72" valign="top" align="center">62&nbsp;&nbsp;(45.2)</td>
 </tr>
 <tr>
<td width="143" valign="top">Diabetes and endocrine-related</td>
<td width="81" valign="top" align="center">19&nbsp;&nbsp;(13.9)</td>
<td width="159" valign="top">Fluid</td>
<td width="72" valign="top" align="center">80&nbsp;&nbsp;(58.4)</td>
 </tr>
 <tr>
<td width="143" valign="top">Others</td>
<td width="81" valign="top" align="center">52&nbsp;&nbsp;(38)</td>
<td width="159" valign="top">Tissue</td>
<td width="72" valign="top" align="center">72&nbsp;&nbsp;(52.6)</td>
 </tr>
 <tr>
<td width="77" rowspan="11">Data elements required</td>
<td width="143" valign="top">HIPAA elements (questions 25-43)</td>
<td width="81" valign="top" align="center">2.04&nbsp;&nbsp;(1.28)</td>
<td width="104" rowspan="11">* Sample requesting criteria (question 21)</td>
<td width="159" valign="top">By anatomic sites</td>
<td width="72" valign="top" align="center">61&nbsp;&nbsp;(44.5)</td>
 </tr>
 <tr>
<td width="143" valign="top">Demographics (question 44-53)</td>
<td width="81" valign="top" align="center">2.99&nbsp;&nbsp;(1.37)</td>
<td width="159" valign="top">Either by normal or diseased</td>
<td width="72" valign="top" align="center">76&nbsp;&nbsp;(55.5)</td>
 </tr>
 <tr>
<td width="143" valign="top">General health conditions (questions 54-60)</td>
<td width="81" valign="top" align="center">3.63&nbsp;&nbsp;(1.35)</td>
<td width="159" valign="top">By diseases</td>
<td width="72" valign="top" align="center">75&nbsp;&nbsp;(54.7)</td>
 </tr>
 <tr>
<td width="143" valign="top">Laboratory findings (questions 61-67)</td>
<td width="81" valign="top" align="center">3.58&nbsp;&nbsp;(1.31)</td>
<td width="159" valign="top">By primary or metastatistic</td>
<td width="72" valign="top" align="center">44&nbsp;&nbsp;(32.1)</td>
 </tr>
 <tr>
<td width="143" valign="top">Sample-specific elements (questions 68-90)</td>
<td width="81" valign="top" align="center">3.75&nbsp;&nbsp;(1.24)</td>
<td width="159" valign="top">By matched normal from the same patient</td>
<td width="72" valign="top" align="center">35&nbsp;&nbsp;(25.5)</td>
 </tr>
 <tr>
<td width="143" valign="top">Sample collection  (questions 68,70,78-79 &amp; 84)</td>
<td width="81" valign="top" align="center">3.83&nbsp;&nbsp;(1.26)</td>
<td width="159" valign="top">By specific treatments on patients</td>
<td width="72" valign="top" align="center">30&nbsp;&nbsp;(21.9)</td>
 </tr>
 <tr>
<td width="143" valign="top">Processing &amp; storage (questions 69 &amp; 80-82)</td>
<td width="81" valign="top" align="center">3.60&nbsp;&nbsp;(1.25)</td>
<td width="159" valign="top">By tissue sources</td>
<td width="72" valign="top" align="center">19&nbsp;&nbsp;(13.9)</td>
 </tr>
 <tr>
<td width="143" valign="top">Sample annotation (questions 71-77)</td>
<td width="81" valign="top" align="center">3.86&nbsp;&nbsp;(1.27)</td>
<td width="159" valign="top">By demographic condition</td>
<td width="72" valign="top" align="center">36&nbsp;&nbsp;(26.3)</td>
 </tr>
 <tr>
<td width="143" valign="top">Sample distribution (questions 86-90)</td>
<td width="81" valign="top" align="center">3.59&nbsp;&nbsp;(1.18)</td>
<td width="159" valign="top">By amount of tissue</td>
<td width="72" valign="top" align="center">38&nbsp;&nbsp;(27.7)</td>
 </tr>
 <tr>
<td width="143" valign="top">&nbsp;</td>
<td width="81" valign="top">&nbsp;</td>
<td width="159" valign="top">By total number of specimens needed</td>
<td width="72" valign="top" align="center">31&nbsp;&nbsp;(22.6)</td>
 </tr>
 <tr>
<td width="143" valign="top">&nbsp;</td>
<td width="81" valign="top">&nbsp;</td>
<td width="159" valign="top">By specimen preparation and preservation method</td>
<td width="72" valign="top" align="center">47&nbsp;&nbsp;(34.3)</td>
 </tr>
</table> 

<p>In terms of work and research related characteristics, the most reported anatomic sites of interests included lymphatic and immune related (N=45, 32.85%), endocrine (N=29, 21.17%), and stem cells and reproductive organs (N=29, 21.7%). In addition, the respondents replied that the most interesting research focus of the diseases were infectious and parasitic-related (N=32, 23.4%), followed by neoplasms (N=26, 19%). The survey respondents listed eighty two unique journal titles as potential publication places and 115 unique professional and academic conferences implying the respondents' involvement in specialized research areas. </p>

<p>The study also asked about sample specific requirements. With slightly over a decade of bio-repository experience in Korea, over 32 percent of the participants (N=44) answered that they used bio-repositories to acquire bio-samples. The majority of the participants also answered that more than one sample type was required for their individual project. Highly demanded types of bio-samples included RNA, cryopreserved cells, serum/whole blood, and frozen samples. In addition, PCR and RT-PCR were identified as the most required sample processing methods. The majority of the respondents also answered that they would like to have the sample stored in fresh-frozen, frozen, and paraffin blocks. Questions regarding sample requesting criteria identified that the most prevalent search criteria included search by anatomic sites, either by normal or diseased, or by disease names. These findings are consistent with previous studies (<a href="#kim07">Kim and Gilbertson 2007</a>; <a href="#kim08">Kim and Rasmussen 2008</a>).</p>

<p>Questions asked about data requirements were further analysed according to Health Insurance Portability and Accountability Act-restricted data elements, demographic data elements, general health condition data elements, laboratory finding elements, and sample specific data elements. The study found that the personally identifiable information that is under restricted distribution by the HIPAA and the Korean Bioethics and Biosafety Law were not frequently requested data elements by the respondents. More importantly, sample specific data such as sample diagnosis, processing, storage medium and condition, were highly demanded by the respondents (Avg=3.75, Std=1.24). These data indicate that researchers require more sample-specific data when they request bio-samples rather than personally identifiable data elements. Laboratory and physiology findings were also considered important data followed by general health condition and patient health history. As shown in the 2003 National Biospecimen Network survey, more than a single source was identified by the respondents as required data to be accompanied with bio-samples (<a href="#fri03">Friede <em>et al.</em> 2003</a>). This implies that frequently required data are complex in nature and cannot be extracted from a single medical information system.</p>

<h4>Distinct characteristics of the respondents by their work roles (research question 2)</h4>

<p>The study intended to characterize participants by their work roles representing medical doctors, pathology specialists, and tissue bankers. The categorization of work roles was not mutually exclusive; therefore, one person could belong to more than one work-related category. These categories were then used to further characterize the study participants in terms of bio-sample and data requirements.</p>

<table width="70%" border="1" cellspacing="0" cellpadding="3" align="center" style="border-right: #99f5fb solid; border-top: #99f5fb solid; font-size: smaller; border-left: #99f5fb solid; border-bottom: #99f5fb solid; font-style: normal; font-family: verdana, geneva, arial, helvetica, sans-serif; background-color: #fdffdd">

<caption align="bottom"><br /><strong>Table 3: Respondents by Sample Requirements</strong><br />
[# symbol in the sample requirement column denotes the results including the most frequently found items in the variables.]</caption> 

 <tr><th width="167">Samples<br /> requirements</th><th width="84">Medical doctor <br />(N=11)</th><th width="77">Non-medical doctor <br />(N=126)</th><th width="80">Pathology<br /> (N=33)</th><th width="87">Non-pathology <br />(N=104)</th><th width="80">Tissue banker<br /> (N=37)</th><th width="83">Non-tissue banker<br /> (N=100)</th></tr>

 <tr><th width="167" valign="top">&nbsp;</th><th width="84" valign="top">Freq.&nbsp;&nbsp;(%)</th><th width="77" valign="top">Freq.&nbsp;&nbsp;(%)</th><th width="80" valign="top">Freq.&nbsp;&nbsp;(%)</th><th width="87" valign="top">Freq.&nbsp;&nbsp;(%)</th><th width="80" valign="top">Freq.&nbsp;&nbsp;(%)</th><th width="83" valign="top">Freq.&nbsp;&nbsp;(%)</th></tr>

 <tr><td width="167" valign="top">Human sample use</td><td width="84" valign="top" align="center">11&nbsp;&nbsp;(100)</td><td width="77" valign="top" align="center">103&nbsp;&nbsp;(75.2)</td><td width="80" valign="top" align="center">33&nbsp;&nbsp;(100)</td><td width="87" valign="top" align="center">81&nbsp;&nbsp;(77.9)</td><td width="80" valign="top" align="center">34&nbsp;&nbsp;(91.9)</td><td width="83" valign="top" align="center">80&nbsp;&nbsp;(70.2)</td> </tr>

 <tr>
<td width="167" valign="top">Animal sample use</td><td align="center" width="84" valign="top">8&nbsp;&nbsp;(81.8)</td><td align="center" width="77" valign="top">81&nbsp;&nbsp;(81.7)</td><td align="center" width="80" valign="top">16&nbsp;&nbsp;(48.5)</td><td align="center" width="87" valign="top">73&nbsp;&nbsp;(70.2)</td><td align="center" width="80" valign="top">24&nbsp;&nbsp;(64.9)</td><td align="center" width="83" valign="top">65&nbsp;&nbsp;(73)</td></tr>

 <tr><td width="167" valign="top">Repository used</td><td align="center" width="84" valign="top">5&nbsp;&nbsp;(45.5)</td><td align="center" width="77" valign="top">88&nbsp;&nbsp;(69.8)</td><td align="center" width="80" valign="top">11&nbsp;&nbsp;(33.3)</td><td align="center" width="87" valign="top">33&nbsp;&nbsp;(31.7)</td><td align="center" width="80" valign="top">16&nbsp;&nbsp;(43.2)</td><td align="center" width="83" valign="top">28&nbsp;&nbsp;(63.6)</td></tr>

 <tr><td width="167" valign="top"># Requesting sources:<br />-Pathology department<br />-Research laboratory<br /></td><td align="center" width="84" valign="top"><br />6&nbsp;&nbsp;(54.5)<br />1&nbsp;&nbsp;(9.1)<br /></td><td align="center" width="77" valign="top"><br />45&nbsp;&nbsp;(35.7)<br />47&nbsp;&nbsp;(37.3)</td><td align="center" width="80" valign="top"><br />21&nbsp;&nbsp;(63.6)<br />5&nbsp;&nbsp;(15.2)</td><td align="center" width="87" valign="top"><br />30&nbsp;&nbsp;(28.8)<br />43&nbsp;&nbsp;(41.3)</td><td align="center" width="80" valign="top"><br />12&nbsp;&nbsp;(32.4)<br />7&nbsp;&nbsp;(18.9)</td><td align="center" width="83" valign="top"><br />39&nbsp;&nbsp;(39)<br />41&nbsp;&nbsp;(41)</td> </tr>

 <tr><td width="167" valign="top"># Sample Type:<br />-RNA<br />-Cryopreserved cells<br />-Serum/whole blood<br />-Frozen</td><td align="center" width="84" valign="top"><br />9&nbsp;&nbsp;(81.8)<br />5&nbsp;&nbsp;(45.5)<br />5&nbsp;&nbsp;(45.5)<br />8&nbsp;&nbsp;(72.7)</td><td align="center" width="77" valign="top"><br />80&nbsp;&nbsp;(63.5)<br />57&nbsp;&nbsp;(63.5)<br />75&nbsp;&nbsp;(59.5)<br />64&nbsp;&nbsp;(50.8)<br /></td><td align="center" width="80" valign="top"><br />21&nbsp;&nbsp;(63.6)<br />14&nbsp;&nbsp;(42.4)<br />15&nbsp;&nbsp;(45.5)<br />20&nbsp;&nbsp;(60.6)<br /></td><td align="center" width="87" valign="top"><br />68&nbsp;&nbsp;(65.4) <br />50&nbsp;&nbsp;(48.1)<br />65&nbsp;&nbsp;(62.5)<br />52&nbsp;&nbsp;(50)<br /></td><td align="center" width="80" valign="top"><br />25&nbsp;&nbsp;(67.6)<br />24&nbsp;&nbsp;(64.9)<br />21&nbsp;&nbsp;(56.8)<br />22&nbsp;&nbsp;(59.5)<br /></td><td align="center" width="83" valign="top"><br />64&nbsp;&nbsp;(64)<br />41&nbsp;&nbsp;(41)<br />59&nbsp;&nbsp;(59)<br />50&nbsp;&nbsp;(50)<br /></td></tr>

 <tr>
<td width="167" valign="top"># Sample processing:<br />-PCR<br />-RT-PCR</td>
<td align="center" width="84" valign="top"><br />8&nbsp;&nbsp;(72.7) <br />7&nbsp;&nbsp;(63.6)</td>
<td align="center" width="77" valign="top"><br />92&nbsp;&nbsp;(73) <br />91&nbsp;&nbsp;(72.2)</td>
<td align="center" width="80" valign="top"><br />25&nbsp;&nbsp;(75.8) <br />22&nbsp;&nbsp;(66.7)</td>
<td align="center" width="87" valign="top"><br />75&nbsp;&nbsp;(72.1) <br />76&nbsp;&nbsp;(73.1)</td>
<td align="center" width="80" valign="top"><br />27&nbsp;&nbsp;(73) <br />30&nbsp;&nbsp;(81.1)</td>
<td align="center" width="83" valign="top"><br />73&nbsp;&nbsp;(73)<br />68&nbsp;&nbsp;(68)</td>
 </tr>
 <tr>
<td width="167" valign="top"># Sample storage:<br />-Fresh-frozen<br />-Frozen<br />-Paraffin</td>
<td align="center" width="84" valign="top"><br />8&nbsp;&nbsp;(72.7)<br />5&nbsp;&nbsp;(45.5)<br />8&nbsp;&nbsp;(72.7)</td>
<td align="center" width="77" valign="top"><br />59&nbsp;&nbsp;(46.8)<br />66&nbsp;&nbsp;(52.4)<br />42&nbsp;&nbsp;(33.3)</td>
<td align="center" width="80" valign="top"><br />17&nbsp;&nbsp;(51.5)<br />16&nbsp;&nbsp;(48.5)<br />17&nbsp;&nbsp;(51.5)</td>
<td align="center" width="87" valign="top"><br />50&nbsp;&nbsp;(48.1)<br />55&nbsp;&nbsp;(52.9)<br />33&nbsp;&nbsp;(31.7)</td>
<td align="center" width="80" valign="top"><br />20&nbsp;&nbsp;(54.1)<br />19&nbsp;&nbsp;(51.4)<br />16&nbsp;&nbsp;(43.2)</td><td align="center" width="83" valign="top"><br />47&nbsp;&nbsp;(47)<br />52&nbsp;&nbsp;(52)<br />34&nbsp;&nbsp;(34)</td>
 </tr>
 
 <tr>
<td width="167" valign="top">Amount required</td><td align="center" width="84" valign="top">22g/y</td><td align="center" width="77" valign="top">50g/y</td><td align="center" width="80" valign="top">15g/y</td><td align="center" width="87" valign="top">58g/y</td><td align="center" width="80" valign="top">15g/y</td><td align="center" width="83" valign="top">59g/y</td>
 </tr>
 
 <tr>
<td width="167" valign="top">Frequency required</td>
<td align="center" width="84" valign="top">2.18/y</td>
<td align="center" width="77" valign="top">29/y</td>
<td align="center" width="80" valign="top">30.68/y</td>
<td align="center" width="87" valign="top">26.16/</td>
<td align="center" width="80" valign="top">44.1/y</td>
<td align="center" width="83" valign="top">21.02/y</td>
 </tr>
 
 <tr>
<td width="167" valign="top">Price willing to pay</td>
<td align="center" width="84" valign="top">$430</td>
<td align="center" width="77" valign="top">$500</td>
<td align="center" width="80" valign="top">$342</td>
<td align="center" width="87" valign="top">$542</td>
<td align="center" width="80" valign="top">$534</td>
<td align="center" width="83" valign="top">$478</td>
 </tr>
 
 <tr>
<td width="167" valign="top"># Anatomic sites:<br />-Immune systems related<br />-Respiratory</td>
<td align="center" width="84" valign="top"><br />0<br />4&nbsp;&nbsp;(36.4)</td>
<td align="center" width="77" valign="top"><br />45&nbsp;&nbsp;(35.7)<br />9&nbsp;&nbsp;(7.1)</td>
<td align="center" width="80" valign="top"><br />5&nbsp;&nbsp;(15.2)<br />6&nbsp;&nbsp;(18.2)</td>
<td align="center" width="87" valign="top"><br />40&nbsp;&nbsp;(38.5)<br />7&nbsp;&nbsp;(6.7)</td>
<td align="center" width="80" valign="top"><br />6&nbsp;&nbsp;(16.2)<br />3&nbsp;&nbsp;(8.1)</td>
<td align="center" width="83" valign="top"><br />39&nbsp;&nbsp;(39)<br />10&nbsp;&nbsp;(10)</td>
 </tr>

 <tr>
<td width="167" valign="top"># Diseases of interest:<br />-Liver/digestive<br />-Immunology-related<br />-Cancer-related</td>
<td align="center" width="84" valign="top"><br />1&nbsp;&nbsp;(9.1)<br />1&nbsp;&nbsp;(9.1)<br />5&nbsp;&nbsp;(45.5)</td>
<td align="center" width="77" valign="top"><br />26&nbsp;&nbsp;(20.6)<br />31&nbsp;&nbsp;(24.6)<br />21&nbsp;&nbsp;(16.7)</td>
<td align="center" width="80" valign="top"><br />3&nbsp;&nbsp;(9.1)<br />4&nbsp;&nbsp;(12.1)<br />8&nbsp;&nbsp;(24.2)</td>
<td align="center" width="87" valign="top"><br />24&nbsp;&nbsp;(23.1)<br />28&nbsp;&nbsp;(26.9)<br />18&nbsp;&nbsp;(17.3)</td>
<td align="center" width="80" valign="top"><br />7&nbsp;&nbsp;(18.9)<br />8&nbsp;&nbsp;(21.6)<br />9&nbsp;&nbsp;(24.3)</td>
<td align="center" width="83" valign="top"><br />20&nbsp;&nbsp;(20)<br />24&nbsp;&nbsp;(24)<br />17&nbsp;&nbsp;(17)</td>
 </tr>

 <tr>
<td width="167" valign="top">Requesting criteria:<br />-By anatomic sites<br />-Normal or diseased<br />-By disease<br />
-Primary or metastatic<br />-Matched normal<br />-By treatment<br />-Tissue sources<br />-By demographic<br />-By available amounts<br />-By available numbers<br />-By sample preparation</td>
<td align="center" width="84" valign="top"><br />7&nbsp;&nbsp;(63.6)<br />5&nbsp;&nbsp;(45.5)<br />7&nbsp;&nbsp;(63.6)<br />6&nbsp;&nbsp;(54.5)<br />4&nbsp;&nbsp;(36.4)<br />4&nbsp;&nbsp;(36.4)<br />2&nbsp;&nbsp;(18.2)<br />4&nbsp;&nbsp;(36.4)<br />4&nbsp;&nbsp;(36.4)<br />5&nbsp;&nbsp;(45.5)<br />6&nbsp;&nbsp;(54.5)</td>
<td align="center" width="77" valign="top"><br />54&nbsp;&nbsp;(42.9)<br />71&nbsp;&nbsp;(56.3)<br />68&nbsp;&nbsp;(54)<br />38&nbsp;&nbsp;(30.2)<br />31&nbsp;&nbsp;(24.6)<br />26&nbsp;&nbsp;(20.6)<br />17&nbsp;&nbsp;(13.5)<br />32&nbsp;&nbsp;(25.4)<br />34&nbsp;&nbsp;(27)<br />26&nbsp;&nbsp;(20.6)<br />41&nbsp;&nbsp;(32.5)</td>
<td align="center" width="80" valign="top"><br />14&nbsp;&nbsp;(42.4)<br />16&nbsp;&nbsp;(48.5)<br />18&nbsp;&nbsp;(54.5)<br />14&nbsp;&nbsp;(42.4)<br />7&nbsp;&nbsp;(21.2)<br />9&nbsp;&nbsp;(27.3)<br />2&nbsp;&nbsp;(6.1)<br />9&nbsp;&nbsp;(27.3)<br />6&nbsp;&nbsp;(18.2)<br />6&nbsp;&nbsp;(18.2)<br />11&nbsp;&nbsp;(33.3)</td>
<td align="center" width="87" valign="top"><br />47&nbsp;&nbsp;(45.2)<br />60&nbsp;&nbsp;(57.7)<br />57&nbsp;&nbsp;(54.8)<br />30&nbsp;&nbsp;(28.8)<br />28&nbsp;&nbsp;(26.9)<br />21&nbsp;&nbsp;(20.3)<br />17&nbsp;&nbsp;(16.3)<br />27&nbsp;&nbsp;(26)<br />32&nbsp;&nbsp;(30.8)<br />25&nbsp;&nbsp;(24)<br />36&nbsp;&nbsp;(34.6)</td>
<td align="center" width="80" valign="top"><br />16&nbsp;&nbsp;(43.2)<br />20&nbsp;&nbsp;(54.1)<br />22&nbsp;&nbsp;(59.5)<br />15&nbsp;&nbsp;(40.5)<br />10&nbsp;&nbsp;(27)<br />9&nbsp;&nbsp;(24.3)<br />6&nbsp;&nbsp;(16.2)<br />13&nbsp;&nbsp;(35.1)<br />10&nbsp;&nbsp;(27)<br />11&nbsp;&nbsp;(29.7)<br />16&nbsp;&nbsp;(43.2)</td>
<td align="center" width="83" valign="top"><br />45&nbsp;&nbsp;(45)<br />56&nbsp;&nbsp;(56)<br />53&nbsp;&nbsp;(53)<br />29&nbsp;&nbsp;(29)<br />25&nbsp;&nbsp;(25)<br />21&nbsp;&nbsp;(21)<br />13&nbsp;&nbsp;(13)<br />23&nbsp;&nbsp;(23)<br />28&nbsp;&nbsp;(28)<br />20&nbsp;&nbsp;(20)<br />31&nbsp;&nbsp;(31)</td>
 </tr>

 <tr>
<td width="167" valign="top">Journals<br />-Clinical<br />-Basic science<br />-Unknown</td>
<td align="center" width="84" valign="top"><br />6&nbsp;&nbsp;(54.5)<br />3&nbsp;&nbsp;(27.3)<br />4&nbsp;&nbsp;(36.4)</td>
<td align="center" width="77" valign="top"><br />37&nbsp;&nbsp;(29.4)<br />50&nbsp;&nbsp;(39.7)<br />58&nbsp;&nbsp;(46)<br /></td>
<td align="center" width="80" valign="top"><br />11&nbsp;&nbsp;(33.3)<br />7&nbsp;&nbsp;(21.2)<br />17&nbsp;&nbsp;(51.5)<br /></td>
<td align="center" width="87" valign="top"><br />32&nbsp;&nbsp;(30.8)<br />46&nbsp;&nbsp;(44.2)<br />45&nbsp;&nbsp;(43.3)</td>
<td align="center" width="80" valign="top"><br />16&nbsp;&nbsp;(43.2)<br />15&nbsp;&nbsp;(40.5)<br />14&nbsp;&nbsp;(37.8)</td>
<td align="center" width="83" valign="top"><br />27&nbsp;&nbsp;(27)<br />38&nbsp;&nbsp;(38)<br />48&nbsp;&nbsp;(48)</td>
 </tr>
 <tr>
<td width="167" valign="top">Professional associations<br />-Clinical<br />-Basic science<br />-Unknown</td>
<td align="center" width="84" valign="top"><br /> 7&nbsp;&nbsp;(63.6)<br />2&nbsp;&nbsp;(18.2)<br />2&nbsp;&nbsp;(18.2)</td>
<td align="center" width="77" valign="top"><br /> 37&nbsp;&nbsp;(29.4)<br />72&nbsp;&nbsp;(57.1)<br />29&nbsp;&nbsp;(23)</td>
<td align="center" width="80" valign="top"><br />15&nbsp;&nbsp;(45.5)<br />7&nbsp;&nbsp;(21.2)<br />8&nbsp;&nbsp;(24.2)</td>
<td align="center" width="87" valign="top"><br />29&nbsp;&nbsp;(27.9)<br />67&nbsp;&nbsp;(64.4)<br />23&nbsp;&nbsp;(22.1)</td>
<td align="center" width="80" valign="top"><br />18&nbsp;&nbsp;(48.6)<br />13&nbsp;&nbsp;(35.1)<br />8&nbsp;&nbsp;(21.6)</td>
<td align="center" width="83" valign="top"><br />26&nbsp;&nbsp;(26)<br />61&nbsp;&nbsp;(61)<br />23&nbsp;&nbsp;(23)</td>
 </tr>
</table>

<p>As shown in Table 3, all medical doctor and pathology respondents (100%) replied that they used human tissues, while slightly over 70 percent of other groups used human samples. Compared to other groups, fewer pathology respondents used animal sample users (N=16, 48.5%). More pathology and medical doctor respondents identified that they requested bio-samples through pathology rather than other bio-sample sources. This implies that easier accessibility to traditional bio-repository such as pathology is given to clinical groups compared to non-clinical groups. For sample processing and sample type, a majority of medical doctors answered that RNA (N=9, 81.8%) and frozen samples (N=8, 72.7%) were the most demanded samples, while other groups demanded other types of samples equally. It is also interesting to note that non-medical doctors, non-pathology, and non-tissue bankers reported that they required a larger amount of human samples and were less willing to pay for use of human samples compared to other groups. Cancer-related research was the most frequent disease of interest identified by the medical doctor group (N=5, 45.5%). More clinically oriented journals were identified as potential publication places by medical doctors compared to other groups (N=6, 54.5%). This result was consistent in clinically oriented conferences and associations that medical doctors and pathology groups normally attend.</p>

<p>Data requirements by different groups are shown in Table 3. The highest data requirement score reported for the Health Insurance Portability and Accountability Act-related data was demanded by non-tissue bankers (Mean=2.07, Standard deviation=1.31) followed by non-pathology (Mean=2.06, Standard deviation=1.29) and non-medical doctors (Mean=1.99, Standard deviation=1.26). More general demographics about patients were reported as important data for medical doctors (Mean=3.06, Standard deviation=1.43) compared to other groups. Important scores of data requirements by different groups were distinct in health history elements. Medical doctors (Mean=3.97, Standard deviation=1.28), pathology (Mean=3.83, Standard deviation=1.27), and tissue bankers (Mean=3.90, Standard deviation=1.19) were more likely to require health history than other groups. For laboratory findings, there were slightly higher scores reported by tissue bankers (Mean=3.66, Standard deviation=1.17), however, the difference was not significant. For sample specific requirements, medical doctors (Mean=3.89, Standard deviation=1.28), pathology (Mean=3.75, Standard deviation=1.22), and tissue bankers (Mean=3.80, Standard deviation=1.18) scored slightly more than other groups.</p>

<table width="70%" border="1" cellspacing="0" cellpadding="3" align="center" style="border-right: #99f5fb solid; border-top: #99f5fb solid; font-size: smaller; border-left: #99f5fb solid; border-bottom: #99f5fb solid; font-style: normal; font-family: verdana, geneva, arial, helvetica, sans-serif; background-color: #fdffdd">
<caption align="bottom"><br /><strong>Table 4: Respondents by data requirements</strong></caption>

 <tr>
<th width="120">Respondents' data requirements</th>
<th width="80">Medical doctor<br /> Mean (SDev.)</th>
<th width="67">Non-medical doctor <br />Mean (SDev.)</th>
<th width="83">Pathology <br />Mean (SDev.)</th>
<th width="94">Non-pathology <br />Mean (SDev.)</th>
<th width="84">Tissue banker <br />Mean (SDev.)</th>
<th width="84">Non-tissue banker <br />Mean (SDev.)</th>
 </tr>
 <tr>
<td width="120" valign="top">HIPAA</td>
<td align="center" width="80" valign="bottom">1.74&nbsp;&nbsp;(1.34)</td>
<td align="center" width="67" valign="bottom">1.99&nbsp;&nbsp;(1.26 )</td>
<td align="center" width="83" valign="bottom">1.67&nbsp;&nbsp;(1.12)</td>
<td align="center" width="94" valign="bottom">2.06&nbsp;&nbsp;(1.29)</td>
<td align="center" width="84" valign="bottom">1.70&nbsp;&nbsp;(1.07)</td>
<td align="center" width="84" valign="bottom">2.07&nbsp;&nbsp;(1.31)</td>
 </tr>
 <tr>
<td width="120" valign="top">Non-HIPAA</td>
<td align="center" width="80" valign="bottom">3.68&nbsp;&nbsp;(1.28)</td>
<td align="center" width="67" valign="bottom">3.53&nbsp;&nbsp;(1.29)</td>
<td align="center" width="83" valign="bottom">3.57&nbsp;&nbsp;(1.26)</td>
<td align="center" width="94" valign="bottom">3.53&nbsp;&nbsp;(1.29)</td>
<td align="center" width="84" valign="bottom">3.61&nbsp;&nbsp;(1.22)</td>
<td align="center" width="84" valign="bottom">3.51&nbsp;&nbsp;(1.31)</td>
 </tr>
 <tr>
<td width="120" valign="top">Demographics</td>
<td align="center" width="80" valign="bottom">3.06&nbsp;&nbsp;(1.43)</td>
<td align="center" width="67" valign="bottom">2.98&nbsp;&nbsp;(1.36)</td>
<td align="center" width="83" valign="bottom">2.91&nbsp;&nbsp;(1.40)</td>
<td align="center" width="94" valign="bottom">3.02&nbsp;&nbsp;(1.35)</td>
<td align="center" width="84" valign="bottom">2.93&nbsp;&nbsp;(1.37)</td>
<td align="center" width="84" valign="bottom">3.01&nbsp;&nbsp;(1.36)</td>
 </tr>
 <tr>
<td width="120" valign="top">Non demographic</td>
<td align="center" width="80" valign="bottom">3.18&nbsp;&nbsp;(1.28)</td>
<td align="center" width="67" valign="bottom">3.14&nbsp;&nbsp;(1.26)</td>
<td align="center" width="83" valign="bottom">3.08&nbsp;&nbsp;(1.19)</td>
<td align="center" width="94" valign="bottom">3.15&nbsp;&nbsp;(1.28)</td>
<td align="center" width="84" valign="bottom">3.11&nbsp;&nbsp;(1.15)</td>
<td align="center" width="84" valign="bottom">3.14&nbsp;&nbsp;(1.30)</td>
 </tr>
 <tr>
<td width="120" valign="top">Health history</td>
<td align="center" width="80" valign="bottom">3.97&nbsp;&nbsp;(1.28)</td>
<td align="center" width="67" valign="bottom">3.60&nbsp;&nbsp;(1.34)</td>
<td align="center" width="83" valign="bottom">3.83&nbsp;&nbsp;(1.27)</td>
<td align="center" width="94" valign="bottom">3.57&nbsp;&nbsp;(1.36)</td>
<td align="center" width="84" valign="bottom">3.90&nbsp;&nbsp;(1.19)</td>
<td align="center" width="84" valign="bottom">3.53&nbsp;&nbsp;(1.38)</td>
 </tr>
 <tr>
<td width="120" valign="top">Non-health history</td>
<td align="center" width="80" valign="bottom">3.05&nbsp;&nbsp;(1.30)</td>
<td align="center" width="67" valign="bottom">3.05&nbsp;&nbsp;(1.27)</td>
<td align="center" width="83" valign="bottom">2.96&nbsp;&nbsp;(1.22)</td>
<td align="center" width="94" valign="bottom">3.08&nbsp;&nbsp;(1.28)</td>
<td align="center" width="84" valign="bottom">2.99&nbsp;&nbsp;(1.18)</td>
<td align="center" width="84" valign="bottom">3.07&nbsp;&nbsp;(1.30)</td>
 </tr>
 <tr>
<td width="120" valign="top">Laboratory findings</td>
<td align="center" width="80" valign="bottom">3.52&nbsp;&nbsp;(1.10)</td>
<td align="center" width="67" valign="bottom">3.58&nbsp;&nbsp;(1.31)</td>
<td align="center" width="83" valign="bottom">3.66&nbsp;&nbsp;(1.17)</td>
<td align="center" width="94" valign="bottom">3.55&nbsp;&nbsp;(1.32)</td>
<td align="center" width="84" valign="bottom">3.59&nbsp;&nbsp;(1.20)</td>
<td align="center" width="84" valign="bottom">3.58&nbsp;&nbsp;(1.33)</td>
 </tr>
 <tr>
<td width="120" valign="top">Non-laboratory finding</td>
<td align="center" width="80" valign="bottom">3.10&nbsp;&nbsp;(1.32)</td>
<td align="center" width="67" valign="bottom">3.05&nbsp;&nbsp;(1.27)</td>
<td align="center" width="83" valign="bottom">2.92&nbsp;&nbsp;(1.23)</td>
<td align="center" width="94" valign="bottom">3.03&nbsp;&nbsp;(1.29)</td>
<td align="center" width="84" valign="bottom">3.03&nbsp;&nbsp;(1.18)</td>
<td align="center" width="84" valign="bottom">3.07&nbsp;&nbsp;(1.31)</td>
 </tr>
 <tr>
<td width="120" valign="top">Sample specific</td>
<td align="center" width="80" valign="bottom">3.89&nbsp;&nbsp;(1.28)</td>
<td align="center" width="67" valign="bottom">3.71&nbsp;&nbsp;(1.23)</td>
<td align="center" width="83" valign="bottom">3.75&nbsp;&nbsp;(1.22)</td>
<td align="center" width="94" valign="bottom">3.72&nbsp;&nbsp;(1.24)</td>
<td align="center" width="84" valign="bottom">3.80&nbsp;&nbsp;(1.18)</td>
<td align="center" width="84" valign="bottom">3.70&nbsp;&nbsp;(1.26)</td>
 </tr>
 <tr>
<td width="120" valign="top">Not-sample specific</td>
<td align="center" width="80" valign="bottom">2.72&nbsp;&nbsp;(1.31)</td>
<td align="center" width="67" valign="bottom">2.76&nbsp;&nbsp;(1.30)</td>
<td align="center" width="83" valign="bottom">1.94&nbsp;&nbsp;(2.00)</td>
<td align="center" width="94" valign="bottom">2.05&nbsp;&nbsp;(1.65)</td>
<td align="center" width="84" valign="bottom">2.67&nbsp;&nbsp;(1.18)</td>
<td align="center" width="84" valign="bottom">2.79&nbsp;&nbsp;(1.34)</td>
</tr>
</table>

<h4>Natural groups of the participants (research question 3)</h4>

<p>The study assessed whether there are any natural groups of the participants whose bio-sample and data requirements are similar within groups. A complete summary of TwoStep Cluster results can be found in Table 5. TwoStep Cluster analysis resulted in two distinct groups including Cluster 1 (N=116) and Cluster 2 (N=21). Cluster 1 includes the majority of specialty groups (medical doctors, pathology, and tissue bankers), while Cluster 2 include more non-specialty groups (non-medical doctors, non-pathology, and non-tissue bankers). These data imply that the respondent category used in this study is helpful in grouping bio-sample users which can be implemented in modelling distinct work roles for bio-sample-related users.</p>

<table width="70%" border="1" cellspacing="0" cellpadding="3" align="center" style="border-right: #99f5fb solid; border-top: #99f5fb solid; font-size: smaller; border-left: #99f5fb solid; border-bottom: #99f5fb solid; font-style: normal; font-family: verdana, geneva, arial, helvetica, sans-serif; background-color: #fdffdd">

<caption align="bottom"><br /><strong>Table 5: TwoStep cluster analysis of respondents based on their data requirements</strong></caption>

 <tr>
  <th width="348" colspan="2" valign="top">Respondents' variables</th>
  <th width="132">Cluster 1  (N=116, 84.67%)</th>
  <th width="127">Cluster 2 (N=21, 15.33%)</th>
 </tr>
 <tr>
  <td width="119" rowspan="3">Respondent category</td>
  <td width="229" valign="top">Medical doctors (N=11)</td>
  <td width="132" valign="top" align="center">10&nbsp;&nbsp;(90.91)</td>
  <td width="127" valign="top" align="center">1&nbsp;&nbsp;(9.09)</td>
 </tr>
 <tr>
  <td width="229" valign="top">Pathology (N=33)</td>
  <td width="132" valign="top" align="center">30&nbsp;&nbsp;(90.91)</td>
  <td width="127" valign="top" align="center">3&nbsp;&nbsp;(9.09)</td>
 </tr>
 <tr>
  <td width="229" valign="top">Tissue bankers (N=37)</td>
  <td width="132" valign="top" align="center">33&nbsp;&nbsp;(89.19)</td>
  <td width="127" valign="top" align="center">4&nbsp;&nbsp;(10.81)</td>
 </tr>
 <tr>
<td width="119" rowspan="15">Sample requirements</td>
  <td width="229" valign="top">Human sample used</td>
  <td width="132" valign="top" align="center">101&nbsp;&nbsp;(88.59)</td>
  <td width="127" valign="top" align="center">13&nbsp;&nbsp;(61.9)</td>
</tr>
 <tr>
  <td width="229" valign="top">Animal sample used</td>
  <td width="132" valign="top" align="center">76&nbsp;&nbsp;(65.51)</td>
  <td width="127" valign="top" align="center">13&nbsp;&nbsp;(61.9)</td>
</tr>
 <tr>
 <td width="229" valign="top">Repository used</td>
<td width="132" valign="top" align="center">38&nbsp;&nbsp;(32.75)</td>
<td width="127" valign="top" align="center">6&nbsp;&nbsp;(28.57)</td>
</tr>
 <tr>
 <td width="229" valign="top"># Requesting sources:<br />  
 -Pathology<br />
-Other Labs</td>
<td width="132" valign="top" align="center"><br />
45&nbsp;&nbsp;(38.79)<br />
40&nbsp;&nbsp;(34.48)</td>
<td width="127" valign="top" align="center"><br />
6&nbsp;&nbsp;(28.57)<br />
8&nbsp;&nbsp;(38.10)</td>
</tr>
 <tr>
<td width="229" valign="top">Sample Type<br />
  -RNA<br />
  -Cryopreserved cells<br />
  -Serum or whole blood<br />
  -Frozen
  </td>

  <td width="132" valign="top" align="center"><br />
   77&nbsp;&nbsp;(66.38)<br />
  53&nbsp;&nbsp;(45.69)<br />
  70&nbsp;&nbsp;(60.34)<br />
  62&nbsp;&nbsp;(53.45)
  </td>

  <td width="127" valign="top" align="center"><br />
  12&nbsp;&nbsp;(57.14)<br />
  9&nbsp;&nbsp;(42.86)<br />
  10&nbsp;&nbsp;(47.62)<br />
  10&nbsp;&nbsp;(47.62)
  </td>
</tr>
 <tr>
  <td width="229" valign="top">Sample processing<br />
  -PCR<br />
  -RT-PCR</td>
<td width="132" valign="top" align="center"><br />
  85&nbsp;&nbsp;(73.28)<br />
  85&nbsp;&nbsp;(73.28)</td>
 <td width="127" valign="top" align="center"><br />   
  15&nbsp;&nbsp;(71.43)<br />
  13&nbsp;&nbsp;(61.9)</td>
</tr>
 <tr>
  <td width="229" valign="top">Sample storage<br />
  -Fresh-frozen<br />
  -Frozen<br />
 -Paraffin</td>
 <td width="132" valign="top" align="center"><br />
  60&nbsp;&nbsp;(52.59)<br />
  61&nbsp;&nbsp;(53.45)<br />
  44&nbsp;&nbsp;(37.93)</td>
  <td width="127" valign="top" align="center"><br />
  7&nbsp;&nbsp;(33.33)<br />
  10&nbsp;&nbsp;(47.62)<br />
  6&nbsp;&nbsp;(28.57)</td>
</tr>
 <tr>
  <td width="229" valign="top">Amount required</td>
  <td width="132" valign="top" align="center">54g/year</td>
  <td width="127" valign="top" align="center">11g/year</td>
</tr>
 <tr>
  <td width="229" valign="top">Frequency required</td>
 <td width="132" valign="top" align="center">25.89/year</td>
 <td width="127" valign="top" align="center">34.76/year</td>
</tr>
 <tr>
 <td width="229" valign="top">Price willing to pay</td>
 <td width="132" valign="top" align="center">$502</td>
<td width="127" valign="top" align="center">$448</td>
</tr>
 <tr>
 <td width="229" valign="top">Anatomic sites:<br />
  -Immune systems related<br />
  -Respiratory</td>
 <td width="132" valign="top" align="center"><br />
  38&nbsp;&nbsp;(32.75)<br />
  13&nbsp;&nbsp;(11.21)</td>
  <td width="127" valign="top" align="center"><br />
  7&nbsp;&nbsp;(33.33)<br />
  0</td>
</tr>
 <tr>
  <td width="229" valign="top">Diseases of interest:<br />
  -Liver or digestive system<br />
  -Immunology-related<br />
  -Cancer-related</td>
  <td width="132" valign="top" align="center"><br />
  53&nbsp;&nbsp;(45.69)<br />
  67&nbsp;&nbsp;(57.76)<br />
  65&nbsp;&nbsp;(56.03)  </td>
  <td width="127" valign="top" align="center"><br />
  8&nbsp;&nbsp;(38.10)<br />
  9&nbsp;&nbsp;(42.86)<br />
  10&nbsp;&nbsp;(47.62)  </td>
</tr>
 <tr>
  <td width="229" valign="top">Requesting criteria: <br />
  -By anatomic sites<br />
  -Normal or diseased<br />
  -By disease<br />
  -Primary or metastatic<br />
  -Matched normal<br />
  -By treatment<br />
  -Tissue sources<br />
  -By demographic<br />
  -By available amounts<br />
  -By available numbers<br />
  -By sample preparation</td>
  <td width="132" valign="top" align="center"><br />
   21&nbsp;&nbsp;(18.10)<br />
  13&nbsp;&nbsp;(11.21)<br />
  8&nbsp;&nbsp;(6.90)<br />
  13&nbsp;&nbsp;(11.21)<br />
  26&nbsp;&nbsp;(22.31)<br />
  22&nbsp;&nbsp;(18.97)<br />
  8&nbsp;&nbsp;(6.90)<br />
  17&nbsp;&nbsp;(14.66)<br />
  10&nbsp;&nbsp;(11.6)<br />
  7&nbsp;&nbsp;(5.56)<br />
  10&nbsp;&nbsp;(11.6)</td>

  <td width="127" valign="top" align="center"><br />
  6&nbsp;&nbsp;(28.57)<br />
  4&nbsp;&nbsp;(19.05)<br />
  1&nbsp;&nbsp;(4.76)<br />
  6&nbsp;&nbsp;(28.57)<br />
  6&nbsp;&nbsp;(28.57)<br />
  4&nbsp;&nbsp;(19.05)<br />
  2&nbsp;&nbsp;(9.52)<br />
  2&nbsp;&nbsp;(9.52)<br />
  2&nbsp;&nbsp;(9.52)<br />
  1&nbsp;&nbsp;(4.76)<br />
  3&nbsp;&nbsp;(14.29)</td>
</tr>
 <tr>
 <td width="229" valign="top">Journals<br />
-Clinical<br />
-Basic science<br />
-Unknown</td>
  <td width="132" valign="top" align="center"><br />
  37&nbsp;&nbsp;(31.90)<br />
  42&nbsp;&nbsp;(36.20)<br />
  54&nbsp;&nbsp;(46.55)</td>
  <td width="127" valign="top" align="center"><br />
  6&nbsp;&nbsp;(28.57)<br />
  11&nbsp;&nbsp;(52.38)<br />
  8&nbsp;&nbsp;(38.10)</td>
</tr>
 <tr>
  <td width="229" valign="top">Associations<br />  -Clinical<br />   -Basic Science<br />  -Unknown</td>
  <td width="132" valign="top" align="center"><br />
  37&nbsp;&nbsp;(31.90)<br />
  61&nbsp;&nbsp;(52.59)<br />
  27&nbsp;&nbsp;(23.28)</td>
 <td width="127" valign="top" align="center" align="center"><br />
  7&nbsp;&nbsp;(33.33)<br />
  13&nbsp;&nbsp;(61.9)<br />
  4&nbsp;&nbsp;(19.05)</td>
</tr>
 <tr>
  <td width="119" rowspan="10">Data requirements</td>
  <td width="229" valign="top">HIPAA</td>
  <td width="132" valign="top" align="center">1.92&nbsp;&nbsp;(1.18)</td>
  <td width="127" valign="top" align="center">1.97&nbsp;&nbsp;(1.26)</td>
 </tr>
 <tr>
  <td width="229" valign="top">Non-HIPAA</td>
  <td width="132" valign="top" align="center">3.75&nbsp;&nbsp;(1.09)</td>
  <td width="127" valign="top" align="center">3.48&nbsp;&nbsp;(1.31)</td>
 </tr>
 <tr>
  <td width="229" valign="top">Demographics</td>
  <td width="132" valign="top" align="center">3.17&nbsp;&nbsp;(1.26)</td>
  <td width="127" valign="top" align="center">2.99&nbsp;&nbsp;(1.36)</td>
 </tr>
 <tr>
  <td width="229" valign="top">Non-demographic</td>
  <td width="132" valign="top" align="center">3.44&nbsp;&nbsp;(1.07)</td>
  <td width="127" valign="top" align="center">3.23&nbsp;&nbsp;(1.28)</td>
 </tr>
 <tr>
  <td width="229" valign="top">Health history</td>
  <td width="132" valign="top" align="center">3.92&nbsp;&nbsp;(1.11)</td>
  <td width="127" valign="top" align="center">3.63&nbsp;&nbsp;(1.34)</td>
 </tr>
 <tr>
  <td width="229" valign="top">Non-health history</td>
  <td width="132" valign="top" align="center">3.25&nbsp;&nbsp;(1.10)</td>
  <td width="127" valign="top" align="center">3.07&nbsp;&nbsp;(1.29)</td>
 </tr>
 <tr>
  <td width="229" valign="top">Laboratory findings</td>
  <td width="132" valign="top" align="center">3.88&nbsp;&nbsp;(1.06)</td>
  <td width="127" valign="top" align="center">3.58&nbsp;&nbsp;(1.29)</td>
 </tr>
 <tr>
  <td width="229" valign="top">Non-laboratory finding</td>
  <td width="132" valign="top" align="center">3.26&nbsp;&nbsp;(1.12)</td>
  <td width="127" valign="top" align="center">3.08&nbsp;&nbsp;(1.30)</td>
 </tr>
 <tr>
  <td width="229" valign="top">Sample specific</td>
  <td width="132" valign="top" align="center">4.05&nbsp;&nbsp;(0.93)</td>
  <td width="127" valign="top" align="center">3.73&nbsp;&nbsp;(1.24)</td>
 </tr>
 <tr>
  <td width="229" valign="top">Not-sample specific</td>
  <td width="132" valign="top" align="center">3.22&nbsp;&nbsp;(1.15)</td>
  <td width="127" valign="top" align="center">3.04&nbsp;&nbsp;(1.31)</td>
 </tr>
</table>

<p>For use of bio-samples, a larger portion of bio-sample users were clustered in Cluster 1 (N=101, 88.59%) than Cluster 2 (N=13, 61.9%). A higher percentage of people in Cluster 1 (N=38, 32.75%) used bio-repositories compared to Cluster 2 (N=6, 28.57%). About 39 percent of Cluster 1 (N=45) requested bio-samples through pathology, while only less than 30 percent of Cluster 2 (N=6) requested bio-samples through pathology. Heavy bio-sample users requiring greater amounts and more frequently required bio-samples were clustered in Cluster 1 than Cluster 2. Cluster 1 indicated more requesting criteria when searching bio-samples than Cluster 2. These indicate less demands of the associated bio-sample data by Cluster 2 compared to Cluster 1. Distinct characteristics between clinical and non-clinical groups were also found in these two clusters. Cluster 1 is more likely clinically oriented compared to Cluster 2 whose publication places and professional associations (and conferences) were more likely basic science-oriented. </p>

<p>Some distinct characteristics between the two clusters were reported based on the importance scores of the various data elements given. For almost every data requirement, Cluster 2 scored higher than Cluster 1. Only the non-HIPAA-related data group was slightly more demanded by Cluster 2 (Avg=1.97, Std=1.26) compared to Cluster 1 (Avg=1.92, Std=1.18). Sample-specific data requirements (Avg=4.05, Std=0.93) were the most highly scored data requirement by Cluster 1 followed by health history (Avg=3.92, Std=1.11). The highest scored data requirement by Cluster 1 was sample specific (Avg=3.73, Std=1.24) data. Least scored elements by both Cluster 1 (Avg=1.92, Std=1.18) and Cluster 2 (Avg=1.97, Std=1.26) were HIPAA restricted data elements. Overall, Cluster 2 included active and heavy bio-sample users whose members were more clinically oriented compared to Cluster 1. This result supports previous research findings about two distinct bio-sample user groups including clinically oriented (Cluster 1) and basic science-oriented groups (Cluster 2).</p>

<h4>Development of an information seeking model of bio-sample-centric users (research question 4)</h4>

<p>The proposed conceptual framework of information requirements of bio-sample using researchers based on the current study findings and the reviews of the previous studies was developed. A basic framework to develop the current study model was adopted from Leckie's model (<a href="#lec96">1996</a>). A complete model is depicted in Figure 1. The basic components of the proposed model are the professional <em>work roles</em> and their <em>related tasks</em> performed by tissue users in their daily practice. In addition, interacting components of information seeking included characteristics of <em>data and bio-sample needs</em> and <em>factors affecting information seeking</em> (sources and awareness) which resulted in the <em>outcome</em> component. Individual boxes in Figure 1 represent each core component of the model. The model proposed here is not intended to be comprehensive but is a conceptual model that can be further tested and refined in a larger empirical study. The following section will discuss key components contained in the proposed framework.</p>

<div align="center"><img src="p427fig1.gif"></div>

<div align="center"><br /><strong>Figure 1: A proposed framework of information requirements of bio-sample-based users (Adopted from Leckie <em>et al.</em> 1996: 180)</strong></div>

<p>As found in various empirical studies, professionals work in complicated environments and play complex roles in regard to information seeking. The first box of Figure 1 lists five professional roles found in the current study and previous research: biomedical researchers, bio-repository technicians, clinical care practioners, administrators and supporters, and biomedical educators and students. The individual roles of tissue users are highly related to their professional tasks described in the <em>tasks</em> section in Figure 1. For instance, the main tasks of biomedical researchers are to design and conduct studies, collect and analyse data, report findings and plan for further studies. These roles are not different from any other researchers. bio-repository technicians, who were identified as tissue bankers in the current study, play important roles in collecting, processing, storing, retrieving, and distributing bio-samples and their related data which are also linked to quality assessment for the collected samples and data. Clinical practitioners who were identified as medical doctors and pathologists focus on clinical care; therefore, their professional roles are more relevant to prevention, diagnosis, treatment, prognosis and management of clinical care. Administrators and supporters are professionals whose roles focus on making managerial decisions, operating the bio-repositories, building external and internal networks, and improving efficacy, safety, minimizing risks and maximising benefits of the bio-repository. The student group is assigned in the last user role along with biomedical educators. The majority of the respondents to the current study were junior researchers who were still in training. Their bio-samples and the associated data requirements should not be neglected. </p>

<p>There are also other general characteristics of information needs that enter into the component of the proposed model. The study identified three distinct characteristics of information requirements: demographic and general characteristics, bio-sample requirements and data requirements. Demographic and general characteristics are components that describe individual users. The current study's survey questions used individual characteristics along with the questions asked about bio-sample specific requirements and data specific requirements. The data findings and bio-sample requirements listed in the third box of Figure 1 are the core of the information seeking model because these findings directly relate the information required to those who request the information. </p>

<p><em>Factors affecting information seeking</em> are listed in the fourth box in Figure 1. Numerous information seeking studies of professionals have sought to determine what factors affect information seeking behaviour.  In the current study, sources of bio-samples and data were identified as a major factor in users seeking information from a number of bio-sample and data sources including bio-repositories, pathology laboratories, personal laboratories, electronic medical records, surgical pathology reports and cancer registries. The study also included <em>awareness of bio-sample and data</em>, which may affect information seeking. The identified factors should be further explored to assess whether a certain factor might have influence on other factors (compounding influence on multiple factors).</p>

<p>In the last information seeking process, <em>outcome</em>, is the 'end point of the work-related requirements of specific roles and tasks' (<a href="#wil00">Wilson 2000</a>). The study identified measurements of bio-sample seeking and data seeking. These measures should be further developed so that quality assurance and successful data and bio-sample distribution may be scientifically measured. The number of sample distributions, satisfaction with data and bio-sample related services, organizational cost-benefit analysis, and the amount of derived funding and grants as well as the number of publications can be used as measures of the outcome. More importantly, feedback should be carefully conducted so that information seeking does not flow in one direction, but is an interactive process based on the identified needs of information seekers.</p>

<h2>Discussion </h2>

<p>The study was designed to describe the general characteristics of biomedical scientists who require human biological samples and associated data. The findings uncovered specific requirements of the physical bio-samples and the related data from multiple information systems. The results reported here are not directly indicative of what science, technology, and medical libraries should acquire, annotate, circulate and serve for their current clientele. Rather, this study should lead to the discussion of new types of information sources (e.g., human tissues and blood), the complex nature of the information seeking behaviour of biomedical scientists (and, in particular, those using bio-samples) and the expanded boundary of information service centres such as bio-repository libraries. The following discussion of the study findings highlights the data presented in the previous section. </p>

<p>First, the finding of distinct characteristics for different professional roles in seeking biological samples and associated data will be beneficial to the development of a framework for information seeking behaviour. In a previous study, Kim and Gilbertson found two distinct user groups described by their sample and data requirements. The current study also confirmed that there were distinct characteristics among two clusters of users. One group (C1) was more likely to be clinically-oriented than the other group (C2). For instance, the clinical group has easier access to human samples compared to the non-clinical group, so the sample requirements and requesting criteria reported are diverse and detailed. Not only the sample requirements but also the data requirements confirmed that the basic science group (C2) is less discriminating regarding their choice of data requesting criteria (e.g., by anatomic sites, normal and matched abnormal, and primary and metastatic samples) (<a href="#kim07">Kim and Gilbertson 2007</a>). This implies that there will be potential benefit for basic scientists to develop more elaborate sample-specific study variables if the data are accessible through a well-managed bio-repository database. Likewise, more elaborate efforts should be made to identify clinically relevant genomic variables to be recorded with human tissue samples.</p>

<p>Secondly, the current study found that tissue users were mostly junior investigators from multiple academic disciplines conducting various types of basic, developmental, translational, and clinical research and they were largely from academic medical centres whose interests in anatomic sites and diseases varied. This finding supports the growing trends towards biomedical research requiring human samples in both clinical and basic sciences among relatively new investigators, whose bio-sample requirements are to be supported through a systematic management of bio-repositories. The wide variety of sample and data requirements found in this study is a strong indicator of the necessity of a nationwide network to serve biomedical researchers with less accessibility to human biological samples. For instance, if a junior investigator wants to demonstrate the potential of biomarkers in the development of a cancer drug such as Trastuzumb, a recombinant monoclonal antibody in Asian populations, could benefit by accessing biomarker-based patient selection at an early stage in the clinical trial process, which could optimize the development of successful cancer therapy (<a href="#fri03">Friede <em>et al.</em> 2003</a>). Through a biological resource sharing network, researchers from non-clinical disciplines can also access valuable resources at lower cost.</p>

<p>Thirdly, the current study set out to discover complex and multidisciplinary biomedical researchers whose information seeking not only focuses on textual data but also on physical items that have not been given much attention by information scientists. The study learned that managing physical bio-samples is very different from managing conventional, textual information sources. For instance, various sample-specific requirements should be tracked, including sample location, condition, availability, history and significant events (such as sample thaws, loss, destruction and processing of any kind), as well as specimen distribution through a unique identifier which can be also linked to other established databases. Frequent updates are required for most of the sample-specific information, especially when the samples are dispensed. This means that physical objects, such as human tissues, are not expected to be returned for continued use. For this reason, some tissue banks recommend researchers deposit the study results (data generated using the distributed samples) rather than the physical tissues. So, the designer of the bio-repository system should anticipate the resulting scientific data to be stored, retrieved and redistributed for further use. In other words, once a library holding (e.g., a tissue sample) is circulated to a researcher, the library should plan for acquiring the end result as a reciprocal benefit. This will ensure that other researchers do not have to repeat a study with the same samples, if the study results are accessible to them. Designing information systems that manage biological samples and the multi-faceted biomedical data is not easy task for information scientists who are used to conventional bibliographic management systems. Although the findings are limited to the survey respondents, the results can be used as representative use cases with which to construct bio-sample research databases. </p>

<p>Most importantly, the study findings indicate that a standardized information set for sample collection, processing methods and storage conditions should be available to the user to aid in selecting specimens for testing. Along with the relevant biomedical data, which are to be retrieved from multiple information sources, all of the sample specific requirements ought to be accessible through a standardized data format. For instance, polymerase chain reaction amplification techniques require only a few nanograms, which can be served by a milligram of tissue left over from standardized diagnostic procedures. For further advanced techniques, such as specimens for whole-proteome analysis or microarrays for whole-genome analysis, it is advisable to develop a strategic plan to save tissue for future use when more advanced procedures become available. </p>

<p>Standardization is significant not only for physical samples, but also for relevant data. As the study findings suggest, the respondents extensively require various bio-sample-related data. As the International Society for Biological and Environmental Repositories Best Practice (<a href="#isb08">2008</a>) recommends, information accompanying biological samples should cover a varying degree of data which can be either linked to an external database or directly linked to the repository information system. For instance, information regarding specimen location, status, condition, collection, processing, storage and distribution was found to be relevant data to the study respondents. Considering the nature of human biological samples, additional information regarding patient-specific data such as age at the time of collection, sex, occupation, race or ethnicity, diagnosis, diagnostic procedure, types of treatment, surgical procedure, medical and family history, and other health behavioural information such as smoking and nutrition, physiological and clinical laboratory data, and availability of other biological samples would be beneficial to the biomedical researcher who might not consider combining clinical data with genomic data (single nucleotide polymorphisms, mutations, microarrays) and proteomic data (specific protein biomarkers, two-dimensional gel data, mass spectral analyses). Obviously, the information stored will vary depending on the nature, purpose, and intended uses for the biological collection; however, careful consideration should be given to what the repository information system should or can contain depending on who the potential users are. As discussed in numerous behaviour studies, information that is easily accessible is used more by scientists (<a href="#kim09">Kim 2009</a>). Therefore, the challenge for information scientists is to construct an effective and efficient information vehicle through which biomedical scientists can easily identify samples along with relevant data. </p>

<p>Lastly, the major contribution of the current study is the development of an information requirement model based on researchers use of biological samples and their related data requirements. The characteristics and variables developed in the current study can be used for future empirical studies assessing potential users, types of information required depending on work-related roles, factors affecting information seeking, and evaluation of information seeking effectiveness. The current study found that information defined in the tissue-related model is different from Leckie's generic model. Information required by these groups is not only a physical entity such as tissues, serum, plasma and molecular material, but also the entity's associated data extracted from various clinical and research databases such as electronic medical records, surgical pathology reports, cancer registry and genome analysis information systems. Likewise, the proposed model stresses the requirements of bio-samples and their associated data that make tissue users distinct from other professionals. Therefore, the growing trend toward the systematic management of bio-samples and relevant data, are to be carefully studied for effective resource sharing. </p>

<h2>Conclusion</h2>

<p>The study formed a framework to capture the information seeking behaviour of researchers using bio-samples. This framework introduces a greater understanding of the professional roles and tasks, which relates both directly and indirectly to the use of bio-samples and data. The findings show that the information requirements of bio-sample-related researchers are heavily influenced by the professional role-task relationship. Therefore, general factors such as researchers' demographics, professional career stages, or recurring needs should be carefully explored in relation to the role-task relationship for further study (<a href="#fid04">Fidel and Green 2004</a>; <a href="#kar09">Kari 2009</a>; <a href="#bel08">Bellazzi and Zupan 2008</a>). Moreover, a noticeable dearth of information has been reported in the area of predictive data mining in clinical medicine where limited use of genomic and proteomic findings were included to develop a clinical outcome prediction model (<a href="#gil04">Gilbertson <em>et al.</em> 2004</a>). </p>

<p>One of the limitations of the current study is that the concept of information seeking behaviour (Leckie's) was used to model a framework. Leckie's model was built to understand seeking and behaviour rather than information requirements in an early stage of system development. Although it is the closest and the most relevant information model of professionals, it may not be perfectly suited to the current study. So, the findings will be more beneficial to designing an information system than to understand the overall information-seeking behaviour of scientists (<a href="#meh03">Meho and Tibbo 2003</a>). Additionally, the survey was limited to Korean researchers and should be expanded to researchers other than Koreans so that the study results are more representative of other populations. </p>

<p>In conclusion, bio-sample resource sharing and distribution will not be successful unless both the physical integrity of the biomolecules and the value-added information associated with the sample are made accessible through a formal information requirement analysis (<a href="#sch02">Schilsky <em>et al.</em> 2002</a>; <a href="#hu04">Hu <em>et al.</em> 2004</a>; <a href="#han98">Hanson 1998</a>). In order to expand the roles of information professionals in science, technology, and medicine, the following must be clearly understood in the emerging genetic era: Who needs what information why and when, and in order to accomplish what tasks?</p>

<h2>Acknowledgements</h2> 
 
<p>This project was supported in part by a faculty research grant by college of communication and information studies, University of Kentucky. In addition, this publication was made possibly by Grant Number P20RR-16481 from the National Centre for Research Resources, a component of the National Institutes of Health. Its contents are solely the responsibility of the authors and do not necessarily represent the official views of National Centre for Research Resources or National Institutes of Health.</p>

<h2>About the author</h2>

<p>Dr. Sujin Kim, is an assistant professor in the School of Library and Information Science and is jointly appointed to the Department of Pathology and Laboratory Medicine, University of Kentucky, Lexington, Kentucky, USA. She was trained in the pathology and oncology informatics group at the University of Pittsburgh, USA, working with microscopic images and human bio-sample repositories. Her specialty is biomedical informatics and health science librarianship. She can be contacted at <a href="mailto:sujinkim@uky.edu" target="_blank">sujinkim@uky.edu</a>.</p>
 
<form target="_blank" onsubmit="return window.confirm(&quot;You are submitting information to an external page. \nAre you sure?&quot;);"> 
<fieldset> 
<legend style="color:white;background-color:#5E96FD;font-size:medium;padding:.1ex .5ex;border-right:1px solid navy;border-bottom:1px solid navy;font-weight:bold">References</legend> 

<ul> 
<li><a name="alv03"></a>Alvarez, I., Saldias, M., Wodowoz, O., Campos, H., Machin, D., Silva, W. et al. (2003). Progress of National Multi-tissue Bank in Uruguay in the International Atomic Energy Agency (IAEA) Tissue Banking Programme. <i>Cell Tissue Bank</i>, <b>4</b>(2-4), 173-178.</li>

<li><a name="bec00"></a>Becich, M.J. (2000). The role of the pathologist as tissue refiner and data miner: the impact of functional genomics on the modern pathology laboratory and the critical roles of pathology informatics and bioinformatics. <i>Molecular Diagnosis</i>, <b>5</b>(4) 287-299.</li>

<li><a name="bel08"></a>Bellazzi, R., &amp; Zupan, B. (2008). Predictive data mining in clinical medicine: current issues and guidelines. <i>International Journal of Medical Informatics</i>, <b>77</b>(2), 81-97.</li>

<li><a name="ber03"></a>Berman, J.J., Edgerton, M.E., &amp; Friedman, B.A. (2003). <a href="http://www.webcitation.org/5oHXiQD8P">The tissue microarray data exchange specification: a community-based, open source tool for sharing tissue microarray data.</a> <i>BMC Medical Informatics and Decision Making</i>, <b>3</b>(5).Retrieved August 15 2008, from http://bit.ly/fVWHJ (Archived by WebCite&reg; at  http://www.webcitation.org/5oHXiQD8P)</li>

<li><a name="cap08"></a> College of American Pathologists. (2008). <a href="http://www.webcitation.org/5oG2aaoQU"><em>Cancer protocols and checklists</em>. </a> Retrieved 15 March 2010, from http://www.cap.org/apps/docs/committees/cancer/cancer_protocols/2009/BreastDCIS_09protocol.pdf</a> (Archived by WebCite&reg; at http://www.webcitation.org/5oG2aaoQU)</li>

<li><a name="com05"></a>Compton, C. (2005). <em>NCI's efforts to develop best practices for bio-repositories.</em> Paper presented at the NCI Symposium: International Harmonization of bio-repository Practices, held November 10, 2005 in Washington, D.C.</li>

<li><a name="cov03"></a>Covitz, P.A., Hartel, F., Schaefer, C., De Coronado, S., Fragoso, G., Sahni, H., <em>et al.</em> (2003). caCORE: a common infrastructure for cancer informatics. <i>Bioinformatics</i>, <b>19</b>(18), 2404-2412.</li>

<li><a name="eis03"></a>Eiseman, E.B., Bloom, G., Brower, J., Clancy, N. &amp; Olmsted, S.S. (2003). <em>Case studies of existing human tissue repositories: &quot;best practices&quot; for a biospecimen resource for the genomic and proteomic era.</em> Santa Monica, CA: RAND Corporation.</li>

<li><a name="fid04"></a>Fidel, R. &amp; Green, M. (2004). The many faces of accessibility: engineers&#39; perception of information sources. <i>Information Processing and Management</i>, <b>40</b>(3), 563-581.</li>

<li><a name="fri03"></a>Friede, A., Grossman, R., Hunt, R., Li, R.M. &amp; Stern, S. (Eds.). (2003). <a href="http://www.webcitation.org/5oG2ewgit">National biospecimen network blueprint.</a> Durham, NC: Constella Group. Retrieved 15 March, 2010 from http://www.c-changetogether.org/pubs/pubs/FINAL_NBN_Blueprint.pdf  (Archived by WebCite&reg; at http://www.webcitation.org/5oG2ewgit)</li>

<li><a name="gaj02"></a>Gajiwala, A.L. (2002). Setting up a tissue bank in India: the Tata Memorial Hospital experience. <i>Cell Tissue Bank</i>, <b>4</b>(2-4), 193-201.</li>

<li><a name="gil04"></a>Gilbertson, J.R., Gupta, R., Nie, Y., Patel, A.A. &amp; Becich, M.J. (2004). Automated clinical annotation of tissue bank specimens. <i>Medinfo</i>, <b>11</b>(1), 607-610.</li>

<li><a name="gil04"></a>Gilder, M.R., Zhao, M, Temkin, J.M. &amp; Sarachan, B.D. (2004). BioPET: a comprehensive online database for elucidating and visualizing biological pathways, protein interactions, and microarray data. Transactions of the Integrated Biomedical Informatics &amp; Enabling Technologies Symposium Journal. 1: 124-130.</li>

<li><a name="goe05"></a>Goebell, P.J., Groshen, S., Schmitz-Drager, B.J., Sylvester, R., Kogevinas, M., Malats, N., <em>et al.</em> (2005). Concepts for banking tissue in urologic oncology: the International Bladder Cancer Bank. <i>Clinical Cancer Research</i>, <b>2</b>(1), 413-415.</li>

<li><a name="han98"></a>Hanson, T.A. &amp; Day, J.M. (Eds). (1998). <i>Managing the electronic library: a practical guide for information professionals</i>. London: Bowker-Saur.</li>

<li><a name="hu04"></a>Hu, H., Brzeski, H., Hutchins, J., Ramaraj, M., Qu, L., Xiong, R., <em>et al.</em> (2004). Biomedical informatics: development of a comprehensive data warehouse for clinical and genomic breast cancer research. <em>Pharmacogenomics</em>, <b>5</b>(7), 933-941.</li>

<li><a name="isb08"></a>International Society for Biological and Environmental Repositories. (2008). ISBER best practices for repositories I: collection, storage, and retrieval of human biological materials for research. <i>Cell Preservation Technology</i>, <b>6</b>(1), 33-37.</li>

<li><a name="kar09"></a>Kari, J. (2009). Informational uses of spiritual information: an analysis of messages reportedly transmitted by extraphysical means. <i>Journal of Information Science</i>, <b>35</b>(4), 453-468.</li>

<li><a name="kim07"></a>Kim, S. &amp; Gilbertson, J. (2007). Information requirements of cancer center researchers focusing on human biological samples and associated data. <em>Information Processing &amp; Management</em>, <b>43</b>(5), 1383-1401.</li>

<li><a name="kim08"></a>Kim, S. &amp; Rasmussen, E. (2008). Characteristics of tissue-centric biomedical researchers using a survey and cluster analysis. <i>Journal of American Society for Information Science and Technology</i>, <b>59</b>(8), 1210-1223.</li>

<li><a name="kim08"></a>Kim, S. (2008). Development of a human bio-repository information system at the University of Kentucky Markey CancerCenter. In <em>Proceedings of the International Conference on BioMedical Engineering and Informatics: May 27-30, 2008,  Sanya, Hainan, China.</em> (pp. 621-625). Los Alamitos, CA: IEEE Computer Society.</li>

<li><a name="lec96"></a>Leckie, G.J., Pettigrew, K.E. &amp; Sylvain, C. (1996). Modeling the information seeking of professionals: a general model derived from research on engineers, health care professionals, and lawyers. <i>Library Quarterly</i>, <b>66</b>(2), 161-193</li>

<li><a name="lee06"></a>Lee, H.W., Park, Y.R., Sim, J., Park, R.W., Kim, W.H. &amp; Kim, J.H. (2006). The tissue microarray object model: a data model for storage, analysis, and exchange of tissue microarray experimental data. <i>Archives of Pathology &amp; Laboratory Medicine</i>, <b>130</b>(7), 1004-1013.</li>

<li><a name="man01"></a>Manley, S., Mucci, N.R., DeMarzo, A.M. &amp; Rubin, M.A. (2001). Relational database structure to manage high-density tissue microarray data and images for pathology studies focusing on clinical outcome: the prostate specialized program of research excellence model. <i>The American Journal of Cardiovascular Pathology</i>, <b>159</b>(3), 837-843.</li>

<li><a name="mcs08"></a>McSherry, T.C. &amp; Paul, K. (2008). <i><a href="http://www.webcitation.org/5oG2kJwE0"> caTISSUE core end user manual</a></i> . Retrieved 15 March, 2010 from http://gforge.nci.nih.gov/docman/view.php/18/2371/caTissue%20Core%20User%20Manual%20DRAFT.pdf (Archived by WebCite&reg; at http://www.webcitation.org/5oG2kJwE0)</li>

<li><a name="meh03"></a>Meho, L.L. &amp; Tibbo, H.R. (2003). Modeling the information seeking behaviour of social scientists: Ellis&#39; study revisited. <i>Journal of the American Society for Information Science and Technology</i>, <b>54</b>(6), 570-587.</li>

<li><a name="mer00"></a>Mericka, P. (2000). Brief history of the tissue bank, Charles University hospital, Hradec Kralove, Czech Republic. <i>Cell Tissue Bank</i>, <b>1</b>(1), 17-25.</li>

<li><a name="mit02"></a>Mitchell, P. (2002). UK launches ambitious tissue data bank project. <i>Nature Biotechnology</i>, <b>20</b>(6), 529.</li>

<li><a name="nci08"></a> National Cancer Institute. <em>Cancer Diagnosis Program</em>. (2008). <a href="http://www.webcitation.org/5oG2ulcuk" >Finding human tissues for research.</a> Bethesda, MD: National Cancer Institute. Retrieved 15 March, 2010 from http://www.cancerdiagnosis.nci.nih.gov/specimens/finding.htm  (Archived by WebCite&reg; at  http://www.webcitation.org/5oG2ulcuk)</li>

<li><a name="nci06"></a>National Institutes of Health. <em>Department of Health and Human Services.</em> (2006). <a href="http://www.webcitation.org/5oG6qdE86">First-generation guidelines for NCI-supported bio-repositories.</a>
<i> Federal Register</i>, <strong>71</strong>(82), 25184-25185. </i> Retrieved 15 March, 2010 from http://edocket.access.gpo.gov/2006/pdf/06-4000.pdf    (Archived by WebCite&reg; at http://www.webcitation.org/5oG6qdE86)</li>

<li><a name="orr70"></a>Orr, R.H. (1970). The scientist as an information processor: a conceptual model illustrated with data on variables related to library utilization. In C.E. Nelson and P. Donald, (Eds.). <i>Communication among scientists and engineers</i> (pp. 143-189). Lexington, MA: Heath Lexington Books.</li>

<li><a name="pat06"></a>Patel, A.A., Gilbertson, J.R., Parwani, A.V., Dhir, R., Datta, M.W., Gupta, R., <em>et al.</em> (2006). <a href="http://www.webcitation.org/5oH01rX9A">An informatics model for tissue banks: lessons learned from the Cooperative Prostate Cancer Tissue Resource</a>. <i>BMC Cancer</i>, <strong>6</strong>, paper 120. Retrieved 15 March, 2010 from http://www.biomedcentral.com/1471-2407/6/120  (Archived by WebCite&reg; at http://www.webcitation.org/5oH01rX9A)</li>

<li><a name="sch02"></a>Schilsky, R.L., Dressler, L.M., Bucci, D., Monovich, L., Jewell, S., Suster, S. et al. (2002). Cooperative group tissue banks as research resources: the cancer and leukemia group B tissue repositories. <i>Clinical Cancer Research</i>, <b>8</b>(5), 943-948.</li>

<li><a name="sps08"></a>SPSS Inc. (2008). <i>SPSS v.14 help manual</i>. Chicago, IL: SPSS Inc.</li>

<li><a name="tet05"></a>Tettamanti, M., Tralamazza, S., Berati, M., Molteni, M. &amp; Gamba, N. (2005) Human research tissue banks: the ATRA Project for establishing a human research tissue bank in Switzerland. <i>Alternatives to Laboratory Animals: ATLA</i>, <b>33</b>(1), 29-36.</li>

<li><a name="tob06"></a>Tobias, J., Chilukuri, R., Komatsoulis, G.A., Mohanty, S., Sioutos, N., Warzel, D.B., <em>et al.</em> (2006). The CAP cancer protocols: a case study of caCORE based data standards implementation to integrate with the Cancer Biomedical Informatics Grid. <i>BMC Medical Informatics and Decision Making</i>, <b>6</b>(1) 25.</li>

<li><a name="wil00"></a>Wilson, T.D. (2000). <a href="http://www.webcitation.org/5oG327k3O">Human information behaviour.</a>  <i>Informing Science</i>, <b>3</b>(2), 49-56. Retrieved May 15 2009 from http://inform.nu/Articles/Vol3/v3n2p49-56.pdf </a> (Archived by WebCite&reg; at http://www.webcitation.org/5oG327k3O)</a></li>
 </ul> 
</fieldset> 
</form> 
 <br />
<br />




 
<form target="_blank" onsubmit="return window.confirm(&quot;You are submitting information to an external page. \nAre you sure?&quot;);"> 
<fieldset> 
<legend style="color:white;background-color:#5E96FD;font-size:medium;padding:.1ex .5ex;border-right:1px solid navy;border-bottom:1px solid navy;font-weight:bold">How to cite this paper</legend> 
<div><br /> 

Kim, S. (2010). &quot;A conceptual framework of information requirements for scientists using human biological samples.&quot; <em>Information Research</em>, <strong>15</strong>(1) paper 427. [Available at http://InformationR.net/ir/15-1/paper427.html]</div> 
</fieldset> 
</form> 
 
<table cellspacing="10" align="center"> 
	<tr> 
		<td colspan="3" align="center" style="background-color:#5E96FD;color:white;font-family:verdana;font-size:small;font-weight:bold">Find other papers on this subject</td></tr> 
	<tr> 
		<td align="center" valign="top"> 
 

<form method="get" action="http://scholar.google.com/scholar" target="_blank" onsubmit="return window.confirm(&quot;You are submitting information to an external page. \nAre you sure?&quot;);"> 
			<table bgcolor="#ffffff"> 
				<tr> 
					<td nowrap valign="top" align="center" height="32"><input type="hidden" name="q" size="31" maxlength="255" value="biological samples bio-samples information seeking"> <br /> 
<input type="submit" name="sa" value="Scholar Search" style="font-size:small;font-family:Verdana;font-weight:bold"> 
<input type="hidden" name="num" value="100"> 
					</td> 
				</tr> 
			</table> 
 
</form> 
		</td> 
		<td align="center" valign="top"> 
 
 
<form method="get" action="http://www.google.com/custom" target="_blank" onsubmit="return window.confirm(&quot;You are submitting information to an external page. \nAre you sure?&quot;);"> 
			<table bgcolor="#ffffff"> 
				<tr> 
					<td nowrap valign="top" align="center" height="32"><input type="hidden" name="q" size="31" maxlength="255" value="biological samples bio-samples information seeking"><br /> 

<input type="submit" name="sa" value="Google Search" style="font-family:Verdana;font-weight:bold;font-size:small"> 
<input type="hidden" name="client" value="pub-5081678983212084"> 
<input type="hidden" name="forid" value="1"> 
 
<input type="hidden" name="ie" value="ISO-8859-1"> 
<input type="hidden" name="oe" value="ISO-8859-1"> 
<input type="hidden" name="cof" value="GALT:#0066CC;GL:1;DIV:#999999;VLC:336633;AH:center;BGC:FFFFFF;LBGC:FF9900;ALC:0066CC;LC:0066CC;T:000000;GFNT:666666;GIMP:666666;FORID:1;"> 
<input type="hidden" name="hl" value="en"> 
					</td> 
				</tr> 
			</table> 
</form> 
</td> 
<td align="center" valign="top"> 
<form method="get" action="http://www.live.com/" target="_blank" onsubmit="return window.confirm(&quot;You are submitting information to an external page. \nAre you sure?&quot;);"> 
			<table bgcolor="#ffffff"> 
				<tr> 
 
					<td nowrap valign="top" align="center" height="32"><input type="hidden" name="q" size="31" maxlength="255" value="biological samples bio-samples information seeking"> <br /> 

<input type="submit" name="sa" value="Bing" style="font-size:small;font-family:Verdana;font-weight:bold"> 
<input type="hidden" name="num" value="100"> 
					</td> 
				</tr> 
			</table> 
</form> 
 
		</td> 
	</tr> 
</table> 
 
 
<div align="center"> 
<img src="http://images.del.icio.us/static/img/delicious.small.gif" alt="logo"> <a href="http://del.icio.us/post" target="_blank">Bookmark This Page</a> 
</div> 
<hr size="1" style="color:#5E96FD"> 

<h2><a id="appA" name="appA" />Appendix A: Survey Questionnaire (Translated into English)</h2>

<p><strong>Information</strong></p>

<p>You are invited to participate in a study about human biological repositories. Advances in genetic studies have led to significant uses of human biological samples. In this study about 100 individuals who are 18 and older will be asked to complete an online survey. The purpose of this study is to assess information requirements of human biological samples. </p>

<p><strong>Benefits</strong></p>

<p>This study will help tissue-centric users and information professionals including science, technology, and medical librarians as well as system developer alike understand specific requirements of tissue-centric users.</p>

<p><strong>Potential risks</strong></p>

<p>There are no foreseeable risks associated with this study. However, if you feel uncomfortable answering the survey questions, you may choose to skip a question or withdraw from the study at any time.</p>

<p><strong>Confidentiality</strong></p>

<p>The information collected form this survey will be accessible only to the researcher (PI: Sujin Kim). No personal identifiers will be used, and information will be presented in aggregate form, text delimited file only accessible to the researcher. No individual machine IP will be obtained. Random numbers will be assigned to obtain the aggregated data once the survey is submitted to the secure server. The survey will be collected on a server with SSL (Secure Sockets Layer) capabilities, which is one of the best providers of Internet security available, but there is always a risk that a third party may intercept the survey answers. </p>

<p><strong>Contact</strong></p>

<p>If you have questions, you may contact the principal investigator, Sujin Kim, at the University of Kentucky, School of Library and Information Science, 518 King Library, Lexington, KY, 40506, or <a href="mailto:sujinkim@uky.edu" target="_blank">sujinkim@uky.edu</a>. If you have any questions about your rights as a volunteer in this research, contact XXXXX, Research Compliance Officer (<a href="mailto:XXXX@uky.edu" target="_blank">XXXX@uky.edu</a>) in the Office of Research Integrity at the University of Kentucky at XXX-XXX-XXXX or toll free at 1-800-XXX-XXXX. </p>

<p><strong>Participation</strong></p>

<p>Your participation in this study is voluntary; you may refuse to participate without penalty. If you withdraw from the study before data collection is completed, your data will be destroyed.</p>

<p><strong>Consent</strong></p>

I have read this form, am 18 years of age or older, and agree to take part in this study. 

<p>____ Yes </p>
<p>____ No</p>

<hr style="color: Navy;" size="3" />

<h2>GENERAL BACKGROUND</h2>

<p>1. Year of birth (e.g., 1950):    ____________  </p>                      

<p>2. Gender:  a. Male ____ b. Female ____</p>

<p>3. Highest degree earned: a. College student b. BS c. MS d. Ph.D. e. MD. F. Other _______</p>

<p>4. Your current institution:</p>
<p style="text-indent:11.0pt">a. Government</p> 
<p style="text-indent:11.0pt">b. Non-profit research center</p> 
<p style="text-indent:11.0pt">c. University hospital</p> 
<p style="text-indent:11.0pt">d. Industry-sponsored research institution</p>
<p style="text-indent:11.0pt">e. University-based research center</p>
<p style="text-indent:11.0pt">f. Other: ____________________</p>
<p>5. Years of work experience: __________years</p>
<p>6. Specialty: </p>
<p style="text-indent:11.25pt">a. Molecular biology</p>
<p style="text-indent:11.25pt">b. Genetics</p>
<p style="text-indent:11.25pt">c. Pathology</p>
<p style="text-indent:11.25pt">d. Biochemistry</p>
<p style="text-indent:11.25pt">e. Public health</p>
<p style="text-indent:11.25pt">f. Biostatistics</p>
<p style="text-indent:11.25pt">g. Administration</p>
<p style="text-indent:11.25pt">h. Other________________________</p>
<p>7. Do you require human biological samples?  Yes ______  No _______</p>
<p>8. Do you require animal and plant samples?  Yes ______  No _______</p>
<p>9. Have you ever used human bio-repositories?  Yes ______  No _______</p>
<p>10. Have you ever worked in human bio-repositories?  Yes ______  No _______</p>
<p>11. Types of bio-repository work (please answer if you say 'Yes' in question 10.)</p>
<p style="text-indent:11.25pt">a. Sample collection</p> 
<p style="text-indent:11.25pt">b. Sample processing</p>
<p style="text-indent:11.25pt">c. Sample storage</p>
<p style="text-indent:11.25pt">d. Sample distribution</p>
<p style="text-indent:11.25pt">e. Data management</p>
<p style="text-indent:11.25pt">f. Ethnical, legal, and procedural issue management</p>
<p style="text-indent:11.25pt">g. Other _________________________</p>
<p>12. What type(s) of samples do you require for your work?</p>
<p>1) Molecular</p>

<table width="40%" border="0" cellspacing="0" cellpadding="3" style="margin-left: 2%;">
<tr><td>a. cDNA/DNA</td><td>&nbsp;</td><td>e. RNA, cytoplasmic</td></tr>
<tr><td>b. Not specified</td><td>&nbsp;</td><td>f.	RNA, nuclear</td></tr>
<tr><td>c. Protein</td><td>&nbsp;</td><td>g. RNA, poly-A enriched</td></tr>
<tr><td>d. RNA</td><td>&nbsp;</td><td>&nbsp;</td></tr>
</table>

<p>2) Cell </p>

<table width="40%" border="0" cellspacing="0" cellpadding="3" style="margin-left: 2%;">
<tr><td>a. Cryopreserved cells</td><td>&nbsp;</td><td>c. Frozen cell block</td></tr>
<tr><td>b. Fixed cell block</td><td>&nbsp;</td><td>d. Frozen cell pellet</td></tr>
</table>

<p>3) Fluid </p>

<table width="40%" border="0" cellspacing="0" cellpadding="3" style="margin-left: 2%;">
<tr><td>a. Feces</td><td>&nbsp;</td><td>k. Body cavity fluid</td></tr>
<tr><td>b. Sweat</td><td>&nbsp;</td><td>l. Milk</td></tr>
<tr><td>c. Synovial fluid</td><td>&nbsp;</td><td>m. Pericardial fluid</td></tr>
<tr><td>d. Bile</td><td>&nbsp;</td><td>n. Lavage</td></tr>
<tr><td>e. Cerebrospinal fluid</td><td>&nbsp;</td><td>o. Whole blood</td></tr>
<tr><td>f. Amniotic fluid</td><td>&nbsp;</td><td>p. Vitreous fluid</td></tr>
<tr><td>g. Serum</td><td>&nbsp;</td><td>q. Gastric fluid</td></tr>
<tr><td>h. Whole bone marrow</td><td>&nbsp;</td><td>r. Bone marrow plasma</td></tr>
<tr><td>i. Saliva</td><td>&nbsp;</td><td>s. Urine</td></tr>
<tr><td>j. Plasma</td><td>&nbsp;</td><td>&nbsp;</td></tr>
</table>

<p>4) Tissue </p>

<table width="40%" border="0" cellspacing="0" cellpadding="3" style="margin-left: 2%;">
<tr>
<td>a. Frozen tissue</td><td>&nbsp;</td><td>e. Fixed tissue slide</td></tr>
<tr><td>b. Microdissected</td><td>&nbsp;</td><td>f. Frozen tissue block</td></tr>
<tr><td>c. Fresh tissue </td><td>&nbsp;</td><td>g. Fixed tissue block</td></tr>
<tr><td>d. Frozen tissue slide</td><td>&nbsp;</td><td>h. Fixed tissue</td></tr>
</table>

<p>5) Other: ______________________________</p>

 <p>13. What processing techniques do you require for your samples?</p>

<table width="40%" border="0" cellspacing="0" cellpadding="3" style="margin-left: 2%;">
<tr><td>a. PCR</td><td>&nbsp;</td><td>j. IHC</td></tr>
<tr><td>b. FISH</td><td>&nbsp;</td><td>k. Mass spec.</td></tr>
<tr><td>c. CGH</td><td>&nbsp;</td><td>l. Westerns</td></tr>
<tr><td>d. Sequencing</td><td>&nbsp;</td><td>m. 1D/2D Gels</td></tr>
<tr><td>e. ENPs</td><td>&nbsp;</td><td>n. Ultra structure</td></tr>
<tr><td>f. Microarrays</td><td>&nbsp;</td><td>o. Microscopy (Light and EM)</td></tr>
<tr><td>g. RT-PCR</td><td>&nbsp;</td><td>p. Subcellular localization</td></tr>
<tr><td>h. Northern</td><td>&nbsp;</td><td>q. Other _________________</td></tr>
<tr><td>i. In situ hybridization</td><td>&nbsp;</td><td>&nbsp;</td></tr>
</table>

<p>14. What are the storage format(s) in which you would like to keep your samples?</p>

<table width="40%" border="0" cellspacing="0" cellpadding="3" style="margin-left: 2%;">
<tr><td>a. Formalin</td><td>&nbsp;</td><td>e. Paraffin</td></tr>
<tr><td>b. Fresh-frozen</td><td>&nbsp;</td><td>f. Vials</td></tr>
<tr><td>c. Frozen </td><td>&nbsp;</td><td>g. Other _________________</td></tr>
<tr><td>d. OCT </td><td>&nbsp;</td><td>&nbsp;</td></tr>
</table>

 
15. What is your estimate of the amounts of samples you need for one study?
<p style="text-indent:22.0pt">Amount: ________________   Unit of measurement: _____________</p>
<p>16. How often do you require samples? ______________times/year</p>
<p>17. Where do you normally seek bio-samples?</p>

<table width="40%" border="0" cellspacing="0" cellpadding="3" style="margin-left: 2%;">
<tr><td>a. Pathology department in university hospitals</td><td>&nbsp;</td><td>e. Academic or non-profit research laboratories</td></tr>
<tr><td>b. Sample donors directly through study protocol</td><td>&nbsp;</td><td>f. Industrial/profit research laboratories</td></tr>
<tr><td>c. Korean bio-repositories</td><td>&nbsp;</td><td>g. Colleagues </td></tr>
<tr><td>d. Oversee bio-repositories</td><td>&nbsp;</td><td>h. Other ____________________.</td></tr>
</table>

<p>18. How much are you willing to spend to acquire samples for your work? __________ KRW</p>

19. Where is your primary funding source?

<table width="60%" border="0" cellspacing="0" cellpadding="3" style="margin-left: 2%;">

<tr><td>a. Korean Science and Technology Research Foundation</td><td>&nbsp;</td><td>e. Industry</td></tr>
<tr><td>b. Ministry of Science and Technology </td><td>&nbsp;</td><td>f. Non-profit research foundation </td></tr>
<tr><td>c. Ministry of Health and Welfare</td><td>&nbsp;</td><td>g. Private resource</td></tr>
<tr><td>d. University hospital</td><td>&nbsp;</td><td>h. Other ______________________</td></tr>
</table>

<p>20. What anatomic region is your primary interest for which you require samples? </p>

<table width="60%" border="0" cellspacing="0" cellpadding="3" style="margin-left: 2%;">
<tr><td>a. Cardiovascular and circulatory system</td><td>&nbsp;</td><td>h. Nervous system</td></tr>
<tr><td>b. Digestive system</td><td>&nbsp;</td><td>i. Respiratory system</td></tr>
<tr><td>c. Embryonic structures</td><td>&nbsp;</td><td>j. Sense organs</td></tr>
<tr><td>d. Endocrine system</td><td>&nbsp;</td><td>k. Stomatognathic system</td></tr>
<tr><td>e. Hemic and immune systems</td><td>&nbsp;</td><td>l. Urogenital system</td></tr>
<tr><td>f. Integumentary system</td><td>&nbsp;</td><td>m. Other _______________</td></tr>
<tr><td>g. Musculoskeletal system</td><td>&nbsp;</td><td>&nbsp;</td></tr>
</table>

<p>21. What are your searching criteria if you specify your sample requirement?</p>

<table width="60%" border="0" cellspacing="0" cellpadding="3" style="margin-left: 2%;">
<tr><td>a. By anatomic sites</td><td>&nbsp;</td><td>g. By tissue sources (e.g., surgical or autopsy etc.)</td></tr>
<tr><td>b. Either by normal or diseased</td><td>&nbsp;</td><td>h. By demographic condition, (e.g., age, race, gender or other limiting characteristics)</td></tr>
<tr><td>c. By disease</td><td>&nbsp;</td><td>i. By amount of tissue(s) and minimum to maximum size or dimension</td></tr>
<tr><td>d. By primary or metastatic</td><td>&nbsp;</td><td>j. By total number of specimens needed</td></tr>
<tr><td>e. By matched normal tissue from the same patient</td><td>&nbsp;</td><td>k. By specimen preparation and preservation methods (e.g., fresh or frozen or fixed etc.)</td></tr>
<tr><td>f. By specific treatment(s) performed on patient (e.g., radiation or chemo or hormone etc.)</td><td>&nbsp;</td>
<td>l. Other ________________________</td></tr>
</table>


<p>22. What diseases are of primary interest to you? </p>
<table width="60%" border="0" cellspacing="0" cellpadding="3" style="margin-left: 2%;">
<tr><td>a. Digestive system diseases</td>,td>&nbsp;</td><td>h. Endocrine system diseases</td></tr>
<tr><td>b. Musculoskeletal diseases</td><td>&nbsp;</td><td>i. Urogenital diseases </td></tr>
<tr><td>c. Congenital, hereditary, and neonatal diseases and abnormalities</td><td>&nbsp;</td><td>j. Respiratory tract diseases</td></tr>
<tr><td>d. Cardiovascular diseases</td><td>&nbsp;</td><td>k. Skin and connective tissue diseases</td></tr>
<tr><td>e. Immune system diseases</td><td>&nbsp;</td><td>l. Nervous system diseases</td></tr>
<tr><td>f. Neoplasms</td><td>&nbsp;</td><td>m. Other ______________________</td></tr>
<tr><td>g. Hemic and lymphatic diseases</td><td>&nbsp;</td><td>&nbsp;</td></tr>
</table>

<p>23. What are your publication outlets to report your work using bio-samples?<br />
______________________________________________</p>

<p>24. What professional conferences do you regularly attend? <br />
_____________________________________________</p>
<br />
 <hr style="color: Navy;" size="3" />
<br />

<h2>INFORMATION REQUIREMENTS</h2>

<p>How much do you agree or disagree that the following medical and personal information should accompany human biological samples with 1 representing rarely useful and 5 representing extremely useful?</p> 

<table width="60%" border="0" cellspacing="0" cellpadding="3" style="margin-left: 2%;">
<tr><td>25. Names</td><td>&nbsp;</td><td>58. Operation history</td></tr>
<tr><td>26. Address</td><td>&nbsp;</td><td>59. Preoperative history</td></tr>
<tr><td>27. Dates</td><td>&nbsp;</td><td>60. Family health history</td></tr>
<tr><td>28. Telephone</td><td>&nbsp;</td><td>61. Clinical blood testing result</td></tr>
<tr><td>29. Fax</td><td>&nbsp;</td><td>62. Blood chemistry</td></tr>
<tr><td>30. E-mail</td><td>&nbsp;</td><td>63. Pulmonary function test</td></tr>
<tr><td>31. Social security number</td><td>&nbsp;</td><td>64. Electrolyte</td></tr>
<tr><td>32. Medical record number</td><td>&nbsp;</td><td>65. Virus</td></tr>
<tr><td>33. Health plan beneficiary numbers</td><td>&nbsp;</td><td>66. Gene expression</td></tr>
<tr><td>34. Insurance account numbers</td><td>&nbsp;</td><td>67. Biomarker</td></tr>
<tr><td>35. Certificate or license numbers</td><td>&nbsp;</td><td>68. Organ site</td></tr>
<tr><td>36. Vehicle identifiers and serial numbers</td><td>&nbsp;</td><td>69. How biological samples are processed</td></tr>
<tr><td>37. Device ID and serial numbers</td><td>&nbsp;</td><td>70. Laterality (which  part of organ was taken, left, right, upper, lower, etc.)</td></tr>
<tr><td>38. Web address</td><td>&nbsp;</td><td>71. Pathologic diagnosis</td></tr>
<tr><td>39. Internet protocol address</td><td>&nbsp;</td><td>72. Radiological findings</td></tr>
<tr><td>40. Biometric identifier</td><td>&nbsp;</td><td>73. Treatment history</td></tr>
<tr><td>41. Photo</td><td>&nbsp;</td><td>74. World Health Organization (WHO) histologic grading</td></tr>
<tr><td>42. Unique id (e.g., university number)</td><td>&nbsp;</td><td>75. Progression or recurrence</td></tr>
<tr><td>43. Patient age</td><td>&nbsp;</td><td>76. Pathologic stage</td></tr>
<tr><td>44. Education</td><td>&nbsp;</td><td>77. Pathologic grade</td></tr>
<tr><td>45. Gender</td><td>&nbsp;</td><td>78. Institution where the sample was collected</td></tr>
<tr><td>46. Marital status</td><td>&nbsp;</td><td>79. Date and time collected</td></tr>
<tr><td>47. Occupation</td><td>&nbsp;</td><td>80. Date and time stored</td></tr>
<tr><td>48. Occupational-environmental</td><td>&nbsp;</td><td>81. Sample processed staff </td></tr>
<tr><td>49. Income</td><td>&nbsp;</td><td>82. Tumor or non-tumor</td></tr>
<tr><td>50. Donors residence</td><td>&nbsp;</td><td>83. Sample type</td></tr>
<tr><td>51. Smoking</td><td>&nbsp;</td><td>84. Sample processing method</td></tr>
<tr><td>52. Nutrition</td><td>&nbsp;</td><td>85. Sample amount available</td></tr>
<tr><td>53. Alcohol</td><td>&nbsp;</td><td>86. How much dispensed</td></tr>
<tr><td>54. OBGYN history</td><td>&nbsp;</td><td>87. Where dispensed</td></tr>
<tr><td>55. Drug history</td><td>&nbsp;</td><td>88. Study results for dispensed samples</td></tr>
<tr><td>56. Physical examination (weight/height)</td><td>&nbsp;</td><td>89. Publication information for the sample that are dispensed for other users</td></tr>
<tr><td>57. Clinical diagnosis</td><td>&nbsp;</td><td>90.  Other _______________________</td></tr>
</table>

 <hr size="1" style="color:#5E96FD"> 
<table align="center" cellpadding="10"> 
<tr><td align="center" valign="top"><div>  <a href="http://www.digits.com/" target="_blank">
    <img src="http://counter.digits.com/?counter={49dd92c4-e9d4-7364-f5e0-c84e94de9fea}&amp;am;template=simple" 
     alt="Hit Counter by Digits" border="0"  />
  </a>
</div></td> 
 
<td align="center" valign="top"><div> 
&copy; the author, 2010. <br />Last updated: 12 March, 2010
</div></td> 
 
 
         <td align="center" valign="middle"><img src="../valid-xhtml10.gif" alt="Valid XHTML 1.0!" height="16" width="44" /><!--ONESTAT SCRIPTCODE START--> 
<!--
// Modification of this code is not allowed and will permanently disable your account!
// Account ID : 281971
// Website URL: http://InformationR.net/ir/
// Copyright (C) 2002-2006 OneStat.com All Rights Reserved
--> 

<div id="OneStatTag"><table border='0' cellpadding='0' cellspacing='0'><tr><td align='center'> 
<script type="text/javascript"> 
<!--
function OneStat_Pageview()
{
    var d=document;
    var sid="281971";
    var CONTENTSECTION="";
    var osp_URL=d.URL;
    var osp_Title=d.title;
    var t=new Date();
    var p="http"+(d.URL.indexOf('paper427.html')==0?'s':'')+"://stat.onestat.com/stat.aspx?tagver=2&sid="+sid;
    p+="&url="+escape(osp_URL);
    p+="&ti="+escape(osp_Title);
    p+="&section="+escape(CONTENTSECTION);
    p+="&rf="+escape(parent==self?document.referrer:top.document.referrer);
    p+="&tz="+escape(t.getTimezoneOffset());
    p+="&ch="+escape(t.getHours());
    p+="&js=1";
    p+="&ul="+escape(navigator.appName=="Netscape"?navigator.language:navigator.userLanguage);
    if(typeof(screen)=="object"){
       p+="&sr="+screen.width+"x"+screen.height;p+="&cd="+screen.colorDepth;
       p+="&jo="+(navigator.javaEnabled()?"Yes":"No");
    }
    d.write('<a href="http://www.onestatfree.com/aspx/login.aspx?sid='+sid+'" target=_blank><img id="ONESTAT_TAG" border="0" src="'+p+'" alt="This site tracked by OneStatFree.com. Get your own free site tracker."></'+'a>');
}
 
OneStat_Pageview();
//--> 
</script> 
<noscript> 
<a href="http://www.onestatfree.com/"><img border="0" src="http://stat.onestat.com/stat.aspx?tagver=2&amp;sid=281971&amp;js=No&amp;" alt="online web site analytics" /></a> 
</noscript> 
</td></tr><tr><td align='center'><div style="COLOR:black;display:none;FONT-FAMILY:'Verdana';"><a href="http://www.onestat.com/" style="text-decoration:none;">online web site analytics</a><br /></div></td></tr></table></div> 
<!--ONESTAT SCRIPTCODE END--> 

</td></tr> 
</table> 
<hr size="3" style="color: #5E96FD;" /> 
 
<table align="center"><tr><td><div class="button"> 
 

<ul>
	<li><a href="infres151.html">Contents</a> | </li>
	<li><a href="../iraindex.html">Author index</a> | </li>
	<li><a href="../irsindex.html">Subject index</a> | </li>
	<li><a href="../search.html">Search</a> | </li>
	<li><a href="../index-2.html">Home</a></li>
</ul> 
 
</div></td></tr></table> 
 
 <hr size="3" style="color: #5E96FD;" /> 
<script src="http://www.google-analytics.com/urchin.js" type="text/javascript"> 
</script> 
<script type="text/javascript"> 
_uacct = "UA-672528-1";
urchinTracker();

</script> 
</body> 

<!-- Mirrored from informationr.net/ir/15-1/paper427.html by HTTrack Website Copier/3.x [XR&CO'2010], Sat, 25 Dec 2010 19:33:51 GMT -->
</html> 